diff -Naur orig/libvisio.pc.in patched/libvisio.pc.in
--- orig/libvisio.pc.in	2014-04-16 19:13:10.000000000 +0200
+++ patched/libvisio.pc.in	2014-06-06 23:55:36.951293937 +0200
@@ -10,4 +10,4 @@
 Libs: -L${libdir} -lvisio-@VSD_MAJOR_VERSION@.@VSD_MINOR_VERSION@
 Cflags: -I${includedir}/libvisio-@VSD_MAJOR_VERSION@.@VSD_MINOR_VERSION@
 
-Requires.private: icu-i18n libxml-2.0 zlib
+Requires.private: libxml-2.0 zlib
