diff -Naur orig/tkffmpeg.desktop mod/tkffmpeg.desktop
--- orig/tkffmpeg.desktop	2009-06-15 11:24:08.000000000 +0200
+++ mod/tkffmpeg.desktop	2014-06-23 18:47:36.000000000 +0200
@@ -4,7 +4,8 @@
 Comment= FFmpeg video converter GUI
 Comment[ru]= Интерфейс FFmpeg для конвертирования видео
 Comment[uk]= Інтерфейс FFmpeg для конвертування відео
-Exec=tkffmpeg.tcl
+Comment[hu]=FFmpeg videó konvertáló felület
+Exec=tkffmpeg
 Icon=tkffmpeg
 Terminal=false
-Categories=AudioVideo;
\ Nincs újsor a fájl végén
+Categories=AudioVideo;Video;AudioVideoEditing;
