diff -Naur orig/rules/base.xml.in patched/rules/base.xml.in
--- orig/rules/base.xml.in	2013-08-26 00:29:54.000000000 +0200
+++ patched/rules/base.xml.in	2014-01-19 12:19:30.722370054 +0100
@@ -3356,6 +3356,12 @@
         </variant>
         <variant>
           <configItem>
+            <name>101_qwertz_comma_alldead</name>
+            <_description>101/qwertz/comma/All dead keys</_description>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
             <name>101_qwertz_comma_dead</name>
             <_description>Hungarian (101/qwertz/comma/dead keys)</_description>
           </configItem>
@@ -3368,6 +3374,12 @@
         </variant>
         <variant>
           <configItem>
+            <name>101_qwertz_dot_alldead</name>
+            <_description>101/qwertz/dot/All dead keys</_description>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
             <name>101_qwertz_dot_dead</name>
             <_description>Hungarian (101/qwertz/dot/dead keys)</_description>
           </configItem>
@@ -3380,6 +3392,12 @@
         </variant>
         <variant>
           <configItem>
+            <name>101_qwerty_comma_alldead</name>
+            <_description>101/qwerty/comma/All dead keys</_description>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
             <name>101_qwerty_comma_dead</name>
             <_description>Hungarian (101/qwerty/comma/dead keys)</_description>
           </configItem>
@@ -3392,6 +3410,12 @@
         </variant>
         <variant>
           <configItem>
+            <name>101_qwerty_dot_alldead</name>
+            <_description>101/qwerty/dot/All dead keys</_description>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
             <name>101_qwerty_dot_dead</name>
             <_description>Hungarian (101/qwerty/dot/dead keys)</_description>
           </configItem>
@@ -3404,6 +3428,12 @@
         </variant>
         <variant>
           <configItem>
+            <name>102_qwertz_comma_alldead</name>
+            <_description>102/qwertz/comma/All dead keys</_description>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
             <name>102_qwertz_comma_dead</name>
             <_description>Hungarian (102/qwertz/comma/dead keys)</_description>
           </configItem>
@@ -3416,6 +3446,12 @@
         </variant>
         <variant>
           <configItem>
+            <name>102_qwertz_dot_alldead</name>
+            <_description>102/qwertz/dot/All dead keys</_description>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
             <name>102_qwertz_dot_dead</name>
             <_description>Hungarian (102/qwertz/dot/dead keys)</_description>
           </configItem>
@@ -3428,6 +3464,12 @@
         </variant>
         <variant>
           <configItem>
+            <name>102_qwerty_comma_alldead</name>
+            <_description>102/qwerty/comma/All dead keys</_description>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
             <name>102_qwerty_comma_dead</name>
             <_description>Hungarian (102/qwerty/comma/dead keys)</_description>
           </configItem>
@@ -3439,6 +3481,12 @@
           </configItem>
         </variant>
         <variant>
+          <configItem>
+            <name>102_qwerty_dot_alldead</name>
+            <_description>102/qwerty/dot/All dead keys</_description>
+          </configItem>
+        </variant>
+        <variant>
           <configItem>
             <name>102_qwerty_dot_dead</name>
             <_description>Hungarian (102/qwerty/dot/dead keys)</_description>
diff -Naur orig/symbols/hu patched/symbols/hu
--- orig/symbols/hu	2014-01-19 12:17:42.000000000 +0100
+++ patched/symbols/hu	2014-01-19 12:19:30.722370054 +0100
@@ -54,6 +54,22 @@
 
 // Main layouts
 
+// 101_qwertz_comma_alldead
+// 101 key qwertz layout
+// with decimal comma on keypad
+// and with dead key support for all keys
+partial
+xkb_symbols "101_qwertz_comma_alldead" {
+    name[Group1] = "Hungary - 101/qwertz/comma/All dead keys";
+    include "latin"
+    include "hu(def_101)"
+    include "hu(def_qwertz)"
+    include "keypad(comma)"
+    include "hu(def_alldead)"
+    include "hu(def_common)"
+    include "level3(ralt_switch)"
+};
+
 // 101_qwertz_comma_dead
 // 101 key qwertz layout
 // with decimal comma on keypad
@@ -86,6 +102,22 @@
     include "level3(ralt_switch)"
 };
 
+// 101_qwertz_dot_alldead
+// 101 key qwertz layout
+// with decimal dot on keypad
+// and with dead key support for all keys
+partial
+xkb_symbols "101_qwertz_dot_alldead" {
+    name[Group1] = "Hungary - 101/qwertz/dot/All dead keys";
+    include "latin"
+    include "hu(def_101)"
+    include "hu(def_qwertz)"
+    include "hu(def_dot)"
+    include "hu(def_alldead)"
+    include "hu(def_common)"
+    include "level3(ralt_switch)"
+};
+
 // 101_qwertz_dot_dead
 // 101 key qwertz layout
 // with decimal dot on keypad
@@ -118,6 +150,22 @@
     include "level3(ralt_switch)"
 };
 
+// 101_qwerty_comma_alldead
+// 101 key qwerty layout
+// with decimal comma on keypad
+// and with dead key support for all keys
+partial
+xkb_symbols "101_qwerty_comma_alldead" {
+    name[Group1] = "Hungary - 101/qwerty/comma/All dead keys";
+    include "latin"
+    include "hu(def_101)"
+    include "hu(def_qwerty)"
+    include "keypad(comma)"
+    include "hu(def_alldead)"
+    include "hu(def_common)"
+    include "level3(ralt_switch)"
+};
+
 // 101_qwerty_comma_dead
 // 101 key qwerty layout
 // with decimal comma on keypad
@@ -150,6 +198,22 @@
     include "level3(ralt_switch)"
 };
 
+// 101_qwerty_dot_alldead
+// 101 key qwerty layout
+// with decimal dot on keypad
+// and with dead key support for all keys
+partial
+xkb_symbols "101_qwerty_dot_alldead" {
+    name[Group1] = "Hungary - 101/qwerty/dot/All dead keys";
+    include "latin"
+    include "hu(def_101)"
+    include "hu(def_qwerty)"
+    include "hu(def_dot)"
+    include "hu(def_alldead)"
+    include "hu(def_common)"
+    include "level3(ralt_switch)"
+};
+
 // 101_qwerty_dot_dead
 // 101 key qwerty layout
 // with decimal dot on keypad
@@ -182,6 +246,22 @@
     include "level3(ralt_switch)"
 };
 
+// 102_qwertz_comma_alldead
+// 102 key qwertz layout
+// with decimal comma on keypad
+// and with dead key support for all keys
+partial
+xkb_symbols "102_qwertz_comma_alldead" {
+    name[Group1] = "Hungary - 102/qwertz/comma/All dead keys";
+    include "latin"
+    include "hu(def_102)"
+    include "hu(def_qwertz)"
+    include "keypad(comma)"
+    include "hu(def_alldead)"
+    include "hu(def_common)"
+    include "level3(ralt_switch)"
+};
+
 // 102_qwertz_comma_dead
 // 102 key qwertz layout
 // with decimal comma on keypad
@@ -214,6 +294,22 @@
     include "level3(ralt_switch)"
 };
 
+// 102_qwertz_dot_alldead
+// 102 key qwertz layout
+// with decimal dot on keypad
+// and with dead key support for all keys
+partial
+xkb_symbols "102_qwertz_dot_alldead" {
+    name[Group1] = "Hungary - 102/qwertz/dot/All dead keys";
+    include "latin"
+    include "hu(def_102)"
+    include "hu(def_qwertz)"
+    include "hu(def_dot)"
+    include "hu(def_alldead)"
+    include "hu(def_common)"
+    include "level3(ralt_switch)"
+};
+
 // 102_qwertz_dot_dead
 // 102 key qwertz layout
 // with decimal dot on keypad
@@ -246,6 +342,22 @@
     include "level3(ralt_switch)"
 };
 
+// 102_qwerty_comma_alldead
+// 102 key qwerty layout
+// with decimal comma on keypad
+// and with dead key support for all keys
+partial
+xkb_symbols "102_qwerty_comma_alldead" {
+    name[Group1] = "Hungary - 102/qwerty/comma/All dead keys";
+    include "latin"
+    include "hu(def_102)"
+    include "hu(def_qwerty)"
+    include "keypad(comma)"
+    include "hu(def_alldead)"
+    include "hu(def_common)"
+    include "level3(ralt_switch)"
+};
+
 // 102_qwerty_comma_dead
 // 102 key qwerty layout
 // with decimal comma on keypad
@@ -278,6 +390,22 @@
     include "level3(ralt_switch)"
 };
 
+// 102_qwerty_dot_alldead
+// 102 key qwerty layout
+// with decimal dot on keypad
+// and with dead key support for all keys
+partial
+xkb_symbols "102_qwerty_dot_alldead" {
+    name[Group1] = "Hungary - 102/qwerty/dot/All dead keys";
+    include "latin"
+    include "hu(def_102)"
+    include "hu(def_qwerty)"
+    include "hu(def_dot)"
+    include "hu(def_alldead)"
+    include "hu(def_common)"
+    include "level3(ralt_switch)"
+};
+
 // 102_qwerty_dot_dead
 // 102 key qwerty layout
 // with decimal dot on keypad
@@ -355,6 +483,24 @@
 }; 	 
  
 
+// def_alldead:
+// All the possible keys are dead keys
+hidden partial alphanumeric_keys
+xkb_symbols "def_alldead" {
+    key <AE01>  { [            1,   apostrophe,        dead_tilde,      asciitilde ] };
+    key <AE02>  { [            2,     quotedbl,        dead_caron,           caron ] };
+    key <AE03>  { [            3,         plus,   dead_circumflex,     asciicircum ] };
+    key <AE04>  { [            4,       exclam,        dead_breve,           breve ] };
+    key <AE05>  { [            5,      percent,    dead_abovering,          degree ] };
+    key <AE06>  { [            6,        slash,       dead_ogonek,          ogonek ] };
+    key <AE07>  { [            7,        equal,        dead_grave,           grave ] };
+    key <AE08>  { [            8,    parenleft,     dead_abovedot,        abovedot ] };
+    key <AE09>  { [            9,   parenright,        dead_acute,           acute ] };
+    key <AE10>  { [   odiaeresis,   Odiaeresis,  dead_doubleacute,     doubleacute ] };
+    key <AE11>  { [   udiaeresis,   Udiaeresis,    dead_diaeresis,       diaeresis ] };
+    key <AE12>  { [       oacute,       Oacute,      dead_cedilla,         cedilla ] };
+};
+
 // def_dead:
 // Dead keys support part
 hidden partial alphanumeric_keys
