--- ModemManager-0.4.orig/configure	2010-06-16 19:31:58.000000000 +0200
+++ ModemManager-0.4/configure	2011-03-27 20:38:03.000000000 +0200
@@ -12618,7 +12618,7 @@
 if test "$GCC" = "yes" -a "$set_more_warnings" != "no"; then
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
-	CFLAGS="-Wall -Werror -std=gnu89 $CFLAGS"
+	CFLAGS="-Wall -std=gnu89 $CFLAGS"
 
 	for option in -Wshadow -Wmissing-declarations -Wmissing-prototypes \
 		      -Wdeclaration-after-statement -Wstrict-prototypes \
