diff -Naur coreutils-5.93.orig/src/su.c coreutils-5.93/src/su.c
--- coreutils-5.93.orig/src/su.c	2005-08-23 17:09:13.000000000 +0200
+++ coreutils-5.93/src/su.c	2006-02-02 19:56:21.000000000 +0100
@@ -122,18 +122,10 @@
 #endif
 
 /* The default PATH for simulated logins to non-superuser accounts.  */
-#ifdef _PATH_DEFPATH
-# define DEFAULT_LOGIN_PATH _PATH_DEFPATH
-#else
-# define DEFAULT_LOGIN_PATH ":/usr/ucb:/bin:/usr/bin"
-#endif
+#define DEFAULT_LOGIN_PATH "/bin:/usr/bin"
 
 /* The default PATH for simulated logins to superuser accounts.  */
-#ifdef _PATH_DEFPATH_ROOT
-# define DEFAULT_ROOT_LOGIN_PATH _PATH_DEFPATH_ROOT
-#else
-# define DEFAULT_ROOT_LOGIN_PATH "/usr/ucb:/bin:/usr/bin:/etc"
-#endif
+#define DEFAULT_ROOT_LOGIN_PATH "/sbin:/bin:/usr/sbin:/usr/bin"
 
 /* The shell to run if none is given in the user's passwd entry.  */
 #define DEFAULT_SHELL "/bin/sh"
