diff -urN ebtables-v2.0.6.orig/Makefile ebtables-v2.0.6/Makefile
--- ebtables-v2.0.6.orig/Makefile	2003-11-02 19:22:56.000000000 +0100
+++ ebtables-v2.0.6/Makefile	2004-06-21 15:12:36.000000000 +0200
@@ -4,7 +4,7 @@
 PROGVERSION:=2.0.6
 PROGDATE:=November\ 2003
 
-MANDIR?=/usr/local/man
+MANDIR?=$(DESTDIR)/usr/share/man
 CFLAGS:=-Wall -Wunused
 CC:=gcc
 
@@ -18,10 +18,10 @@
 
 KERNEL_INCLUDES?=include/
 
-ETHERTYPESPATH?=/etc
+ETHERTYPESPATH?=$(DESTDIR)/etc
 ETHERTYPESFILE:=$(ETHERTYPESPATH)/ethertypes
 
-BINPATH?=/sbin/
+BINPATH?=$(DESTDIR)/sbin/
 BINFILE:=$(BINPATH)ebtables
 
 PROGSPECS:=-DPROGVERSION=\"$(PROGVERSION)\" \
