diff -Naurd orig/Makefile mod/Makefile
--- orig/Makefile	2010-02-10 20:38:55.000000000 +0100
+++ mod/Makefile	2010-03-12 20:13:40.000000000 +0100
@@ -1,7 +1,7 @@
 # Makefile
 #
 
-INSTALL_PATH = /usr/local
+INSTALL_PATH = /usr
 
 # For SCO
 CFLAGS = -b elf -O -D_SVID
@@ -23,7 +23,7 @@
 
 SHELL = /bin/sh
 
-all: shc ask_for_test
+all: shc #ask_for_test
 
 shc: shc.c
 	$(CC) $(CFLAGS) $@.c -o $@
@@ -73,9 +73,8 @@
 
 install: shc
 	@echo '***	Installing shc and shc.1 on '$(INSTALL_PATH)
-	@echo -n '***	¿Do you want to continue? '; read ANS; case "$$ANS" in y|Y|yes|Yes|YES) ;; *) exit 1;; esac;
-	install -c -s shc $(INSTALL_PATH)/bin/
-	install -c -m 644 shc.1 $(INSTALL_PATH)/man/man1/
+	install -c -s shc $(DESTDIR)$(INSTALL_PATH)/bin/
+	install -c -m 644 shc.1 $(DESTDIR)$(INSTALL_PATH)/share/man/man1/
 
 clean:
 	rm -f *.o *~ *.x.c
