diff -Naur orig/xdo.c patched/xdo.c
--- orig/xdo.c	2011-05-30 09:17:18.000000000 +0200
+++ patched/xdo.c	2013-09-14 14:22:55.821014907 +0200
@@ -27,7 +27,7 @@
 #include <X11/Xutil.h>
 #include <X11/extensions/XTest.h>
 #include <X11/extensions/Xinerama.h>
-#include <X11/keysym.h>
+#include <X11/XKBlib.h>
 #include <X11/cursorfont.h>
 
 #include "xdo.h"
@@ -1475,8 +1475,8 @@
         offset = 2;
       }
 
-      if (XKeycodeToKeysym(xdo->xdpy, key, 0 + offset) != sym
-          && XKeycodeToKeysym(xdo->xdpy, key, 1 + offset) != sym) {
+      if (XkbKeycodeToKeysym(xdo->xdpy, key, 0 + offset, 0) != sym
+          && XkbKeycodeToKeysym(xdo->xdpy, key, 1 + offset, 0) != sym) {
         key = 0;
       }
     }
@@ -1488,9 +1488,7 @@
       (*keys)[*nkeys].code = 0;
     } else {
       /* Inject a shift key if we need to press shift to reach this keysym */
-      //if (xdo->keymap[key * xdo->keysyms_per_keycode] == sym
-          //|| sym == NoSymbol) {
-      if ((XKeycodeToKeysym(xdo->xdpy, key, 0) == sym)
+      if ((XkbKeycodeToKeysym(xdo->xdpy, key, 0, 0) == sym)
           || sym == NoSymbol) {
         /* sym is NoSymbol if we give a keycode to type */
         (*keys)[*nkeys].index = 0;
