diff -Naur orig/tools/Linux/xbmc.desktop mod/tools/Linux/xbmc.desktop
--- orig/tools/Linux/xbmc.desktop	2013-12-12 22:47:49.000000000 +0100
+++ mod/tools/Linux/xbmc.desktop	2014-06-06 21:13:28.000000000 +0200
@@ -1,22 +1,11 @@
 [Desktop Entry]
-Version=1.0
 Name=XBMC Media Center
 GenericName=Media Center
-Comment=Manage and view your media 
+GenericName[hu]=Média központ
+Comment=Manage and view your media
+Comment[hu]=Videók és zenék kezelése
 Exec=xbmc
 Icon=xbmc
 Terminal=false
 Type=Application
-Categories=AudioVideo;Video;Player;TV;
-
-Actions=Fullscreen;Standalone;
-
-[Desktop Action Fullscreen]
-Name=Open in fullscreen
-Exec=xbmc -fs
-OnlyShowIn=Unity;
-
-[Desktop Action Standalone]
-Name=Open in standalone mode
-Exec=xbmc --standalone
-OnlyShowIn=Unity;
+Categories=AudioVideo;Video;Player;
