diff -Naur orig/tools/Linux/xbmc.desktop mod/tools/Linux/xbmc.desktop
--- orig/tools/Linux/xbmc.desktop	2013-03-20 17:14:42.000000000 +0100
+++ mod/tools/Linux/xbmc.desktop	2013-07-28 17:21:51.000000000 +0200
@@ -7,16 +7,5 @@
 Icon=xbmc
 Terminal=false
 Type=Application
-Categories=AudioVideo;Video;Player;TV;
+Categories=AudioVideo;Video;Player;TV;X-UHU-Menu/Multimedia/VideoPlayers;
 
-Actions=Fullscreen;Standalone;
-
-[Desktop Action Fullscreen]
-Name=Open in fullscreen
-Exec=xbmc -fs
-OnlyShowIn=Unity;
-
-[Desktop Action Standalone]
-Name=Open in standalone mode
-Exec=xbmc --standalone
-OnlyShowIn=Unity;
