diff -Naurd a/src/font.cpp b/src/font.cpp
--- a/src/font.cpp	2007-09-30 18:07:08.000000000 +0200
+++ b/src/font.cpp	2007-11-08 20:27:20.000000000 +0100
@@ -74,13 +74,6 @@
 
     vertical_spacing = (desc.vertical_spacing == -1) ? surface->h : desc.vertical_spacing;
 
-    if (surface->format->BitsPerPixel != 32)
-      {
-        std::cout << "Error: '" << desc.pathname.str() << "' invalid, fonts need to be RGBA, but is "
-                  << surface->format->BitsPerPixel << "bpp" << std::endl;
-        assert(0);
-      }
-        
     SDL_LockSurface(surface);
     
     if (!desc.monospace)
