diff -up ./src/input/evdev_device.cpp.orig ./src/input/evdev_device.cpp
--- ./src/input/evdev_device.cpp.orig	2007-10-01 19:24:54.000000000 +0200
+++ ./src/input/evdev_device.cpp	2009-02-19 16:35:26.000000000 +0100
@@ -25,6 +25,8 @@
 #include <stdexcept>
 #include <iostream>
 #include <errno.h>
+#include <cstring>
+#include <cstdio>
 
 #include "evdev_device.hpp"
 
diff -up ./src/tinygettext/po_file_reader.cpp.orig ./src/tinygettext/po_file_reader.cpp
--- ./src/tinygettext/po_file_reader.cpp.orig	2007-10-18 19:54:01.000000000 +0200
+++ ./src/tinygettext/po_file_reader.cpp	2009-02-19 15:11:32.000000000 +0100
@@ -26,6 +26,7 @@
 #include <string.h>
 #include "iconv.hpp"
 #include "po_file_reader.hpp"
+#include <cstdio>
 
 namespace TinyGetText {
 
