--- hugin.orig/src/hugin1/hugin/hugin.desktop	2009-12-01 23:39:01.000000000 +0100
+++ hugin/src/hugin1/hugin/hugin.desktop	2010-02-08 20:01:41.000000000 +0100
@@ -4,7 +4,7 @@
 Name[de]=Hugin - Panorama-Editor
 Name[fr]=Hugin - Créateur de panoramas
 Name[ru]=Сшиватель панорам Hugin
-Name[hu]=Hugin panoráma készítő
+Name[hu]=Hugin
 Comment=Stitch photographs together
 Comment[cs]=Skládání panoramatických snímků
 Comment[de]=Erzeugt Panoramabildern aus mehreren Einzelbildern.
@@ -17,7 +17,7 @@
 GenericName[cs]=Nástroj pro skládání panoramat
 GenericName[ru]=Сшиватель панорам
 GenericName[hu]=Panoráma összefűző
-Categories=Graphics;
+Categories=GTK;Graphics;X-UHU-Menu/Graphics;
 Exec=hugin
 Icon=hugin
 StartupNotify=false
