http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=560516

diff -Nru strace-4.5.19/configure strace-4.5.19/configure
--- strace-4.5.19/configure	2009-10-21 19:41:11.000000000 +0200
+++ strace-4.5.19/configure	2010-01-26 08:47:33.000000000 +0100
@@ -5880,7 +5880,7 @@
 do :
   as_ac_Header=`$as_echo "ac_cv_header_$ac_header" | $as_tr_sh`
 ac_fn_c_check_header_compile "$LINENO" "$ac_header" "$as_ac_Header" "#include <stddef.h>
-#include <linux/socket.h>
+#include <sys/socket.h>
 "
 eval as_val=\$$as_ac_Header
    if test "x$as_val" = x""yes; then :

