diff -Naur orig/lxqtleave/translations/lxqt-hibernate_hu.desktop mod/lxqtleave/translations/lxqt-hibernate_hu.desktop
--- orig/lxqtleave/translations/lxqt-hibernate_hu.desktop	1970-01-01 01:00:00.000000000 +0100
+++ mod/lxqtleave/translations/lxqt-hibernate_hu.desktop	2015-05-04 20:09:10.758345514 +0200
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Type=Application
+Name=Hibernate
+GenericName=Hibernate
+Comment=hibernate the machine
+Exec=lxqt-leave --hibernate
+Icon=system-suspend-hibernate
+Categories=LXQt;X-Leave
+OnlyShowIn=LXQt
+
+#TRANSLATIONS_DIR=../translations
+
+# Translations
+Comment[hu]=Mély készenléti állapot
+Name[hu]=Mélykészenlét
diff -Naur orig/lxqtleave/translations/lxqt-lockscreen_hu.desktop mod/lxqtleave/translations/lxqt-lockscreen_hu.desktop
--- orig/lxqtleave/translations/lxqt-lockscreen_hu.desktop	1970-01-01 01:00:00.000000000 +0100
+++ mod/lxqtleave/translations/lxqt-lockscreen_hu.desktop	2015-05-04 20:16:49.942327268 +0200
@@ -0,0 +1,16 @@
+[Desktop Entry]
+Type=Application
+Name=Lock Screen
+GenericName=Lock Screen
+Comment=Lock the current session
+Exec=lxqt-leave --lockscreen
+Icon=system-lock-screen
+Categories=LXQt;Screensaver
+OnlyShowIn=LXQt;
+
+#TRANSLATIONS_DIR=../translations
+
+
+# Translations
+Comment[hu]=A gép zárolása
+Name[hu]=Képernyőzár
diff -Naur orig/lxqtleave/translations/lxqt-logout_hu.desktop mod/lxqtleave/translations/lxqt-logout_hu.desktop
--- orig/lxqtleave/translations/lxqt-logout_hu.desktop	1970-01-01 01:00:00.000000000 +0100
+++ mod/lxqtleave/translations/lxqt-logout_hu.desktop	2015-05-04 20:17:14.030326310 +0200
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Type=Application
+Name=Logout
+GenericName=Logout
+Comment=Logout from the current session
+Exec=lxqt-leave --logout
+Icon=system-log-out
+Categories=LXQt;X-Leave
+OnlyShowIn=LXQt;
+
+#TRANSLATIONS_DIR=../translations
+
+# Translations
+Comment[hu]=A munkamenet befelyezése
+Name[hu]=Kijelentkezés
diff -Naur orig/lxqtleave/translations/lxqt-reboot_hu.desktop mod/lxqtleave/translations/lxqt-reboot_hu.desktop
--- orig/lxqtleave/translations/lxqt-reboot_hu.desktop	1970-01-01 01:00:00.000000000 +0100
+++ mod/lxqtleave/translations/lxqt-reboot_hu.desktop	2015-05-04 20:17:30.510325656 +0200
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Type=Application
+Name=Reboot
+GenericName=Reboot
+Comment=reboot the machine
+Exec=lxqt-leave --reboot
+Icon=system-reboot
+Categories=LXQt;X-Leave
+OnlyShowIn=LXQt;
+
+#TRANSLATIONS_DIR=../translations
+
+# Translations
+Comment[hu]=A gép újraindítása
+Name[hu]=Újraindítás
diff -Naur orig/lxqtleave/translations/lxqt-shutdown_hu.desktop mod/lxqtleave/translations/lxqt-shutdown_hu.desktop
--- orig/lxqtleave/translations/lxqt-shutdown_hu.desktop	1970-01-01 01:00:00.000000000 +0100
+++ mod/lxqtleave/translations/lxqt-shutdown_hu.desktop	2015-05-04 20:17:43.142325154 +0200
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Type=Application
+Name=Shutdown
+GenericName=Shutdown
+Comment=shutdown the machine
+Exec=lxqt-leave --shutdown
+Icon=system-shutdown
+Categories=LXQt;X-Leave
+OnlyShowIn=LXQt;
+
+#TRANSLATIONS_DIR=../translations
+
+# Translations
+Comment[hu]=A gép kikapcsolása
+Name[hu]=Kikapcsolás
diff -Naur orig/lxqtleave/translations/lxqt-suspend_hu.desktop mod/lxqtleave/translations/lxqt-suspend_hu.desktop
--- orig/lxqtleave/translations/lxqt-suspend_hu.desktop	1970-01-01 01:00:00.000000000 +0100
+++ mod/lxqtleave/translations/lxqt-suspend_hu.desktop	2015-05-04 20:06:49.823550404 +0200
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Type=Application
+Name=Suspend
+GenericName=Suspend
+Comment=suspend the machine
+Exec=lxqt-leave --suspend
+Icon=system-suspend
+Categories=LXQt;X-Leave
+OnlyShowIn=LXQt
+
+#TRANSLATIONS_DIR=../translations
+
+# Translations
+Comment[hu]=A gép készenléti állapotba helyezése
+Name[hu]=Készenlét
