diff -urN xfig.3.2.4.orig/SimpleMenu.c xfig.3.2.4/SimpleMenu.c
--- xfig.3.2.4.orig/SimpleMenu.c	2002-09-20 22:28:45.000000000 +0200
+++ xfig.3.2.4/SimpleMenu.c	2003-04-23 19:05:02.000000000 +0200
@@ -39,10 +39,10 @@
 #include <X11/IntrinsicP.h>
 #include <X11/StringDefs.h>
 
-#include <X11/Xaw3d/XawInit.h>
-#include <X11/Xaw3d/SimpleMenP.h>
-#include <X11/Xaw3d/SmeBSB.h>
-#include <X11/Xaw3d/Cardinals.h>
+#include <X11/Xaw/XawInit.h>
+#include <X11/Xaw/SimpleMenP.h>
+#include <X11/Xaw/SmeBSB.h>
+#include <X11/Xaw/Cardinals.h>
 
 #include <X11/Xmu/Initer.h>
 #include <X11/Xmu/CharSet.h>
