diff -Naurd a/wxmaxima.desktop b/wxmaxima.desktop
--- a/wxmaxima.desktop	2012-01-19 11:05:06.000000000 +0100
+++ b/wxmaxima.desktop	2013-06-09 07:43:46.000000000 +0200
@@ -1,9 +1,11 @@
 [Desktop Entry]
+Version=1.1
 Name=wxMaxima
 Comment=Perform symbolic and numeric calculations using Maxima
+Comment[hu]=Matematikai programcsomag
 Exec=wxmaxima %f
 Icon=wxmaxima.png
 Terminal=false
 Type=Application
-Categories=Education;Science;Math;X-Red-Hat-Base;X-Red-Hat-Base-Only;
+Categories=Education;Science;Math;X-UHU-Menu/Edutainment/Mathematics;
 MimeType=text/x-wxmaxima-batch;
