diff -Naurdp a/src/gnome-terminal.schemas.in b/src/gnome-terminal.schemas.in
--- a/src/gnome-terminal.schemas.in	2008-09-11 15:18:56.000000000 +0200
+++ b/src/gnome-terminal.schemas.in	2008-09-11 15:20:15.000000000 +0200
@@ -221,7 +221,7 @@
       <applyto>/apps/gnome-terminal/profiles/Default/allow_bold</applyto>
       <owner>gnome-terminal</owner>
       <type>bool</type>
-      <default>true</default>
+      <default>false</default>
       <locale name="C">
          <short>Whether to allow bold text</short>
          <long>
@@ -451,7 +451,7 @@
       <applyto>/apps/gnome-terminal/profiles/Default/font</applyto>
       <owner>gnome-terminal</owner>
       <type>string</type>
-      <default>Monospace 12</default>
+      <default>DejaVu Sans Mono 10</default>
       <locale name="C">
          <short>Font</short>
          <long>
@@ -598,7 +598,7 @@
       <applyto>/apps/gnome-terminal/profiles/Default/use_system_font</applyto>
       <owner>gnome-terminal</owner>
       <type>bool</type>
-      <default>true</default>
+      <default>false</default>
       <locale name="C">
          <short>Whether to use the system font</short>
          <long>
