diff -Naurd orig/data/viewnior.desktop.in.in mod/data/viewnior.desktop.in.in
--- orig/data/viewnior.desktop.in.in	2009-06-24 12:51:23.000000000 +0200
+++ mod/data/viewnior.desktop.in.in	2009-07-11 13:23:08.000000000 +0200
@@ -7,5 +7,5 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Graphics;RasterGraphics;Viewer;
+Categories=GNOME;GTK;Graphics;RasterGraphics;Viewer;X-UHU-Menu/Tools/Viewers;
 MimeType=image/bmp;image/gif;image/jpeg;image/jpg;image/pjpeg;image/png;image/tiff;image/x-bmp;image/x-gray;image/x-icb;image/x-ico;image/x-png;image/x-portable-anymap;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/x-xbitmap;image/x-xpixmap;image/x-pcx;image/svg+xml;image/svg+xml-compressed;image/vnd.wap.wbmp;
