diff -Naur orig/dlls/ntdll/server.c patched/dlls/ntdll/server.c
--- orig/dlls/ntdll/server.c	2013-03-09 15:46:13.000000000 +0100
+++ patched/dlls/ntdll/server.c	2013-03-09 15:51:52.652348138 +0100
@@ -95,6 +95,8 @@
 #define SOCKETNAME "socket"        /* name of the socket file */
 #define LOCKNAME   "lock"          /* name of the lock file */
 
+#define SCHED_RESET_ON_FORK 0x40000000
+
 #ifdef __i386__
 static const enum cpu_type client_cpu = CPU_x86;
 #elif defined(__x86_64__)
diff -Naur orig/server/rtkit.c patched/server/rtkit.c
--- orig/server/rtkit.c	2013-03-09 15:46:13.000000000 +0100
+++ patched/server/rtkit.c	2013-03-09 15:52:29.508688310 +0100
@@ -47,6 +47,8 @@
 
 #define FUNCPTR(fn) static typeof(fn) *p ##fn
 
+#define SCHED_RESET_ON_FORK 0x40000000
+
 FUNCPTR(dbus_error_init);
 FUNCPTR(dbus_error_free);
 FUNCPTR(dbus_bus_get);
