diff -Naur orig/server/ptrace.c patched/server/ptrace.c
--- orig/server/ptrace.c	2013-09-27 20:32:14.000000000 +0200
+++ patched/server/ptrace.c	2013-09-29 23:00:48.312192005 +0200
@@ -533,10 +533,11 @@
 
 #ifdef HAVE_SYS_USER_H
 #include <sys/user.h>
-#endif
+#else
 #ifdef HAVE_ASM_USER_H
 #include <asm/user.h>
 #endif
+#endif
 
 /* debug register offset in struct user */
 #define DR_OFFSET(dr) ((((struct user *)0)->u_debugreg) + (dr))
