diff -Naur mdadm-3.1.1-orig/Assemble.c mdadm-3.1.1/Assemble.c
--- mdadm-3.1.1-orig/Assemble.c	2009-11-19 06:13:29.000000000 +0100
+++ mdadm-3.1.1/Assemble.c	2009-11-28 20:30:21.000000000 +0100
@@ -746,6 +746,7 @@
 	content->update_private = NULL;
 
 	if (devcnt == 0) {
+		if (verbose >= 0)
 		fprintf(stderr, Name ": no devices found for %s\n",
 			mddev);
 		if (st)
diff -Naur mdadm-3.1.1-orig/mdadm.c mdadm-3.1.1/mdadm.c
--- mdadm-3.1.1-orig/mdadm.c	2009-11-19 06:13:29.000000000 +0100
+++ mdadm-3.1.1/mdadm.c	2009-11-28 20:32:03.000000000 +0100
@@ -1197,11 +1197,13 @@
 				}
 #endif
 				if (cnt == 0 && rv == 0) {
+					if (verbose)
 					fprintf(stderr, Name ": No arrays found in config file or automatically\n");
 					rv = 1;
 				} else if (cnt)
 					rv = 0;
 			} else if (cnt == 0 && rv == 0) {
+				if (verbose)
 				fprintf(stderr, Name ": No arrays found in config file\n");
 				rv = 1;
 			}
