diff -Naur orig/Makefile patched/Makefile
--- orig/Makefile	2007-01-29 13:25:50.000000000 +0100
+++ patched/Makefile	2008-01-18 03:17:46.000000000 +0100
@@ -9,14 +9,14 @@
 	$(CC) $(CFLAGS) -c smokingcessationapplet.c
 
 install:
-	mkdir -p $(installroot)/usr/lib/gnome-panel/
-	cp -f smokingcessationapplet $(installroot)/usr/lib/gnome-panel/
-	mkdir -p $(installroot)/usr/share/pixmaps/
-	cp -f smokingcessationcigarette.png $(installroot)/usr/share/pixmaps/
-	mkdir -p $(installroot)/usr/share/pixmaps/
-	cp -f smokingcessationcigarettecrossed.png $(installroot)/usr/share/pixmaps/
-	mkdir -p $(installroot)/usr/lib/bonobo/servers/
-	cp -f SmokingCessationApplet_Factory.server $(installroot)/usr/lib/bonobo/servers/
+	mkdir -p ${UB_INSTALLDIR}/usr/lib/gnome-panel/
+	cp -f smokingcessationapplet ${UB_INSTALLDIR}/usr/lib/gnome-panel/
+	mkdir -p ${UB_INSTALLDIR}/usr/share/pixmaps/
+	cp -f smokingcessationcigarette.png ${UB_INSTALLDIR}/usr/share/pixmaps/
+	mkdir -p ${UB_INSTALLDIR}/usr/share/pixmaps/
+	cp -f smokingcessationcigarettecrossed.png ${UB_INSTALLDIR}/usr/share/pixmaps/
+	mkdir -p ${UB_INSTALLDIR}/usr/lib/bonobo/servers/
+	cp -f SmokingCessationApplet_Factory.server ${UB_INSTALLDIR}/usr/lib/bonobo/servers/
 
 uninstall:
 	@echo You must be root to uninstall.
