diff -Naur orig/Thunar-bulk-rename.desktop.in.in patched/Thunar-bulk-rename.desktop.in.in
--- orig/Thunar-bulk-rename.desktop.in.in	2010-12-05 16:07:58.000000000 +0100
+++ patched/Thunar-bulk-rename.desktop.in.in	2010-12-06 18:08:45.000000000 +0100
@@ -7,6 +7,6 @@
 Terminal=false
 StartupNotify=true
 Type=Application
-Categories=System;Utility;Core;GTK;Filesystem;
+Categories=System;Utility;Core;GTK;Filesystem;X-UHU-Menu/Tools/File_Managers;
 
 # vi:set encoding=UTF-8:
diff -Naur orig/Thunar.desktop.in.in patched/Thunar.desktop.in.in
--- orig/Thunar.desktop.in.in	2010-12-05 16:07:58.000000000 +0100
+++ patched/Thunar.desktop.in.in	2010-12-06 18:09:36.000000000 +0100
@@ -7,6 +7,6 @@
 Terminal=false
 StartupNotify=true
 Type=Application
-Categories=System;Utility;Core;GTK;FileTools;FileManager;
+Categories=System;Utility;Core;GTK;FileTools;FileManager;X-UHU-Menu/Tools/File_Managers;
 
 # vi:set encoding=UTF-8:
diff -Naur orig/Thunar-folder-handler.desktop.in.in patched/Thunar-folder-handler.desktop.in.in
--- orig/Thunar-folder-handler.desktop.in.in	2010-12-05 16:07:58.000000000 +0100
+++ patched/Thunar-folder-handler.desktop.in.in	2010-12-06 18:09:12.000000000 +0100
@@ -9,7 +9,7 @@
 Terminal=false
 StartupNotify=true
 Type=Application
-Categories=System;Utility;Core;GTK;FileTools;FileManager;
+Categories=System;Utility;Core;GTK;FileTools;FileManager;X-UHU-Menu/Tools/File_Managers;
 MimeType=inode/directory;
 
 # vi:set encoding=UTF-8:
