diff -Naur orig/panel-plugin/systemload.c patched/panel-plugin/systemload.c
--- orig/panel-plugin/systemload.c	2012-06-23 12:26:11.000000000 +0200
+++ patched/panel-plugin/systemload.c	2013-05-07 15:32:36.552569345 +0200
@@ -185,7 +185,8 @@
         hours = (global->uptime->value_read / 3600) % 24;
         mins = (global->uptime->value_read / 60) % 60;
         if (days > 0) {
-            g_snprintf(caption, sizeof(caption), ngettext("%d day", "%d days", days), days);
+            g_snprintf(caption, sizeof(caption),
+                       ngettext("%d day %d:%02d", "%d days %d:%02d", days), days, hours, mins);
             gtk_label_set_text(GTK_LABEL(global->uptime->label),
                                caption);
             g_snprintf(caption, sizeof(caption),
diff -Naur orig/po/hu.po patched/po/hu.po
--- orig/po/hu.po	2012-05-16 20:53:25.000000000 +0200
+++ patched/po/hu.po	2013-05-07 16:48:11.946429520 +0200
@@ -46,10 +46,10 @@
 
 #: ../panel-plugin/systemload.c:190
 #, c-format
-msgid "%d day"
-msgid_plural "%d days"
-msgstr[0] "%d nap"
-msgstr[1] "%d nap"
+msgid "%d day %d:%02d"
+msgid_plural "%d days %d:%02d"
+msgstr[0] "%d nap %d.%02d"
+msgstr[1] "%d nap %d.%02d"
 
 #: ../panel-plugin/systemload.c:194
 #, c-format
@@ -119,11 +119,3 @@
 msgid "File /proc/uptime not found!"
 msgstr "A /proc/uptime fájl nem található"
 
-#~ msgid "Select color"
-#~ msgstr "Válasszon színt"
-
-#~ msgid "Show monitor"
-#~ msgstr "Figyelő megjelenítése"
-
-#~ msgid "Uptime:"
-#~ msgstr "Működési idő:"
