diff -Naur orig/mozilla/ipc/chromium/src/base/file_util.cc patched/mozilla/ipc/chromium/src/base/file_util.cc
--- orig/mozilla/ipc/chromium/src/base/file_util.cc	2012-11-20 03:46:00.000000000 +0100
+++ patched/mozilla/ipc/chromium/src/base/file_util.cc	2012-11-21 12:08:21.528937491 +0100
@@ -8,7 +8,7 @@
 #include <io.h>
 #endif
 #include <stdio.h>
-#if defined(ANDROID) || defined(OS_POSIX)
+#if defined(ANDROID) || defined(OS_POSIX) || defined(OS_LINUX)
 #include <unistd.h>
 #endif
 
diff -Naur orig/mozilla/ipc/chromium/src/base/message_pump_libevent.cc patched/mozilla/ipc/chromium/src/base/message_pump_libevent.cc
--- orig/mozilla/ipc/chromium/src/base/message_pump_libevent.cc	2012-11-20 03:46:00.000000000 +0100
+++ patched/mozilla/ipc/chromium/src/base/message_pump_libevent.cc	2012-11-21 12:07:18.812626506 +0100
@@ -6,7 +6,7 @@
 
 #include <errno.h>
 #include <fcntl.h>
-#if defined(ANDROID) || defined(OS_POSIX)
+#if defined(ANDROID) || defined(OS_POSIX) || defined(OS_LINUX)
 #include <unistd.h>
 #endif
 
diff -Naur orig/mozilla/ipc/chromium/src/base/time_posix.cc patched/mozilla/ipc/chromium/src/base/time_posix.cc
--- orig/mozilla/ipc/chromium/src/base/time_posix.cc	2012-11-20 03:46:00.000000000 +0100
+++ patched/mozilla/mozilla/ipc/chromium/src/base/time_posix.cc	2012-11-21 12:07:18.812626506 +0100
@@ -13,7 +13,7 @@
 #else
 #include <time.h>
 #endif
-#if defined(ANDROID) || defined(OS_POSIX)
+#if defined(ANDROID) || defined(OS_POSIX) || defined(OS_LINUX)
 #include <unistd.h>
 #endif
 
