diff -Naur orig/utils/Makefile.in mod/utils/Makefile.in
--- orig/utils/Makefile.in	2013-08-02 22:25:41.000000000 +0200
+++ mod/utils/Makefile.in	2013-12-07 13:52:12.544373518 +0100
@@ -77,8 +77,6 @@
 POST_UNINSTALL = :
 build_triplet = @build@
 host_triplet = @host@
-@LINUX_OS_TRUE@am__append_1 = \
-@LINUX_OS_TRUE@	xc3028-firmware
 
 @HAVE_LINUX_I2C_DEV_TRUE@@LINUX_OS_TRUE@am__append_2 = \
 @HAVE_LINUX_I2C_DEV_TRUE@@LINUX_OS_TRUE@	rds
@@ -154,7 +152,7 @@
 CTAGS = ctags
 DIST_SUBDIRS = dvb libv4l2util libmedia_dev decode_tm6000 keytable \
 	v4l2-compliance v4l2-ctl v4l2-dbg v4l2-sysfs-path rds-ctl \
-	xc3028-firmware rds qv4l2
+	rds qv4l2
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
 am__relativize = \
   dir0=`pwd`; \
