diff -Naur orig/compton-conf.desktop mod/compton-conf.desktop
--- orig/compton-conf.desktop	2014-05-07 07:08:13.000000000 +0200
+++ mod/compton-conf.desktop	2014-09-22 13:56:47.969882484 +0200
@@ -1,9 +1,12 @@
 [Desktop Entry]
 Type=Application
 Name=Window Effects
+Name[hu]=Ablakhatások
 GenericName=Compton Configuration
+GenericName[hu]=Compton beállítása
 Comment=Configure Compton window effects
+Comment[hu]=Compton ablakhatások beállítása
 TryExec=compton-conf
 Exec=compton-conf
 Icon=preferences-desktop-display
-Categories=Settings;DesktopSettings;Qt;LXQt;
+Categories=Settings;DesktopSettings;Qt;
