diff -Naur orig/build/linux/mog.desktop mod/build/linux/mog.desktop
--- orig/build/linux/mog.desktop	2008-03-04 18:41:30.000000000 +0100
+++ mod/build/linux/mog.desktop	2009-02-19 09:34:18.000000000 +0100
@@ -2,9 +2,11 @@
 Encoding=UTF-8
 Name=Maze of Galious
 Comment=Remake of a classic MSX platformer
+Comment[hu]=Klasszikus MSX platform játék felújítás
+GenericName[hu]=Galious útvesztője
 Exec=mog
 Icon=mog.png
 Terminal=false
 Type=Application
-Categories=Application;Game;ArcadeGame;
+Categories=Application;Game;ArcadeGame;X-UHU-Menu/Games/Action;
 StartupNotify=false
