diff -Naurd orig/Makefile.in mod/Makefile.in
--- orig/Makefile.in	2010-01-30 20:11:21.000000000 +0100
+++ mod/Makefile.in	2010-02-11 13:38:28.000000000 +0100
@@ -97,7 +97,7 @@
 	  else run_ldconfig=yes ; \
 	  fi ; \
 	  cd "$(DESTDIR)$(libdir)" && ln -s lib$(libname).so.$(pkgversion) lib$(libname).so.$(soversion) ; \
-	  if [ $${run_ldconfig} = yes ] ; then $(LDCONFIG) "$(DESTDIR)$(libdir)" ; fi ; \
+#	  if [ $${run_ldconfig} = yes ] ; then $(LDCONFIG) "$(DESTDIR)$(libdir)" ; fi ; \
 	fi
 
 install-info :
