--- gnome-control-center.orig/shell/gnomecc.desktop.in.in	2009-05-03 20:30:57.000000000 +0200
+++ gnome-control-center/shell/gnomecc.desktop.in.in	2010-01-24 14:00:30.000000000 +0100
@@ -1,15 +1,15 @@
 [Desktop Entry]
-_Name=Control Center
-_GenericName=The GNOME configuration tool
+Name=GNOME Control Center
+Name[hu]=GNOME Vezérlőközpont
+_Comment=The GNOME configuration tool
 Icon=gnome-control-center
 Exec=gnome-control-center
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;GTK;Settings;
+Categories=GTK;GNOME;Settings;X-UHU-Menu/Settings;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-control-center
 X-GNOME-Bugzilla-Component=shell
 X-GNOME-Bugzilla-Version=@VERSION@
-NoDisplay=true
