diff -Naur orig/pcmanfm/pcmanfm-qt.desktop.in mod/pcmanfm/pcmanfm-qt.desktop.in
--- orig/pcmanfm/pcmanfm-qt.desktop.in	2014-05-07 09:36:38.000000000 +0200
+++ mod/pcmanfm/pcmanfm-qt.desktop.in	2014-07-13 20:58:20.000000000 +0200
@@ -4,7 +4,7 @@
 Name=PCManFM File Manager
 GenericName=File Manager
 Comment=Browse the file system and manage the files
-Categories=FileManager;Utility;Core;Qt;
+Categories=FileManager;System;FileTools;Core;Qt;
 Exec=@CMAKE_INSTALL_PREFIX@/bin/pcmanfm-qt %U
 StartupNotify=true
 Terminal=false
