diff -Naur orig/pinball.desktop mod/pinball.desktop
--- orig/pinball.desktop	2003-12-10 09:44:58.000000000 +0100
+++ mod/pinball.desktop	2010-12-26 16:55:44.000000000 +0100
@@ -1,12 +1,11 @@
 [Desktop Entry]
 Encoding=UTF-8
-Categories=Application;Game;ArcadeGame;X-Red-Hat-Extra
-X-Desktop-File-Install-Version=0.2
+Categories=Application;Game;ArcadeGame;X-UHU-Menu/Games/Arcade;
 Name=Emilia Pinball
 Comment=OpenGL pinball game
+Comment[hu]=flipper játék
 Exec=pinball
-Icon=pinball.xpm
-Terminal=0
+Icon=/usr/share/pinball/pinball.xpm
 Type=Application
                                                                                                                                                              
                                                                                                                                                              
