diff -Naur e2fsprogs-1.41.9-orig/e2fsck/unix.c e2fsprogs-1.41.9/e2fsck/unix.c
--- e2fsprogs-1.41.9-orig/e2fsck/unix.c	2009-08-23 04:44:35.000000000 +0200
+++ e2fsprogs-1.41.9/e2fsck/unix.c	2009-09-04 12:58:29.000000000 +0200
@@ -1379,8 +1379,6 @@
 		    printf(_("\n%s: ***** FILE SYSTEM WAS MODIFIED *****\n"),
 			       ctx->device_name);
 		if (ctx->mount_flags & EXT2_MF_ISROOT) {
-			printf(_("%s: ***** REBOOT LINUX *****\n"),
-			       ctx->device_name);
 			exit_value |= FSCK_REBOOT;
 		}
 	}
