diff -urN pidgin-2.10.9.orig/pidgin.desktop.in pidgin-2.10.9/pidgin.desktop.in
--- pidgin-2.10.9.orig/pidgin.desktop.in	2014-02-02 22:30:51.000000000 +0100
+++ pidgin-2.10.9/pidgin.desktop.in	2014-02-03 17:37:30.787553218 +0100
@@ -1,5 +1,5 @@
 [Desktop Entry]
-_Name=Pidgin Internet Messenger
+_Name=Pidgin
 _GenericName=Internet Messenger
 _Comment=Chat over IM.  Supports AIM, Google Talk, Jabber/XMPP, MSN, Yahoo and more
 Exec=pidgin
@@ -7,5 +7,5 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=Network;InstantMessaging;
+Categories=GTK;Network;InstantMessaging;X-UHU-Menu/Internet/Messengers;
 
