diff -Naur orig/config/lxqt.conf mod/config/lxqt.conf
--- orig/config/lxqt.conf	2015-02-14 14:42:58.000000000 +0100
+++ mod/config/lxqt.conf	2015-05-03 17:23:15.651293533 +0200
@@ -1,5 +1,5 @@
 [General]
-theme=frost
+theme=Light
 icon_theme=oxygen
 single_click_activate=false
 tool_button_style=ToolButtonTextBesideIcon
diff -Naur orig/config/pcmanfm-qt/lxqt/settings.conf.in mod/config/pcmanfm-qt/lxqt/settings.conf.in
--- orig/config/pcmanfm-qt/lxqt/settings.conf.in	2015-02-14 14:42:58.000000000 +0100
+++ mod/config/pcmanfm-qt/lxqt/settings.conf.in	2015-05-03 17:53:40.258405542 +0200
@@ -15,7 +15,7 @@
 
 [Desktop]
 WallpaperMode=stretch
-Wallpaper=@LXQT_SHARE_DIR@/themes/light/simple_blue_widescreen.png
+Wallpaper=@LXQT_SHARE_DIR@/themes/Light/simple_blue_widescreen.png
 BgColor=#000000
 FgColor=#ffffff
 ShadowColor=#000000
