diff -Naur orig/po/LINGUAS patched/po/LINGUAS
--- orig/po/LINGUAS	2008-09-27 12:31:23.000000000 +0200
+++ patched/po/LINGUAS	2009-01-08 21:33:37.000000000 +0100
@@ -1,2 +1,2 @@
 # Set of available languages.
-da de el es fi fr it nl pl ru sv tr zh
+da de el es fi fr hu it nl pl ru sv tr zh
diff -Naur orig/po/Makefile patched/po/Makefile
--- orig/po/Makefile	2008-09-27 12:31:23.000000000 +0200
+++ patched/po/Makefile	2009-01-08 21:35:16.000000000 +0100
@@ -40,10 +40,10 @@
 MSGCONV = msgconv
 MSGFILTER = msgfilter
 
-POFILES =  da.po de.po el.po es.po fi.po fr.po it.po nl.po pl.po ru.po sv.po tr.po zh.po
-GMOFILES =  da.gmo de.gmo el.gmo es.gmo fi.gmo fr.gmo it.gmo nl.gmo pl.gmo ru.gmo sv.gmo tr.gmo zh.gmo
-UPDATEPOFILES =  da.po-update de.po-update el.po-update es.po-update fi.po-update fr.po-update it.po-update nl.po-update pl.po-update ru.po-update sv.po-update tr.po-update zh.po-update
-DUMMYPOFILES =  da.nop de.nop el.nop es.nop fi.nop fr.nop it.nop nl.nop pl.nop ru.nop sv.nop tr.nop zh.nop
+POFILES =  da.po de.po el.po es.po fi.po fr.po hu.po it.po nl.po pl.po ru.po sv.po tr.po zh.po
+GMOFILES =  da.gmo de.gmo el.gmo es.gmo fi.gmo fr.gmo hu.gmo it.gmo nl.gmo pl.gmo ru.gmo sv.gmo tr.gmo zh.gmo
+UPDATEPOFILES =  da.po-update de.po-update el.po-update es.po-update fi.po-update fr.po-update hu.po-update it.po-update nl.po-update pl.po-update ru.po-update sv.po-update tr.po-update zh.po-update
+DUMMYPOFILES =  da.nop de.nop el.nop es.nop fi.nop fr.nop hu.nop it.nop nl.nop pl.nop ru.nop sv.nop tr.nop zh.nop
 DISTFILES.common = Makefile.in.in remove-potcdate.sin \
 $(DISTFILES.common.extra1) $(DISTFILES.common.extra2) $(DISTFILES.common.extra3)
 DISTFILES = $(DISTFILES.common) Makevars POTFILES.in $(DOMAIN).pot stamp-po \
@@ -61,7 +61,7 @@
      ../src/progress.c \
      ../src/ftp-ls.c
 
-CATALOGS =  da.gmo de.gmo el.gmo es.gmo fi.gmo fr.gmo it.gmo nl.gmo pl.gmo ru.gmo sv.gmo tr.gmo zh.gmo
+CATALOGS =  da.gmo de.gmo el.gmo es.gmo fi.gmo fr.gmo hu.gmo it.gmo nl.gmo pl.gmo ru.gmo sv.gmo tr.gmo zh.gmo
 
 # Makevars gets inserted here. (Don't remove this line!)
 # Makefile variables for PO directory in any package using GNU gettext.
