diff -Naur orig/python-ui/hatariui.desktop mod/python-ui/hatariui.desktop
--- orig/python-ui/hatariui.desktop	2012-06-24 12:29:26.000000000 +0200
+++ mod/python-ui/hatariui.desktop	2014-05-05 16:27:49.349860403 +0200
@@ -1,7 +1,8 @@
 [Desktop Entry]
-Encoding=UTF-8
-Version=1.0
 Type=Application
 Name=Hatari UI
 Exec=hatariui
 Icon=hatari-icon
+Comment=Atari emulator
+Comment[hu]=Nagyszerű Atari emulátor
+Categories=Game;Emulator;
