diff -Naur a/src/apps/andromeda/andromeda.desktop b/src/apps/andromeda/andromeda.desktop
--- a/src/apps/andromeda/andromeda.desktop	2013-02-09 14:45:27.000000000 +0100
+++ b/src/apps/andromeda/andromeda.desktop	2013-06-08 21:17:01.233189016 +0200
@@ -25,4 +25,4 @@
 MimeType=inode/directory;
 StartupNotify=true
 Icon=andromeda
-Categories=System;FileManager;
+Categories=System;FileManager;X-UHU-Menu/Tools/File_Managers;
diff -Naur a/src/apps/imageviewer/imageviewer.desktop b/src/apps/imageviewer/imageviewer.desktop
--- a/src/apps/imageviewer/imageviewer.desktop	2013-02-09 14:45:27.000000000 +0100
+++ b/src/apps/imageviewer/imageviewer.desktop	2013-06-08 21:19:29.105189645 +0200
@@ -1,6 +1,8 @@
 [Desktop Entry]
 Name=ImageViewer
 GenericName=ImageViewer
+GenericName[hu]=Qt képnézegető
+Comment[hu]=Egyszerű képnézegető
 Comment[ru]=Просмотрщик изображеий
 Exec=imageviewer
 Terminal=false
@@ -8,4 +10,4 @@
 MimeType=image/bmp;image/gif;image/jpeg;video/x-mng;image/png;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/tiff;image/x-xbitmap;image/x-xpixmap;image/svg+xml;image/x-tga;
 StartupNotify=true
 Icon=imageviewer
-Categories=Graphics;RasterGraphics;Viewer;
+Categories=Graphics;RasterGraphics;Viewer;X-UHU-Menu/Graphics;
