--- cyrus-imapd-2.2.8/imap/squat_build.c.64bit-fixes	2003-06-15 18:14:13.000000000 -0400
+++ cyrus-imapd-2.2.8/imap/squat_build.c	2004-10-26 15:59:55.639973018 -0400
@@ -92,6 +92,7 @@
 #include <config.h>
 
 #include <stdlib.h>
+#include <stdint.h>
 #include <string.h>
 #include <unistd.h>
 #include <assert.h>
@@ -479,7 +480,7 @@ static void delete_doc_word_table(SquatW
     for (i = 0; i < VECTOR_SIZE(t->entries); i++) {
       SquatWordTableEntry* e = &(t->entries[i]);
       
-      if (e->leaf_presence != NULL && ((int)e->leaf_presence & 1) == 0) {
+      if (e->leaf_presence != NULL && ((intptr_t)e->leaf_presence & 1) == 0) {
         free(e->leaf_presence);
       }
     }
@@ -568,7 +569,7 @@ static int add_to_table(SquatIndex* inde
 
   if (word_entry == NULL) {
     /* We are in "per document" mode. */
-    if (((int)e->leaf_presence & 1) != 0) {
+    if (((intptr_t)e->leaf_presence & 1) != 0) {
       /* We currently have a singleton here. */
       int oldch = e->leaf_presence_singleton >> 1;
 
@@ -787,7 +788,7 @@ static int write_words(SquatIndex* index
 
       word[0] = (char)i;
 
-      if (((int)e->leaf_presence & 1) != 0) {
+      if (((intptr_t)e->leaf_presence & 1) != 0) {
 	/* Got a singleton at this branch point. Just output the single word. */
         word[1] = (char)(e->leaf_presence_singleton >> 1);
         e->leaf_presence = NULL; /* clear the leaf out */
--- cyrus-imapd-2.2.8/imap/saslserver.c.64bit-fixes	2004-07-16 08:25:14.000000000 -0400
+++ cyrus-imapd-2.2.8/imap/saslserver.c	2004-10-26 16:02:33.396795962 -0400
@@ -47,6 +47,7 @@
 #include <sasl/sasl.h>
 #include <sasl/saslutil.h>
 
+#include "xmalloc.h" /* xstrdup() */
 #include "prot.h"
 #include "imap_err.h"
 
