diff -Naur orig/src/data-types/mailstream_ssl.c patched/src/data-types/mailstream_ssl.c
--- orig/src/data-types/mailstream_ssl.c	2013-08-17 09:54:36.000000000 +0200
+++ patched/src/data-types/mailstream_ssl.c	2013-11-26 11:32:57.727656303 +0100
@@ -154,6 +154,7 @@
 #endif
 
 // Used to make OpenSSL thread safe
+#ifndef USE_GNUTLS
 #if defined (HAVE_PTHREAD_H) && !defined (WIN32) && defined (USE_SSL) && defined (LIBETPAN_REENTRANT)
   struct CRYPTO_dynlock_value
   {
@@ -224,6 +225,7 @@
     CRYPTO_set_dynlock_destroy_callback(dyn_destroy_function);
   }
 #endif
+#endif
 
 void mailstream_ssl_init_lock(void)
 {
