diff -Naur orig/src/data-types/mailstream_ssl.c patched/src/data-types/mailstream_ssl.c
--- orig/src/data-types/mailstream_ssl.c	2013-08-17 09:54:36.000000000 +0200
+++ patched/src/data-types/mailstream_ssl.c	2013-11-26 12:37:22.967331179 +0100
@@ -555,7 +555,9 @@
   else {
 		timeout_value = timeout;
   }
+#if GNUTLS_VERSION_NUMBER >= 0x030100
 	gnutls_handshake_set_timeout(session, timeout_value);
+#endif
 
   do {
     r = gnutls_handshake(session);
