diff -Naur gcalctool.orig/data/gcalctool.desktop.in gcalctool/data/gcalctool.desktop.in
--- gcalctool.orig/data/gcalctool.desktop.in	2008-09-21 08:40:19.000000000 +0200
+++ gcalctool/data/gcalctool.desktop.in	2008-10-17 18:03:59.000000000 +0200
@@ -6,7 +6,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;GTK;Utility;Calculator;
+Categories=GTK;GNOME;Utility;Calculator;X-UHU-Menu/Tools/Calculators;
 X-GNOME-DocPath=gcalctool/gcalctool.xml
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gcalctool
