diff -Naur orig/data/atril.desktop.in.in patched/data/atril.desktop.in.in
--- orig/data/atril.desktop.in.in	2013-10-11 22:41:25.000000000 +0200
+++ patched/data/atril.desktop.in.in	2014-02-10 23:02:39.960449711 +0100
@@ -13,5 +13,5 @@
 X-MATE-Bugzilla-Product=atril
 X-MATE-Bugzilla-Component=BugBuddyBugs
 X-MATE-Bugzilla-Version=@VERSION@
-Categories=GTK;Office;Viewer;
+Categories=GTK;Office;Viewer;X-UHU-Menu/Tools/Viewers;
 MimeType=@ATRIL_MIME_TYPES@
