diff -Naur a/qtfm.desktop b/qtfm.desktop
--- a/qtfm.desktop	2012-03-20 03:18:05.000000000 +0100
+++ b/qtfm.desktop	2013-06-08 20:56:00.085183653 +0200
@@ -9,6 +9,7 @@
 
 GenericName=Qt File Manager
 GenericName[es]=Qt Gestor de Archivos
+GenericName[hu]=Qt fájlkezelő
 GenericName[sr]=Кјут менаџер фајлова
 GenericName[sr@ijekavian]=Кјут менаџер фајлова
 GenericName[sr@latin]=qtFM menadžer fajlova
@@ -18,6 +19,7 @@
 Comment=A lightweight file manager based on pure Qt
 Comment[de]=Ein schlanker, Qt-basierter, Dateimanager
 Comment[es]=Un gestor de archivos ligero basado en Qt puro
+Comment[hu]=Egy könnyű Qt fájlkezelő
 Comment[it]=Un file manager leggero basato su puro Qt
 Comment[ru]=Легкий файловый менеджер на основе чистого Qt
 Comment[sr]=Лагани менаџер фајлова базиран на чистом КјуТ-у
@@ -31,4 +33,4 @@
 Exec=qtfm
 Terminal=false
 MimeType=application/x-directory;inode/directory
-Categories=Qt;Utility;FileManager
+Categories=Qt;Utility;FileManager;X-UHU-Menu/Tools/File_Managers;
