diff -Naur orig/mp3splt-gtk.desktop.in patched/mp3splt-gtk.desktop.in
--- orig/mp3splt-gtk.desktop.in	2013-05-02 23:01:15.000000000 +0200
+++ patched/mp3splt-gtk.desktop.in	2013-09-15 20:20:33.052589332 +0200
@@ -1,12 +1,13 @@
 [Desktop Entry]
 Name=Mp3splt-gtk audio splitter
+Name[hu]=mp3splt - mp3/ogg/flac vágó
 GenericName=Sound Splitter
 Comment=Split mp3 and ogg files without decoding
-
+Comment[hu]=mp3, ogg, flac fájlok vágása dekódolás nélkül
 Icon=PREFIX/mp3splt-gtk_ico.svg
 
 Type=Application
-Categories=AudioVideo;Audio;AudioVideoEditing;
+Categories=AudioVideo;Audio;AudioVideoEditing;X-UHU-Menu/Multimedia/SoundEdit;
 
 Exec=mp3splt-gtk
 StartupNotify=false
