diff -Naur aqsis-1.6.0-ori/distribution/linux/aqsis.desktop aqsis-1.6.0/distribution/linux/aqsis.desktop
--- aqsis-1.6.0-ori/distribution/linux/aqsis.desktop	2009-10-11 14:34:28.000000000 +0200
+++ aqsis-1.6.0/distribution/linux/aqsis.desktop	2011-04-10 20:47:55.384983949 +0200
@@ -1,13 +1,15 @@
 [Desktop Entry]
+Encoding=UTF-8
 Version=1.0
 Type=Application
 Name=Aqsis Renderer
 GenericName=RenderMan Renderer
 NoDisplay=true
 Comment=Render RenderMan Interface Bytestream (RIB) files
+Comment[hu]=Render RenderMan felület Bytestream (RIB) fájlok
 Icon=aqsis
 TryExec=aqsis
-Exec=/usr/share/aqsis/scripts/aqsis.sh %f
+Exec=aqsis
 Terminal=true
 MimeType=model/x-rib;model/x-rib-gzip;
-Categories=Graphics;3DGraphics;ConsoleOnly;
+Categories=X-UHU-Menu/Graphics/3DGraphics;
diff -Naur aqsis-1.6.0-ori/distribution/linux/aqsl.desktop aqsis-1.6.0/distribution/linux/aqsl.desktop
--- aqsis-1.6.0-ori/distribution/linux/aqsl.desktop	2009-10-11 14:34:28.000000000 +0200
+++ aqsis-1.6.0/distribution/linux/aqsl.desktop	2011-04-10 20:47:53.104984027 +0200
@@ -1,13 +1,17 @@
 [Desktop Entry]
+Encoding=UTF-8
 Version=1.0
 Type=Application
 Name=Aqsis Shader Compiler
+Name[hu]=Aqsis árnyékoló fordító
 GenericName=RenderMan Shader Compiler
+GenericName[hu]=RenderMan árnyékoló fordító
 NoDisplay=true
 Comment=Compile RenderMan Shading Language (RSL) files
+Comment[hu]=RenderMan árnyékoló fordítás Language (RSL) fájlok
 Icon=aqsis
 TryExec=aqsl
-Exec=/usr/share/aqsis/scripts/aqsl.sh %F
+Exec=aqsl
 Terminal=true
 MimeType=text/x-sl;
-Categories=Graphics;3DGraphics;ConsoleOnly;
+Categories=X-UHU-Menu/Graphics/3DGraphics;
diff -Naur aqsis-1.6.0-ori/distribution/linux/aqsltell.desktop aqsis-1.6.0/distribution/linux/aqsltell.desktop
--- aqsis-1.6.0-ori/distribution/linux/aqsltell.desktop	2009-10-11 14:34:28.000000000 +0200
+++ aqsis-1.6.0/distribution/linux/aqsltell.desktop	2011-04-10 20:47:50.512983823 +0200
@@ -1,13 +1,17 @@
 [Desktop Entry]
+Encoding=UTF-8
 Version=1.0
 Type=Application
 Name=Aqsis Shader Inspector
+Name[hu]=Aqsis árnyékoló ellenőrző
 GenericName=RenderMan Shader Inspector
+GenericName[hu]=RenderMan árnyékoló ellenőrző
 NoDisplay=true
 Comment=Inspect compiled RenderMan Shading Language (RSL) files
+Comment[hu]=Ellenőrzi a fordított RenderMan árnyékoló Language (RSL) fájlokat
 Icon=aqsis
 TryExec=aqsltell
-Exec=/usr/share/aqsis/scripts/aqsltell.sh %F
+Exec=aqsltell
 Terminal=true
 MimeType=application/x-slx;
-Categories=Graphics;3DGraphics;ConsoleOnly;
+Categories=X-UHU-Menu/Graphics/3DGraphics;
diff -Naur aqsis-1.6.0-ori/distribution/linux/eqsl.desktop aqsis-1.6.0/distribution/linux/eqsl.desktop
--- aqsis-1.6.0-ori/distribution/linux/eqsl.desktop	2009-10-11 14:34:28.000000000 +0200
+++ aqsis-1.6.0/distribution/linux/eqsl.desktop	2011-04-10 20:47:48.432984390 +0200
@@ -1,11 +1,15 @@
 [Desktop Entry]
+Encoding=UTF-8
 Version=1.0
 Type=Application
 Name=Aqsis User Interface
+Name[hu]=Aqsis felhasználói felület
 GenericName=RenderMan User Interface
+GenericName[hu]=RenderMan felhasználói felület
 Comment=Graphical user interface for Aqsis tools
+Comment[hu]=Grafikus felület Aqsis eszközökhöz
 Icon=aqsis
 TryExec=eqsl
 Exec=eqsl
 Terminal=false
-Categories=Graphics;3DGraphics;
+Categories=X-UHU-Menu/Graphics/3DGraphics;
diff -Naur aqsis-1.6.0-ori/distribution/linux/piqsl.desktop aqsis-1.6.0/distribution/linux/piqsl.desktop
--- aqsis-1.6.0-ori/distribution/linux/piqsl.desktop	2009-10-11 14:34:28.000000000 +0200
+++ aqsis-1.6.0/distribution/linux/piqsl.desktop	2011-04-10 20:47:45.848983969 +0200
@@ -1,13 +1,15 @@
 [Desktop Entry]
+Encoding=UTF-8
 Version=1.0
 Type=Application
 Name=Aqsis Advanced Framebuffer
 GenericName=RenderMan Advanced Framebuffer
 NoDisplay=true
 Comment=Display driver for Aqsis and other RenderMan-compliant renderers
+Comment[hu]=Kijelző meghajtó Aqsis és egyéb RenderMan-kompatibilis megjelenítők
 Icon=aqsis
 TryExec=piqsl
 Exec=piqsl %F
-Terminal=false
+Terminal=0
 MimeType=image/tiff;image/x-exr;
-Categories=Graphics;3DGraphics;Viewer;
+Categories=X-UHU-Menu/Graphics/3DGraphics;
