diff -Naur openexr-1.6.1-orig/exrenvmap/main.cpp openexr-1.6.1/exrenvmap/main.cpp
--- openexr-1.6.1-orig/exrenvmap/main.cpp	2007-04-25 03:07:51.000000000 +0200
+++ openexr-1.6.1/exrenvmap/main.cpp	2009-12-02 00:02:27.000000000 +0100
@@ -45,6 +45,7 @@
 #include <iostream>
 #include <exception>
 #include <stdlib.h>
+#include <cstring>
 
 using namespace Imf;
 using namespace std;
diff -Naur openexr-1.6.1-orig/exrmaketiled/main.cpp openexr-1.6.1/exrmaketiled/main.cpp
--- openexr-1.6.1-orig/exrmaketiled/main.cpp	2007-04-25 03:08:45.000000000 +0200
+++ openexr-1.6.1/exrmaketiled/main.cpp	2009-12-01 23:56:24.000000000 +0100
@@ -45,6 +45,7 @@
 #include <iostream>
 #include <exception>
 #include <string>
+#include <cstring>
 #include <stdlib.h>
 
 using namespace Imf;
