From 747082e87bdf9d0372a6b4ffae04ba14ea995f5a Mon Sep 17 00:00:00 2001
From: Cedric BAIL <cedric.bail@free.fr>
Date: Fri, 04 Jan 2013 00:55:00 +0000
Subject: autotools: move to AC_CONFIG_HEADERS.

Patch by Doug Newgard <scimmia22@outlook.com>.


SVN revision: 82135
---
diff --git a/configure.ac b/configure.ac
index 707bc1b..88d45e8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1052,7 +1052,7 @@ AC_SUBST(THREADLIBS)
 
 basedir=.
 
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 
 AC_OUTPUT([Makefile bg/Makefile doc/Makefile utils/Makefile pix/Makefile
           src/Makefile themes/Makefile])
--
cgit v0.9.2
