diff -Naur a/cantata.desktop b/cantata.desktop
--- a/cantata.desktop	2013-08-14 20:40:41.000000000 +0200
+++ b/cantata.desktop	2013-09-28 18:01:51.054840279 +0200
@@ -21,6 +21,6 @@
 GenericName[tr]=Muzik Çalıcı İstemcisi
 Icon=cantata
 Exec=cantata %U
-Categories=Qt;KDE;AudioVideo;Player;
+Categories=Qt;KDE;AudioVideo;Player;X-UHU-Menu/Multimedia/MusicPlayers;
 X-DBUS-StartupType=Unique
 Keywords=Music;MPD;
