diff -Naur orig/desktop-themes/GreenLaguna/index.theme patched/desktop-themes/GreenLaguna/index.theme
--- orig/desktop-themes/GreenLaguna/index.theme	2013-06-09 22:17:14.000000000 +0200
+++ patched/desktop-themes/GreenLaguna/index.theme	2013-06-18 13:04:12.508215522 +0200
@@ -5,7 +5,7 @@
 Encoding=UTF-8
 GtkTheme=GreenLaguna
 MetacityTheme=GreenLaguna
-IconTheme=menta
+IconTheme=Faenza-Mint
 CursorTheme=mate
 CursorSize=24
 
diff -Naur orig/desktop-themes/GreenLaguna/index.theme.in patched/desktop-themes/GreenLaguna/index.theme.in
--- orig/desktop-themes/GreenLaguna/index.theme.in	2013-06-09 22:16:36.000000000 +0200
+++ patched/desktop-themes/GreenLaguna/index.theme.in	2013-06-18 13:04:15.676244766 +0200
@@ -5,7 +5,7 @@
 Encoding=UTF-8
 GtkTheme=GreenLaguna
 MetacityTheme=GreenLaguna
-IconTheme=menta
+IconTheme=Faenza-Mint
 CursorTheme=mate
 CursorSize=24
 
