diff -Naur inkscape-0.48.2-orig/inkscape.desktop.in inkscape-0.48.2/inkscape.desktop.in
--- inkscape-0.48.2-orig/inkscape.desktop.in	2011-07-08 20:25:09.000000000 +0200
+++ inkscape-0.48.2/inkscape.desktop.in	2011-11-02 17:38:14.000000000 +0100
@@ -5,8 +5,8 @@
 _X-GNOME-FullName=Inkscape Vector Graphics Editor
 _Comment=Create and edit Scalable Vector Graphics images
 Type=Application
-Categories=Graphics;VectorGraphics;GTK;
-MimeType=image/svg+xml;image/svg+xml-compressed;application/vnd.corel-draw;application/pdf;application/postscript;image/x-eps;application/illustrator;
+Categories=GTK;Graphics;VectorGraphics;X-UHU-Menu/Graphics/Vectors;
+MimeType=image/svg+xml;image/svg+xml-compressed;image/cgm;image/x-wmf;application/vnd.corel-draw;application/x-xccx;application/x-xcgm;application/x-xcdt;application/x-xsk1;application/x-xcmx;image/x-xcdr;
 Exec=inkscape %F
 TryExec=inkscape
 Terminal=false
