diff -Naur orig/depend.mk mod/depend.mk
--- orig/depend.mk	1970-01-01 01:00:00.000000000 +0100
+++ mod/depend.mk	2008-03-06 13:08:04.000000000 +0100
@@ -0,0 +1,18 @@
+#
+# Dependencies.
+#
+
+src/pv/number.d src/pv/number.o: src/pv/number.c src/include/config.h src/include/library/gettext.h 
+src/pv/file.d src/pv/file.o: src/pv/file.c src/include/options.h src/include/config.h src/include/library/gettext.h 
+src/pv/display.d src/pv/display.o: src/pv/display.c src/include/config.h src/include/library/gettext.h src/include/options.h src/include/pv.h 
+src/pv/signal.d src/pv/signal.o: src/pv/signal.c src/include/pv.h src/include/config.h src/include/library/gettext.h 
+src/pv/loop.d src/pv/loop.o: src/pv/loop.c src/include/options.h src/include/pv.h src/include/config.h src/include/library/gettext.h 
+src/pv/cursor.d src/pv/cursor.o: src/pv/cursor.c src/include/options.h src/include/pv.h src/include/config.h src/include/library/gettext.h 
+src/pv/transfer.d src/pv/transfer.o: src/pv/transfer.c src/include/options.h src/include/config.h src/include/library/gettext.h 
+src/main/help.d src/main/help.o: src/main/help.c src/include/config.h src/include/library/gettext.h 
+src/main/options.d src/main/options.o: src/main/options.c src/include/config.h src/include/library/gettext.h src/include/options.h src/include/library/getopt.h src/include/pv.h 
+src/main/main.d src/main/main.o: src/main/main.c src/include/config.h src/include/library/gettext.h src/include/options.h src/include/pv.h 
+src/main/remote.d src/main/remote.o: src/main/remote.c src/include/config.h src/include/library/gettext.h src/include/options.h src/include/pv.h 
+src/main/version.d src/main/version.o: src/main/version.c src/include/config.h src/include/library/gettext.h 
+src/library/getopt.d src/library/getopt.o: src/library/getopt.c src/include/config.h src/include/library/gettext.h 
+src/library/gettext.d src/library/gettext.o: src/library/gettext.c src/include/config.h src/include/library/gettext.h 
diff -Naur orig/filelist.mk mod/filelist.mk
--- orig/filelist.mk	1970-01-01 01:00:00.000000000 +0100
+++ mod/filelist.mk	2008-03-06 13:08:01.000000000 +0100
@@ -0,0 +1,53 @@
+# Automatically generated file listings
+#
+# Creation time: Thu Mar  6 12:08:01 GMT 2008
+
+allsrc = src/pv/number.c \
+src/pv/file.c \
+src/pv/display.c \
+src/pv/signal.c \
+src/pv/loop.c \
+src/pv/cursor.c \
+src/pv/transfer.c \
+src/main/help.c \
+src/main/options.c \
+src/main/main.c \
+src/main/remote.c \
+src/main/version.c \
+src/library/getopt.c \
+src/library/gettext.c
+
+allobj = src/pv/number.o \
+src/pv/file.o \
+src/pv/display.o \
+src/pv/signal.o \
+src/pv/loop.o \
+src/pv/cursor.o \
+src/pv/transfer.o \
+src/main/help.o \
+src/main/options.o \
+src/main/main.o \
+src/main/remote.o \
+src/main/version.o \
+src/library/getopt.o \
+src/library/gettext.o \
+src/pv.o \
+src/nls.o \
+src/main.o \
+src/library.o
+
+alldep = src/pv/number.d \
+src/pv/file.d \
+src/pv/display.d \
+src/pv/signal.d \
+src/pv/loop.d \
+src/pv/cursor.d \
+src/pv/transfer.d \
+src/main/help.d \
+src/main/options.d \
+src/main/main.d \
+src/main/remote.d \
+src/main/version.d \
+src/library/getopt.d \
+src/library/gettext.d
+
diff -Naur orig/modules.mk mod/modules.mk
--- orig/modules.mk	1970-01-01 01:00:00.000000000 +0100
+++ mod/modules.mk	2008-03-06 13:08:01.000000000 +0100
@@ -0,0 +1,14 @@
+# Automatically generated module linking rules
+#
+# Creation time: Thu Mar  6 12:08:01 GMT 2008
+
+src/pv.o:  src/pv/cursor.o src/pv/display.o src/pv/file.o src/pv/loop.o src/pv/number.o src/pv/signal.o src/pv/transfer.o
+	$(LD) $(LDFLAGS) -o $@  src/pv/cursor.o src/pv/display.o src/pv/file.o src/pv/loop.o src/pv/number.o src/pv/signal.o src/pv/transfer.o
+
+src/main.o:  src/main/help.o src/main/main.o src/main/options.o src/main/remote.o src/main/version.o
+	$(LD) $(LDFLAGS) -o $@  src/main/help.o src/main/main.o src/main/options.o src/main/remote.o src/main/version.o
+
+src/library.o:  src/library/getopt.o src/library/gettext.o
+	$(LD) $(LDFLAGS) -o $@  src/library/getopt.o src/library/gettext.o
+
+
