diff -Naur orig/src/hardware.cpp mod/src/hardware.cpp
--- orig/src/hardware.cpp	2009-04-10 13:11:28.000000000 +0200
+++ mod/src/hardware.cpp	2010-10-29 22:41:33.000000000 +0200
@@ -262,7 +262,7 @@
        return ContLang - 1;
 
     LogWrite ("Cargando fondo de Selección de lenguaje...");
-    Objeto = load_datafile_object ("./data/bitmaps1.dat", "DFBM_BMLANG");
+    Objeto = load_datafile_object ("/usr/share/games/capitan/data/bitmaps1.dat", "DFBM_BMLANG");
     if (Objeto) {
        draw_sprite (screen, (BITMAP*)Objeto->dat, (screen->w - 640) / 2, (screen->h - 480) / 2);
        unload_datafile_object (Objeto);
@@ -314,7 +314,7 @@
 int Hardware::LoadDF1()
 {
     LogWrite ("Cargando archivo de datos LOADING.DAT: ");
-    DataP = load_datafile ("./data/loading.dat");
+    DataP = load_datafile ("/usr/share/games/capitan/data/loading.dat");
     if (!DataP) {
        allegro_message ("                   ERROR DE E/S\n\n No se encuentra el archivo LOADING.DAT");
        set_gfx_mode(GFX_TEXT,0,0,0,0);
@@ -324,7 +324,7 @@
     LogWrite ("OK.\n");
 
     LogWrite ("Cargando archivo de datos SPRITES.DAT: ");
-    SpritesBM = load_datafile("./data/sprites.dat");
+    SpritesBM = load_datafile("/usr/share/games/capitan/data/sprites.dat");
     if (!SpritesBM) {
       allegro_message ("                   ERROR DE E/S\n\n No se encuentra el archivo SPRITES.DAT");
       set_gfx_mode(GFX_TEXT,0,0,0,0);
@@ -345,7 +345,7 @@
     VuelcaBuffer();
 
     LogWrite ("Cargando archivo de datos SCREENS.DAT: ");
-    ScreensBM = load_datafile_callback ("./data/screens.dat", IndicadorCarga);
+    ScreensBM = load_datafile_callback ("/usr/share/games/capitan/data/screens.dat", IndicadorCarga);
     if (!ScreensBM) {
        allegro_message ("                   ERROR DE E/S\n\n No se encuentra el archivo SCREENS.DAT");
        set_gfx_mode(GFX_TEXT,0,0,0,0);
@@ -355,7 +355,7 @@
     LogWrite ("OK.\n");
 
     LogWrite ("Cargando archivo de datos BITMAPS0.DAT: ");
-    BitMapsBM = load_datafile_callback ("./data/bitmaps0.dat", IndicadorCarga);
+    BitMapsBM = load_datafile_callback ("/usr/share/games/capitan/data/bitmaps0.dat", IndicadorCarga);
     if (!BitMapsBM) {
        allegro_message ("                   ERROR DE E/S\n\n No se encuentra el archivo BITMAPS0.DAT");
        set_gfx_mode(GFX_TEXT,0,0,0,0);
@@ -365,7 +365,7 @@
     LogWrite ("OK.\n");
 
     LogWrite ("Cargando archivo de datos SFX.DAT: ");
-    SFx = load_datafile_callback ("./data/sfx.dat", IndicadorCarga);
+    SFx = load_datafile_callback ("/usr/share/games/capitan/data/sfx.dat", IndicadorCarga);
     if (!BitMapsBM) {
        allegro_message ("                   ERROR DE E/S\n\n No se encuentra el archivo SFX.DAT");
        set_gfx_mode(GFX_TEXT,0,0,0,0);
@@ -380,7 +380,7 @@
 int Hardware::LoadDF3 ()
 {
     LogWrite ("Cargando archivo de datos SCREENS.DAT: ");
-    ScreensBM = load_datafile ("./data/screens.dat");
+    ScreensBM = load_datafile ("/usr/share/games/capitan/data/screens.dat");
     if (!ScreensBM) {
        allegro_message ("                   ERROR DE E/S\n\n No se encuentra el archivo SCREENS.DAT");
        set_gfx_mode(GFX_TEXT,0,0,0,0);
@@ -390,7 +390,7 @@
     LogWrite ("OK.\n");
 
     LogWrite ("Cargando archivo de datos BITMAPS0.DAT: ");
-    BitMapsBM = load_datafile ("./data/bitmaps0.dat");
+    BitMapsBM = load_datafile ("/usr/share/games/capitan/data/bitmaps0.dat");
     if (!BitMapsBM) {
        allegro_message ("                   ERROR DE E/S\n\n No se encuentra el archivo BITMAPS0.DAT");
        set_gfx_mode(GFX_TEXT,0,0,0,0);
@@ -572,7 +572,7 @@
 
      DelFont_af();
      LogWrite ("Cargando fuente TTF (%s)..", s1);
-     PreFuente = load_datafile_object ("./data/fonts.dat", s1);
+     PreFuente = load_datafile_object ("/usr/share/games/capitan/data/fonts.dat", s1);
      LogWrite (".");
 
      for (int n = 0; n < 20; n++) {
@@ -612,7 +612,7 @@
      PALETTE pal;
      DelFont();
      char *names[] = {(char*) s1,(char*) s2};
-     MyFont = load_dat_font("./data/fonts.dat", pal, names);
+     MyFont = load_dat_font("/usr/share/games/capitan/data/fonts.dat", pal, names);
      set_palette (pal);
      font = MyFont;
 }
@@ -770,7 +770,7 @@
      LogWrite ("Cargando %s en %s...", Obj, File);
      DATAFILE *Objeto;
      char Fichero[255];
-     sprintf (Fichero, "./data/%s.dat", File);
+     sprintf (Fichero, "/usr/share/games/capitan/data/%s.dat", File);
      Objeto = load_datafile_object (Fichero, Obj);
      if (Objeto) {
         Dibuja (x, y, (void*)Objeto->dat, BitMap);
@@ -784,7 +784,7 @@
      LogWrite ("Cargando %s en %s...", Obj, File);
      DATAFILE *Objeto;
      char Fichero[255];
-     sprintf (Fichero, "./data/%s.dat", File);
+     sprintf (Fichero, "/usr/share/games/capitan/data/%s.dat", File);
      Objeto = load_datafile_object (Fichero, Obj);
      if (Objeto) {
         Dibuja2x (x, y, (void*)Objeto->dat);
@@ -798,7 +798,7 @@
      LogWrite ("Cargando %s en %s...", Obj, File);
      DATAFILE *Objeto;
      char Fichero[255];
-     sprintf (Fichero, "./data/%s.dat", File);
+     sprintf (Fichero, "/usr/share/games/capitan/data/%s.dat", File);
      Objeto = load_datafile_object (Fichero, Obj);
      if (Objeto) {
         DibujaCentr ((void*)Objeto->dat,x,y);
@@ -812,7 +812,7 @@
      LogWrite ("Cargando %s en %s...", Obj, File);
      DATAFILE *Objeto;
      char Fichero[255];
-     sprintf (Fichero, "./data/%s.dat", File);
+     sprintf (Fichero, "/usr/share/games/capitan/data/%s.dat", File);
      Objeto = load_datafile_object (Fichero, Obj);
      if (Objeto) {
         DibujaNegro (x - w, y, (void*)Objeto->dat);
@@ -865,7 +865,7 @@
 {
      DATAFILE *Objeto;
      char Fichero[255];
-     sprintf (Fichero, "./data/%s.dat", File);
+     sprintf (Fichero, "/usr/share/games/capitan/data/%s.dat", File);
      Objeto = load_datafile_object (Fichero, Obj);
      if (Objeto) {
         Dibuja_Trans (x, y, (void*)Objeto->dat, Trans, Force);
@@ -904,7 +904,7 @@
 {
      DATAFILE *Objeto;
      char Fichero[255];
-     sprintf (Fichero, "./data/%s.dat", File);
+     sprintf (Fichero, "/usr/share/games/capitan/data/%s.dat", File);
      Objeto = load_datafile_object (Fichero, Obj);
      if (Objeto) {
         Dibuja_TransAdd (x, y, (void*)Objeto->dat, Trans, Force);
@@ -1704,7 +1704,7 @@
 
 int Hardware::MapLoadInit()
 {
-    MapaDat = load_datafile("./data/map.dat");
+    MapaDat = load_datafile("/usr/share/games/capitan/data/map.dat");
     if (!MapaDat) {
        allegro_message ("                   ERROR DE E/S\n\n No se encuentra el archivo MAP.DAT");
        set_gfx_mode(GFX_TEXT,0,0,0,0);
diff -Naur orig/src/messages.cpp mod/src/messages.cpp
--- orig/src/messages.cpp	2009-04-10 12:57:23.000000000 +0200
+++ mod/src/messages.cpp	2010-10-29 22:42:08.000000000 +0200
@@ -21,7 +21,7 @@
     if (!MaxMens)
 	   return 0;
 
-    sprintf (FileName, "./lang/lang%i.txt", Lang);
+    sprintf (FileName, "/usr/share/games/capitan/lang/lang%i.txt", Lang);
     LogWrite ("Intentando cargar mensajes de %s...", FileName);
     Source.open (FileName);
     if (Source.fail()) {
diff -Naur orig/src/presentacion.cpp mod/src/presentacion.cpp
--- orig/src/presentacion.cpp	2009-04-10 12:57:23.000000000 +0200
+++ mod/src/presentacion.cpp	2010-10-29 22:42:47.000000000 +0200
@@ -40,17 +40,17 @@
            strcat (Temp, "0");
         sprintf (Titulo, "%s%i", Temp, n);
         LogWrite ("Cargando %s...", Titulo);
-        FrameLogo[n] = load_datafile_object ("./data/bitmaps1.dat",Titulo);
+        FrameLogo[n] = load_datafile_object ("/usr/share/games/capitan/data/bitmaps1.dat",Titulo);
         LogWrite ("OK.\n");
         if (!FrameLogo[n])
            return 1;
         }
     LogWrite ("Cargando DFBM_CEZT_CS...");
-    CuboSolo = load_datafile_object ("./data/bitmaps1.dat","DFBM_CEZT_CS");
+    CuboSolo = load_datafile_object ("/usr/share/games/capitan/data/bitmaps1.dat","DFBM_CEZT_CS");
     LogWrite ("OK.\nCargando DFBM_CEZT_HC...");
-    HaloCubo = load_datafile_object ("./data/bitmaps1.dat","DFBM_CEZT_HC");
+    HaloCubo = load_datafile_object ("/usr/share/games/capitan/data/bitmaps1.dat","DFBM_CEZT_HC");
     LogWrite ("OK.\nCargando DFBM_CEZT_TUB...");
-    Tuberia = load_datafile_object ("./data/bitmaps1.dat","DFBM_CEZT_TUB");
+    Tuberia = load_datafile_object ("/usr/share/games/capitan/data/bitmaps1.dat","DFBM_CEZT_TUB");
     LogWrite ("OK.\n");
 
     if (!HaloCubo || !CuboSolo)
@@ -145,7 +145,7 @@
   DATAFILE *Logo;
 
   LogWrite ("Cargando DFBM_CEZRD...");
-  Logo = load_datafile_object ("./data/bitmaps1.dat","DFBM_CEZRD");
+  Logo = load_datafile_object ("/usr/share/games/capitan/data/bitmaps1.dat","DFBM_CEZRD");
   LogWrite ("OK.\n");
 
   Velar = 255;
@@ -263,7 +263,7 @@
       if (n < 10)
         sprintf (Titulo, "DFBM_INTRO_00%i", n);
       LogWrite ("Cargando %s...", Titulo);
-      Logo = load_datafile_object ("./data/bitmaps1.dat",Titulo);
+      Logo = load_datafile_object ("/usr/share/games/capitan/data/bitmaps1.dat",Titulo);
       if (Logo)
         LogWrite ("OK.\n");
       else
@@ -289,7 +289,7 @@
             case 4:
 /*                if (!Suena[DF_SEXPL]) {
                     LogWrite ("Sample explosión.\n");
-                    Logo = load_datafile_object ("./data/sfx.dat", "DF_SEXPL");
+                    Logo = load_datafile_object ("/usr/share/games/capitan/data/sfx.dat", "DF_SEXPL");
                     play_sample ((SAMPLE*)Logo->dat, Config->VerVolMus(), 127, 1000, 0);
                     }
                 Suena[DF_SEXPL] = true;*/
@@ -432,7 +432,7 @@
       sprintf (Titulo, "DFBM_ENDCOMIC_00%i", n);
 
       LogWrite ("Cargando %s...", Titulo);
-      Logo = load_datafile_object ("./data/bitmaps1.dat",Titulo);
+      Logo = load_datafile_object ("/usr/share/games/capitan/data/bitmaps1.dat",Titulo);
       LogWrite ("OK.\n");
 
       Ciclos = 0;
@@ -544,7 +544,7 @@
     H->cls (0);
     H->CopyFondo();
 
-    Logo = load_datafile_object ("./data/bitmaps1.dat", "DFBM_PERIODICO");
+    Logo = load_datafile_object ("/usr/share/games/capitan/data/bitmaps1.dat", "DFBM_PERIODICO");
 
     y = 50;
 
@@ -569,7 +569,7 @@
 
     GiraPeriodico (Logo->dat);
 
-    Text = load_datafile_object ("./data/bitmaps1.dat", "DFBM_PERTXT");
+    Text = load_datafile_object ("/usr/share/games/capitan/data/bitmaps1.dat", "DFBM_PERTXT");
 
     for (int n = 0; n < 9; n ++) {
 
@@ -581,7 +581,7 @@
 
         unload_datafile_object (Logo);
         sprintf (Titulo, "DFBM_PERIOD_00%i", (int) (rand() % 3));
-        Logo = load_datafile_object ("./data/bitmaps1.dat", Titulo);
+        Logo = load_datafile_object ("/usr/share/games/capitan/data/bitmaps1.dat", Titulo);
         y = 50;
 
         switch (n) {
diff -Naur orig/src/sonido.cpp mod/src/sonido.cpp
--- orig/src/sonido.cpp	2009-04-10 12:57:23.000000000 +0200
+++ mod/src/sonido.cpp	2010-10-29 22:43:51.000000000 +0200
@@ -72,7 +72,7 @@
        alogg_destroy_ogg (ActFx);
        }
 
-    dfMusic = load_datafile_object("./data/music.dat", Nombre);
+    dfMusic = load_datafile_object("/usr/share/games/capitan/data/music.dat", Nombre);
 
     if (!dfMusic) {
        LogWrite ("ERROR: No se ha podido cargar fichero de música: %s\n", Nombre);
