diff -Naur orig/resources/lxqt-about.desktop.in mod/resources/lxqt-about.desktop.in
--- orig/resources/lxqt-about.desktop.in	2014-06-01 06:56:02.000000000 +0200
+++ mod/resources/lxqt-about.desktop.in	2014-12-02 10:14:33.539861263 +0100
@@ -5,7 +5,7 @@
 Comment=Learn more about LXQt
 Exec=lxqt-about
 Icon=help-about
-Categories=Help;LXQt
+Categories=Documentation;LXQt
 OnlyShowIn=LXQt;LXDE;
 
 #TRANSLATIONS_DIR=translations
