diff -Naur ORIG/builds/posix/prefix.linux UHU/builds/posix/prefix.linux
--- ORIG/builds/posix/prefix.linux	2007-05-03 13:24:50.000000000 +0200
+++ UHU/builds/posix/prefix.linux	2009-05-26 08:37:51.000000000 +0200
@@ -22,10 +22,9 @@
 # you may remove it if engine is getting compiled with any other GCC version
 
 COMMON_FLAGS=-ggdb -DFB_SEND_FLAGS=MSG_NOSIGNAL -DLINUX -pipe -MMD -fPIC -fmessage-length=0
-OPTIMIZE_FLAGS=-O3 -march=i586 -mcpu=i686 -fno-omit-frame-pointer -fno-builtin
 WARN_FLAGS=-Wall -Wno-switch -Wno-parentheses -Wno-unknown-pragmas -Wno-unused-variable
 
-PROD_FLAGS=-DNDEBUG $(COMMON_FLAGS) $(OPTIMIZE_FLAGS)
+PROD_FLAGS=-DNDEBUG $(COMMON_FLAGS)
 DEV_FLAGS=-DUSE_VALGRIND -p $(COMMON_FLAGS) $(WARN_FLAGS)
 #DEV_FLAGS=-p $(COMMON_FLAGS) $(WARN_FLAGS)
 
