diff -Naur ORIG/xfprint/xfprint.desktop.in UHU/xfprint/xfprint.desktop.in
--- ORIG/xfprint/xfprint.desktop.in	2008-06-27 23:18:03.000000000 +0200
+++ UHU/xfprint/xfprint.desktop.in	2009-02-04 16:32:24.000000000 +0100
@@ -3,9 +3,9 @@
 _Name=Xfce 4 Print Dialog
 _Comment=Print a file and allow you to set up its layout
 Exec=xfprint4
-Icon=printer
+Icon=xfprinter
 Terminal=false
 Type=Application
-Categories=X-XFCE;Utility;
+Categories=X-XFCE;Utility;X-UHU-Menu/Tools/Utilities;
 OnlyShowIn=XFCE;
 _GenericName=Print Dialog
diff -Naur ORIG/xfprint-manager/xfprint-manager.desktop.in UHU/xfprint-manager/xfprint-manager.desktop.in
--- ORIG/xfprint-manager/xfprint-manager.desktop.in	2008-06-27 23:18:03.000000000 +0200
+++ UHU/xfprint-manager/xfprint-manager.desktop.in	2009-02-04 16:32:52.000000000 +0100
@@ -3,9 +3,9 @@
 _Name=Xfce 4 Print Manager
 _Comment=Show the printer list and allow you to manage their jobs
 Exec=xfprint4-manager
-Icon=printer
+Icon=fxprinter
 Terminal=false
 Type=Application
-Categories=X-XFCE;Utility;
+Categories=X-XFCE;Utility;X-UHU-Menu/Tools/Utilities;
 OnlyShowIn=XFCE;
 _GenericName=Print Manager
