diff -Naur orig/prnt/hpijs/dj9xxvip.cpp patched/prnt/hpijs/dj9xxvip.cpp
--- orig/prnt/hpijs/dj9xxvip.cpp	2014-03-06 06:11:19.000000000 +0100
+++ patched/prnt/hpijs/dj9xxvip.cpp	2014-03-22 18:14:24.249473463 +0100
@@ -2239,7 +2239,7 @@
             {
                 case '5':
                 {
-                    if ((*(pStr+1) & 0xf3) > 1)
+                    if ((*(pStr+1) & 0xf3) > '1')
                     {
                         if (err != NO_ERROR)
                         {
@@ -2254,7 +2254,7 @@
                 }
                 case '6':
                 {
-                    if ((*(pStr+1) & 0xf3) > 1)
+                    if ((*(pStr+1) & 0xf3) > '1')
                     {
                         if (err != NO_ERROR)
                         {
@@ -2269,7 +2269,7 @@
                 }
                 case '7':
                 {
-                    if ((*(pStr+1) & 0xf3) > 1)
+                    if ((*(pStr+1) & 0xf3) > '1')
                     {
                         if (err != NO_ERROR)
                         {
@@ -2285,7 +2285,7 @@
                 }
                 case '8':
                 {
-                    if ((*(pStr+1) & 0xf3) > 1)
+                    if ((*(pStr+1) & 0xf3) > '1')
                     {
                         if (err != NO_ERROR)
                         {
