diff -Naur orig/include/common.h patched/include/common.h
--- orig/include/common.h	2013-07-27 18:24:52.000000000 +0200
+++ patched/include/common.h	2013-12-03 16:28:51.121681445 +0100
@@ -42,15 +42,6 @@
 #include <convert_to_biu.h>
 #include <colors.h>
 
-#if !wxUSE_PRINTING_ARCHITECTURE && !SWIG
-#   error "You must use '--enable-printarch' in your wx library configuration."
-#endif
-
-#if defined( __WXGTK__ )
-#   if !wxUSE_LIBGNOMEPRINT && !wxUSE_GTKPRINT && !SWIG
-#       error "You must use '--with-gnomeprint' or '--with-gtkprint' in your wx library configuration."
-#   endif
-#endif
 
 
 class wxAboutDialogInfo;
