diff -Naurdp a/data/bug-buddy.desktop.in.in b/data/bug-buddy.desktop.in.in
--- a/data/bug-buddy.desktop.in.in	2008-08-31 12:59:45.000000000 +0200
+++ b/data/bug-buddy.desktop.in.in	2008-09-11 00:52:31.000000000 +0200
@@ -1,11 +1,11 @@
 [Desktop Entry]
-_Name=Bug Report Tool
+_Name=Bug Buddy
 _Comment=Report a bug in GNOME-based applications
 Exec=bug-buddy
 Icon=bug-buddy
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Development;
+Categories=GTK;GNOME;Development;X-UHU-Menu/Tools/More;
 StartupNotify=true
 OnlyShowIn=GNOME;
 NoDisplay=true
