diff -Naur orig/config/lxqt-config-powermanagement.desktop.in mod/config/lxqt-config-powermanagement.desktop.in
--- orig/config/lxqt-config-powermanagement.desktop.in	2014-05-01 04:04:14.000000000 +0200
+++ mod/config/lxqt-config-powermanagement.desktop.in	2014-07-12 16:30:02.584386757 +0200
@@ -4,7 +4,6 @@
 GenericName=Power Management Settings
 Exec=lxqt-config-powermanagement
 Icon=preferences-system-power-management
-Categories=Settings;DesktopSettings;Qt;LXQt;
-OnlyShowIn=LXQt;LXDE;
+Categories=Settings;DesktopSettings;Qt;
 
 #TRANSLATIONS_DIR=translations
