diff -Naur orig/src/lid.cpp mod/src/lid.cpp
--- orig/src/lid.cpp	2014-05-01 04:04:14.000000000 +0200
+++ mod/src/lid.cpp	2014-07-12 16:24:35.896972043 +0200
@@ -27,7 +27,7 @@
 #include "lid.h"
 #include <QDBusConnection>
 #include <QDBusReply>
-#include <qt4/QtCore/qobject.h>
+#include "/usr/lib/qt4/include/QtCore/qobject.h"
 #include <QDebug>
 
 Lid::Lid()
diff -Naur orig/src/lidwatcher.cpp mod/src/lidwatcher.cpp
--- orig/src/lidwatcher.cpp	2014-05-01 04:04:14.000000000 +0200
+++ mod/src/lidwatcher.cpp	2014-07-12 16:27:30.244659686 +0200
@@ -27,8 +27,8 @@
 #include <QTimer>
 #include <QCoreApplication>
 #include <lxqt/lxqtautostartentry.h>
-#include <qt4/QtCore/qprocess.h>
-#include <qt4/QtCore/qtextstream.h>
+#include "/usr/lib/qt4/include/QtCore/qprocess.h"
+#include "/usr/lib/qt4/include/QtCore/qtextstream.h"
 #include <QDebug>
 #include "lidwatcher.h"
 #include "../config/powermanagementsettings.h"
