diff -Naur orig/lib/dpkg/parsehelp.c patched/lib/dpkg/parsehelp.c
--- orig/lib/dpkg/parsehelp.c	2014-06-04 01:07:31.000000000 +0200
+++ patched/lib/dpkg/parsehelp.c	2014-06-08 18:10:45.241470389 +0200
@@ -210,11 +210,11 @@
   if (*ptr && !cisdigit(*ptr++))
     return dpkg_put_warn(err, _("version number does not start with digit"));
   for (; *ptr; ptr++) {
-    if (!cisdigit(*ptr) && !cisalpha(*ptr) && strchr(".-+~:", *ptr) == NULL)
+    if (!cisdigit(*ptr) && !cisalpha(*ptr) && strchr(".-+~:_", *ptr) == NULL)
       return dpkg_put_warn(err, _("invalid character in version number"));
   }
   for (ptr = rversion->revision; *ptr; ptr++) {
-    if (!cisdigit(*ptr) && !cisalpha(*ptr) && strchr(".+~", *ptr) == NULL)
+    if (!cisdigit(*ptr) && !cisalpha(*ptr) && strchr(".+~_", *ptr) == NULL)
       return dpkg_put_warn(err, _("invalid character in revision number"));
   }
 
diff -Naur orig/lib/dpkg/test/t-version.c patched/lib/dpkg/test/t-version.c
--- orig/lib/dpkg/test/t-version.c	2014-06-04 02:02:55.000000000 +0200
+++ patched/lib/dpkg/test/t-version.c	2014-06-08 18:10:45.241470389 +0200
@@ -211,8 +211,8 @@
 	test_pass(dpkg_version_compare(&a, &b) == 0);
 
 	/* Test valid characters in upstream version. */
-	b = DPKG_VERSION_OBJECT(0, "09azAZ.-+~:", "0");
-	test_pass(parseversion(&a, "0:09azAZ.-+~:-0", NULL) == 0);
+	b = DPKG_VERSION_OBJECT(0, "09azAZ.-+~:_", "0");
+	test_pass(parseversion(&a, "0:09azAZ.-+~:_-0", NULL) == 0);
 	test_pass(dpkg_version_compare(&a, &b) == 0);
 
 	/* Test valid characters in revision. */
@@ -263,7 +263,7 @@
 
 	/* Test invalid characters in upstream version. */
 	verstr = test_alloc(strdup("0:0a-0"));
-	for (p = "!#@$%&/|\\<>()[]{};,_=*^'"; *p; p++) {
+	for (p = "!#@$%&/|\\<>()[]{};,=*^'"; *p; p++) {
 		verstr[3] = *p;
 		test_fail(parseversion(&a, verstr, &err) == 0);
 		test_warn(err);
