diff -Naur orig/lib/dpkg/dpkg.h patched/lib/dpkg/dpkg.h
--- orig/lib/dpkg/dpkg.h	2013-12-07 07:51:18.000000000 +0100
+++ patched/lib/dpkg/dpkg.h	2013-12-08 20:39:01.553506119 +0100
@@ -73,6 +73,7 @@
 #define POSTINSTFILE       "postinst"
 #define PRERMFILE          "prerm"
 #define POSTRMFILE         "postrm"
+#define POSTRMCUSTOMFILE   "postrm-custom"
 #define TRIGGERSCIFILE     "triggers"
 
 #define STATUSFILE        "status"
diff -Naur orig/src/remove.c patched/src/remove.c
--- orig/src/remove.c	2013-12-07 07:51:18.000000000 +0100
+++ patched/src/remove.c	2013-12-08 20:39:01.553506119 +0100
@@ -206,7 +206,8 @@
 {
   /* We need the postrm and list files for --purge. */
   if (strcmp(filetype, LISTFILE) == 0 ||
-      strcmp(filetype, POSTRMFILE) == 0)
+      strcmp(filetype, POSTRMFILE) == 0 ||
+      strcmp(filetype, POSTRMCUSTOMFILE) == 0)
     return;
 
   debug(dbg_stupidlyverbose, "removal_bulk info not postrm or list");
@@ -638,6 +639,12 @@
     if (unlink(filename) && errno != ENOENT)
       ohshite(_("can't remove old postrm script"));
 
+   filename = pkg_infodb_get_file(pkg, &pkg->installed, POSTRMCUSTOMFILE);
+    debug(dbg_general, "removal_bulk purge done, removing postrm-custom `%s'",
+          filename);
+    if (unlink(filename) && errno != ENOENT)
+      ohshite(_("can't remove old postrm-custom script"));
+    
     pkg_set_status(pkg, stat_notinstalled);
     pkg_set_want(pkg, want_unknown);
 
