diff -Naur orig/dselect/mkcurkeys.pl patched/dselect/mkcurkeys.pl
--- orig/dselect/mkcurkeys.pl	2014-05-30 18:30:50.000000000 +0200
+++ patched/dselect/mkcurkeys.pl	2014-04-29 14:24:40.000000000 +0200
@@ -53,8 +53,8 @@
 open(my $header_fh, '<', $ARGV[1]) or die $!;
 while (<$header_fh>) {
     s/\s+$//;
-    m/#define KEY_(\w+)\s+\d+\s+/p || next;
-    my $rhs = ${^POSTMATCH};
+    m/#define KEY_(\w+)\s+\d+\s+/ || next;
+    my $rhs = $';
     $k= "KEY_$1";
     $_= $1;
     capit();
@@ -122,12 +122,12 @@
     my $o = '';
     y/A-Z/a-z/;
     $_ = " $_";
-    while (m/ (\w)/p) {
-        $o .= ${^PREMATCH} . ' ';
+    while (m/ (\w)/) {
+        $o .= $`.' ';
         $_ = $1;
         y/a-z/A-Z/;
         $o .= $_;
-        $_ = ${^POSTMATCH};
+        $_ = $';
     }
     $_= $o.$_; s/^ //;
 }
@@ -135,6 +135,6 @@
 sub p {
     my ($k, $v) = @_;
 
-    $v =~ s/["\\]/\\${^MATCH}/pg;
+    $v =~ s/["\\]/\\$&/g;
     printf("  { %-15s \"%-20s },\n", $k . ',', $v . '"') or die $!;
 }
