--- module-init-tools-3.2-pre7/modprobe.c.dist	2005-06-01 07:26:28.000000000 -0400
+++ module-init-tools-3.2-pre7/modprobe.c	2005-07-08 15:30:55.000000000 -0400
@@ -1189,8 +1189,8 @@
 
 static const char *default_configs[] = 
 {
-	"/etc/modprobe.conf",
 	"/etc/modprobe.d",
+	"/etc/modprobe.conf",
 };
 
 static void read_toplevel_config(const char *filename,
@@ -1214,9 +1214,8 @@
 
 	/* Try defaults. */
 	for (i = 0; i < ARRAY_SIZE(default_configs); i++) {
-		if (read_config(default_configs[i], name, dump_only, removing,
-				options, commands, aliases, blacklist))
-			return;
+		read_config(default_configs[i], name, dump_only, removing,
+			    options, commands, aliases, blacklist);
 	}
 }
 
