diff -Naur orig/build/resources/0ad.desktop patched/build/resources/0ad.desktop
--- orig/build/resources/0ad.desktop	2013-04-21 18:47:36.000000000 +0200
+++ patched/build/resources/0ad.desktop	2014-05-18 22:20:45.900700926 +0200
@@ -2,6 +2,7 @@
 Version=1.0
 Name=0 A.D.
 Comment=A real-time strategy game of ancient warfare
+Comment[hu]=Valós idejű stratégiai játék
 Comment[it]=Videogioco strategico in tempo reale di guerre antiche
 Exec=0ad
 Icon=0ad
