--- empathy/data/empathy-accounts.desktop.in.in.orig	2011-10-22 18:27:12.000000000 +0200
+++ empathy/data/empathy-accounts.desktop.in.in	2011-10-22 18:27:53.000000000 +0200
@@ -7,7 +7,7 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Settings;DesktopSettings;X-GNOME-PersonalSettings;
+Categories=GNOME;GTK;Settings;DesktopSettings;X-GNOME-PersonalSettings;X-UHU-Menu/Settings;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=empathy
 X-GNOME-Bugzilla-Component=General
