diff -Naur kradio4-4.0.7-ori/kradio4.desktop kradio4-4.0.7/kradio4.desktop
--- kradio4-4.0.7-ori/kradio4.desktop	2013-01-26 18:34:23.000000000 +0100
+++ kradio4-4.0.7/kradio4.desktop	2013-05-15 19:49:32.419763833 +0200
@@ -6,4 +6,4 @@
 X-DocPath=kradio4/index.html
 Comment=V4L/V4L2-Radio Application for KDE4
 Terminal=false
-Categories=Qt;KDE;AudioVideo;
+Categories=Qt;KDE;AudioVideo;Audio;Recorder;
diff -Naur kradio4-4.0.7-ori/src/kradio.desktop kradio4-4.0.7/src/kradio.desktop
--- kradio4-4.0.7-ori/src/kradio.desktop	2013-01-26 18:34:23.000000000 +0100
+++ kradio4-4.0.7/src/kradio.desktop	2013-05-15 19:49:34.467763896 +0200
@@ -10,4 +10,4 @@
 Terminal=false
 Name=KRadio
 Name[de]=KRadio
-Categories=Qt;KDE;AudioVideo;
+Categories=Qt;KDE;AudioVideo;Audio;Recorder;
