diff -Naur podsleuth-0.6.5-orig/configure podsleuth-0.6.5/configure
--- podsleuth-0.6.5-orig/configure	2009-10-02 00:24:22.000000000 +0200
+++ podsleuth-0.6.5/configure	2009-11-11 15:37:52.000000000 +0100
@@ -4469,13 +4469,13 @@
 
 
 	if test "x$LIBSGUTILS_SO" = "xNONE"; then
-		{ $as_echo "$as_me:$LINENO: checking for sg_ll_inquiry in -lsgutils" >&5
-$as_echo_n "checking for sg_ll_inquiry in -lsgutils... " >&6; }
+		{ $as_echo "$as_me:$LINENO: checking for sg_ll_inquiry in -lsgutils2" >&5
+$as_echo_n "checking for sg_ll_inquiry in -lsgutils2... " >&6; }
 if test "${ac_cv_lib_sgutils_sg_ll_inquiry+set}" = set; then
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lsgutils  $LIBS"
+LIBS="-lsgutils2  $LIBS"
 cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
@@ -4541,8 +4541,8 @@
 	fi
 
 	if test "x$LIBSGUTILS_SO" = "xNONE"; then
-		{ { $as_echo "$as_me:$LINENO: error: Error! You need to have libsgutils." >&5
-$as_echo "$as_me: error: Error! You need to have libsgutils." >&2;}
+		{ { $as_echo "$as_me:$LINENO: error: Error! You need to have libsgutils2." >&5
+$as_echo "$as_me: error: Error! You need to have libsgutils2." >&2;}
    { (exit 1); exit 1; }; }
 	fi
 
@@ -5280,7 +5280,8 @@
 if test -n "$CONFIG_FILES"; then
 
 
-ac_cr=''
+ac_cr='
+'
 ac_cs_awk_cr=`$AWK 'BEGIN { print "a\rb" }' </dev/null 2>/dev/null`
 if test "$ac_cs_awk_cr" = "a${ac_cr}b"; then
   ac_cs_awk_cr='\\r'
