diff -Naurdp a/src/loadkeys.y b/src/loadkeys.y
--- a/src/loadkeys.y	2008-10-23 21:03:59.000000000 +0200
+++ b/src/loadkeys.y	2009-12-03 11:27:19.600402819 +0100
@@ -263,6 +263,7 @@ usage(void) {
 "  -m --mktable       output a \"defkeymap.c\" to stdout\n"
 "  -s --clearstrings  clear kernel string table\n"
 "  -u --unicode       implicit conversion to Unicode\n"
+"     --no-unicode    don't perform implicit conversion to Unicode\n"
 "  -v --verbose       report the changes\n"), PACKAGE_VERSION, DEFMAP);
 	exit(1);
 }
@@ -288,12 +289,14 @@ main(int argc, char *argv[]) {
 		{ "mktable",    no_argument, NULL, 'm' },
 		{ "clearstrings", no_argument, NULL, 's' },
 		{ "unicode",	no_argument, NULL, 'u' },
+		{ "no-unicode",	no_argument, NULL, 299 },
 		{ "quiet",	no_argument, NULL, 'q' },
 		{ "verbose",    no_argument, NULL, 'v' },
 		{ "version",	no_argument, NULL, 'V' },
 		{ NULL, 0, NULL, 0 }
 	};
 	int c;
+	int no_unicode = 0;
 	char *console = NULL;
         int warned = 0;
 
@@ -325,7 +328,10 @@ main(int argc, char *argv[]) {
 				opts = 1;
 				break;
 			case 'u':
-				set_charset("unicode");
+				/* accept for compatibility */
+				break;
+			case 299:
+				no_unicode = 1;
 				break;
 			case 'q':
 				quiet = 1;
@@ -341,6 +347,9 @@ main(int argc, char *argv[]) {
 		}
 	}
 
+	if (!no_unicode) {
+		set_charset("unicode");
+	}
 	args = argv + optind - 1;
 	unicode_used = 0;
 	yywrap();	/* set up the first input file, if any */
