diff -Naur umplayer-ori/umplayer.desktop umplayer/umplayer.desktop
--- umplayer-ori/umplayer.desktop	2010-11-28 15:50:42.000000000 +0100
+++ umplayer/umplayer.desktop	2012-07-27 21:42:07.905505330 +0200
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Categories=Qt;AudioVideo;Player;Video;
+Categories=Qt;AudioVideo;Player;Video;X-UHU-Menu/Multimedia/VideoPlayers;
 Comment=A great MPlayer front-end
 Comment[es]=A great MPlayer front-end
 Comment[hu]=Egy grafikus MPlayer kiegészítés
