diff -Naur orig/doc/uhubuild.sgml patched/doc/uhubuild.sgml
--- orig/doc/uhubuild.sgml	2013-10-02 14:37:35.000000000 +0200
+++ patched/doc/uhubuild.sgml	2014-02-23 18:05:33.042183589 +0100
@@ -683,6 +683,12 @@
 	A files kiegészítője arra az esetre, ha az abban csoportosan megadott
 	fájlok közül egy vagy több mégsem tartozik az adott csomagba.
 
+  <tag/excludes [?]/
+
+	Kihagyandó fájlok, soronként egy.
+	A files kiegészítője arra az esetre, ha az abban csoportosan megadott
+	fájlok közül egy vagy több mégsem tartozik az adott csomagba.
+
   <tag/groups [?]/
 
 	Amennyiben a csomag csoporto(ka)t hoz létre, az(oka)t ide tehetjük.
