diff -Naur orig/src/common patched/src/common
--- orig/src/common	2005-08-31 15:54:11.000000000 +0200
+++ patched/src/common	2014-06-15 21:52:59.331878372 +0200
@@ -42,7 +42,7 @@
 
 error ()
 {
-	errors=$(($errors+1))
+	errors=$((errors+1))
 	if [ $errors -gt 100 ]; then
 		die "Tul sok hiba!"
 	fi
@@ -51,13 +51,13 @@
 
 warn ()
 {
-	warnings=$(($warnings+1))
+	warnings=$((warnings+1))
 	blueecho "Warning: $*" >&2
 }
 
 # tmp -------------------------------------------------------------------------
 unset __tmp || true
-declare -a __tmp
+__tmp=() # bash 4.3, declare -a: unbound variable
 tmpfile()
 {
 	local i
@@ -91,7 +91,7 @@
 }
 
 # -----------------------------------------------------------------------------
-declare -a _exit_hooks
+_exit_hooks=() # bash 4.3, declare -a: unbound variable
 exit_hook ()
 {
 	_exit_hooks[${#_exit_hooks[@]}]="$1"
diff -Naur orig/src/helpers/ub_configure patched/src/helpers/ub_configure
--- orig/src/helpers/ub_configure	2009-11-29 11:05:24.000000000 +0100
+++ patched/src/helpers/ub_configure	2014-06-15 21:45:59.422939675 +0200
@@ -9,8 +9,8 @@
 
 export CFLAGS="$UB_CFLAGS"
 
-declare -a argv
-declare -a argx=(
+argv=() # bash 4.3, declare -a: unbound variable
+argx=(
 	--prefix="/usr"             \
 	--sysconfdir="/etc"         \
 	--localstatedir="/var"      \
diff -Naur orig/src/helpers/ub_makeinstall patched/src/helpers/ub_makeinstall
--- orig/src/helpers/ub_makeinstall	2009-03-29 19:51:15.000000000 +0200
+++ patched/src/helpers/ub_makeinstall	2014-06-15 21:45:59.422939675 +0200
@@ -33,7 +33,7 @@
 		;;
 esac
 
-declare -a argv
+argv=() # bash 4.3, declare -a: unbound variable
 if [ "$UB_MAKEINSTALL_USE_DESTDIR" = "yes" ]; then
 	argv[${#argv[@]}]="DESTDIR=$UB_MAKEINSTALL_PREFIX"
 # ez meg szokott lenni:
@@ -57,7 +57,7 @@
 argv[${#argv[@]}]="MAN7EXT=7"
 argv[${#argv[@]}]="MAN8EXT=8"
 
-declare -a argv2
+argv2=() # bash 4.3, declare -a: unbound variable
 shopt -s extglob
 for i in "${argv[@]}"; do
 	pattern="$i"
diff -Naur orig/src/steps/menu patched/src/steps/menu
--- orig/src/steps/menu	2014-05-07 09:58:01.000000000 +0200
+++ patched/src/steps/menu	2014-06-15 21:45:59.422939675 +0200
@@ -21,7 +21,7 @@
 			grep -vw 'Encoding=UTF-8' "$f" | grep '^Encoding=' >/dev/null \
 			  && error "$f: hibás vagy nem támogatott Encoding sor!" \
 			  || true
-			[ $(grep ^Categories= "$f" | wc -l) -le 1 ] \
+			[ $(grep -c ^Categories= "$f") -le 1 ] \
 			  || error "$f: egynél több Categories sort tartalmaz!"
 			fgrep -q '${' "$f" \
 			  && error "$f: maradt benne '\${', szar a configure vagy a Makefile!" \
diff -Naur orig/src/uhubuild-build patched/src/uhubuild-build
--- orig/src/uhubuild-build	2013-08-13 16:42:58.000000000 +0200
+++ patched/src/uhubuild-build	2014-06-15 21:45:59.422939675 +0200
@@ -143,9 +143,9 @@
 #	greenecho " OK"
 fi
 
-steps="$steps python addons cleanup optipng split dirs users-groups changelog suid_wrapper"\
-" perms scripts remove-misc man info mo"\
-" timestamps menu depends shlibs fhs conffiles buildinfo control pack"
+steps="$steps python addons cleanup optipng split dirs users-groups changelog suid_wrapper \
+  perms scripts remove-misc man info mo \
+  timestamps menu depends shlibs fhs conffiles buildinfo control pack"
 
 for step in $steps; do
 	echo "-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=- UHUBUILD -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-" >> "$UB_ADMINDIR/log"
diff -Naur orig/src/uhubuild-chrootbuild patched/src/uhubuild-chrootbuild
--- orig/src/uhubuild-chrootbuild	2014-04-14 13:33:44.000000000 +0200
+++ patched/src/uhubuild-chrootbuild	2014-06-15 21:47:22.415910089 +0200
@@ -81,10 +81,10 @@
 export  UHUBUILD_APT_PREFERENCES
 distribution="$(<"$UHUBUILD_SRCDIR/distribution")"
 prio=1000
-declare -a versions=()
+versions=() # bash 4.3, declare -a: unbound variable
 for version in $(echo "${distribution##* }"|tr ':' ' '); do
 	versions[${#versions[@]}]="$version"
-	prio="$(($prio + 1))"
+	prio="$((prio + 1))"
 	IFS=":" cat >>"$UHUBUILD_APT_PREFERENCES" <<EOF
 Package: *
 Pin: release v=${versions[*]},l=${distribution% *}
@@ -200,13 +200,13 @@
 
 greenecho -n 'Fájlrendszerek csatolása: '
 
-for i in 1 2 3; do
+# for i in 1 2 3; do
 	umount -lf "$UHUBUILD_CHROOTDIR/dev" 2>/dev/null || true
 	umount -lf "$UHUBUILD_CHROOTDIR/proc" 2>/dev/null || true
 	umount -lf "$UHUBUILD_CHROOTDIR/mnt/ccache" 2>/dev/null || true
 	umount -lf "$UHUBUILD_CHROOTDIR/mnt/snapshot" 2>/dev/null || true
 	umount -lf "$UHUBUILD_CHROOTDIR/$UHUBUILD_SRCDIR" 2>/dev/null || true
-done
+# done
 
 echo -n 'dev '
 mkdir -p "$UHUBUILD_CHROOTDIR/dev"
diff -Naur orig/src/uhubuild-commonenvironment patched/src/uhubuild-commonenvironment
--- orig/src/uhubuild-commonenvironment	2006-04-14 15:31:43.000000000 +0200
+++ patched/src/uhubuild-commonenvironment	2014-06-15 21:46:45.995483884 +0200
@@ -3,7 +3,7 @@
 exec 1>&1
 exec 2>&2
 
-unset `bash -c declare | grep -v ^UHUBUILD | grep -v ^BASH_ | cut -d= -f1` 2>/dev/null || true
+unset $(bash -c declare | grep -v ^UHUBUILD | grep -v ^BASH_ | cut -d= -f1) 2>/dev/null || true
 
 for i in /etc/env.d/*.env; do
 	if [ -f "$i" ]; then
diff -Naur orig/src/uhubuild-pack patched/src/uhubuild-pack
--- orig/src/uhubuild-pack	2013-09-18 13:58:09.000000000 +0200
+++ patched/src/uhubuild-pack	2014-06-15 21:45:59.426939726 +0200
@@ -123,7 +123,7 @@
   /usr/src \
   /var; do
 	calc_size
-	total_size="$(($total_size + $size))"
+	total_size="$((total_size + size))"
 	if [ "$size" != 0 ]; then
 		printf " %6s %s\n" "$size" "$dir"
 	fi
