diff -Naur orig/config/macros patched/config/macros
--- orig/config/macros	2014-04-26 17:53:49.682968359 +0200
+++ patched/config/macros	2014-04-26 17:35:11.000000000 +0200
@@ -1,4 +1,4 @@
-%_ autoconf automake bash bc binutils bison bzip2 bzip2-dev ccache coreutils cracklib diffutils dl dpkg file findutils flex flex-dev gawk gcc gcc-lib gettext gettext-tools glibc glibc-dev glibc-locales grep gzip kernel-headers libstdc++ libstdc++-dev libtool licenses m4 make ncurses ncurses-dev optipng pam pam-dev patch perl pkgconfig pretendroot readline readline-dev sed shadow sortdir tar texinfo uhu-base uhubuild-helpers uhubuild-here uhu-pkg uhu-utils which xz-dev zlib zlib-dev
+%_ autoconf automake bash bc binutils bison bzip2 bzip2-dev ccache coreutils cracklib desktop-file-utils diffutils dl dpkg file findutils flex flex-dev gawk gcc gcc-lib gettext gettext-tools glibc glibc-dev glibc-locales grep gzip kernel-headers libstdc++ libstdc++-dev libtool licenses m4 make ncurses ncurses-dev optipng pam pam-dev patch perl pkgconfig pretendroot readline readline-dev sed shadow sortdir tar texinfo uhu-base uhubuild-helpers uhubuild-here uhu-pkg uhu-utils which xz-dev zlib zlib-dev
 
 %alsa-lib	alsa-lib{,-dev}
 %db		db{,-dev}
diff -Naur orig/src/steps/menu patched/src/steps/menu
--- orig/src/steps/menu	2007-08-15 12:26:05.000000000 +0200
+++ patched/src/steps/menu	2014-04-26 23:34:32.844041714 +0200
@@ -8,36 +8,7 @@
 
 for pkg in $UB_PACKAGES; do
 	cd "$UB_PACKAGEDIR/$pkg"
-	for dir in usr/lib/desktop-directories usr/lib/applications; do
-		if [ ! -d "$dir" ]; then
-			continue
-		fi
-		for f in "$dir"/*; do
-			echo "  Converting ${f##*/}..."
-			new="${f/lib/share}"
-			if [ -f "$new" ]; then
-				error "$new already exists!"
-			fi
-			mkdir -p "${new%/*}"
-			if ! grep -q '^X-UHU-Menu=' "$f"; then
-				pretendroot mv "$f" "$new"
-			else
-				if ! grep -q '^Categories=' "$f"; then
-					sed 's,^X-UHU-Menu=\(.*\)$,Categories=X-UHU-Menu/\1;,' <"$f" >"$new"
-				else
-					menu="$(grep ^X-UHU-Menu "$f" | head -n 1 | cut -d= -f2-)"
-					if ! egrep -q '^Categories=(.*;)?$' "$f"; then
-						sed 's,^\(Categories=.*\)$,\1;X-UHU-Menu/'"$menu"';,' <"$f" | grep -v '^X-UHU-Menu=' >"$new"
-					else
-						sed 's,^\(Categories=.*\)$,\1X-UHU-Menu/'"$menu"';,' <"$f" | grep -v '^X-UHU-Menu=' >"$new"
-					fi
-				fi
-				pretendroot rm "$f"
-			fi
-			pretendroot rmdir --ignore-fail-on-non-empty "${f%/*}"
-		done
-	done
-	for dir in usr/share/desktop-directories usr/share/applications; do
+	for dir in usr/share/desktop-directories usr/share/applications usr/lib/desktop-directories usr/lib/applications; do
 		if [ ! -d "$dir" ]; then
 			continue
 		fi
@@ -55,8 +26,11 @@
 			fgrep -q '${' "$f" \
 			  && error "$f: maradt benne '\${', szar a configure vagy a Makefile!" \
 			  || true
-			grep -q -e '^Exec=/' -e '^TryExec=/' "$f" \
-			  && error "$f: Exec és TryExec után ne álljon útvonal!" \
+#			grep -q -e '^Exec=/' -e '^TryExec=/' "$f" \
+#			  && error "$f: Exec és TryExec után ne álljon útvonal!" \
+#			  || true
+			desktop-file-validate "$f" | grep -E 'error|warning' \
+			  && error "$f: hibás desktop fájl!" \
 			  || true
 		done
 	done
