diff -Naur cantor-4.9.4-ori/src/cantor.desktop cantor-4.9.4/src/cantor.desktop
--- cantor-4.9.4-ori/src/cantor.desktop	2012-11-30 08:31:01.000000000 +0100
+++ cantor-4.9.4/src/cantor.desktop	2012-12-17 22:01:09.615978766 +0100
@@ -85,5 +85,5 @@
 GenericName[zh_CN]=数学软件的 KDE 前端
 GenericName[zh_TW]=KDE 數學軟體的前端介面
 Terminal=false
-Categories=Qt;KDE;Education;Math;Science;
+Categories=Qt;KDE;Education;Math;Science;X-UHU-Menu/Edutainment/Mathematics;
 MimeType=application/x-cantor;
