diff -Naur orig/configure.in patched/configure.in
--- orig/configure.in	2014-03-26 07:39:01.000000000 +0100
+++ patched/configure.in	2014-04-12 23:17:31.748212558 +0200
@@ -129,10 +129,6 @@
 AC_CHECK_HEADER(apt-pkg/configuration.h, [],
  AC_ERROR([You need the apt-pkg headers installed to compile synaptic.]))
 
-# if metaindex is there we use a apt with authentication support
-AC_CHECK_HEADER(apt-pkg/metaindex.h,
-        AC_DEFINE(WITH_APT_AUTH, 1, [build with apt auth support] )
-)
 # check if we have apts new-style cdrom.h
 AC_CHECK_HEADER(apt-pkg/cdrom.h, 
                 AC_DEFINE(HAVE_APTPKG_CDROM, 1, 
