diff -Naur orig/src/network/Makefile.am patched/src/network/Makefile.am
--- orig/src/network/Makefile.am	2013-08-06 23:19:33.000000000 +0200
+++ patched/src/network/Makefile.am	2014-03-05 19:45:57.615318695 +0100
@@ -15,7 +15,7 @@
 CLEANFILES = $(GST_TOOL_CLEANFILES)
 EXTRA_DIST = $(GST_TOOL_EXTRA_DIST)
 
-mate_network_admin_LDADD = $(GST_TOOL_LIBS) $(GST_IW_LIBS) $(DBUS_LIBS) $(GIO_LIBS)
+mate_network_admin_LDADD = $(GST_TOOL_LIBS) $(GST_IW_LIBS) $(DBUS_LIBS) $(GIO_LIBS) -lm
 mate_network_admin_DEPENDENCIES = $(GST_TOOL_DEPENDENCIES)
 
 if HAVE_LIBIW_H
diff -Naur orig/src/network/Makefile.in patched/src/network/Makefile.in
--- orig/src/network/Makefile.in	2014-03-01 14:40:03.000000000 +0100
+++ patched/src/network/Makefile.in	2014-03-05 19:46:41.527723963 +0100
@@ -400,7 +400,7 @@
 toolname = mate-network
 CLEANFILES = $(GST_TOOL_CLEANFILES)
 EXTRA_DIST = $(GST_TOOL_EXTRA_DIST)
-mate_network_admin_LDADD = $(GST_TOOL_LIBS) $(GST_IW_LIBS) $(DBUS_LIBS) $(GIO_LIBS)
+mate_network_admin_LDADD = $(GST_TOOL_LIBS) $(GST_IW_LIBS) $(DBUS_LIBS) $(GIO_LIBS) -lm
 mate_network_admin_DEPENDENCIES = $(GST_TOOL_DEPENDENCIES)
 @HAVE_LIBIW_H_TRUE@essid_SOURCES = essid-list.c essid-list.h
 mate_network_admin_SOURCES = \
