diff -Naur orig/xye.desktop mod/xye.desktop
--- orig/xye.desktop	2013-08-13 20:29:58.000000000 +0200
+++ mod/xye.desktop	2014-05-04 19:01:29.135719975 +0200
@@ -1,14 +1,9 @@
-#!/usr/bin/env xdg-open
-
 [Desktop Entry]
-Encoding=UTF-8
 Exec=xye
 Type=Application
-Categories=Application;Game;PuzzleGame
+Categories=Game;LogicGame;
 Name=Xye
-GenericName=Xye
 Comment=Solve puzzles and action challenges in order to get all the gems in the room.
+Comment[hu]=Rendezd el a gyémántokat helyesen!
 Terminal=false
-StartupNotify=false
-Icon=xye.png
-MimeType=application/xye-xml
+Icon=xye
