diff -Naur orig/chrome/browser/ui/gtk/chrome_browser_main_extra_parts_gtk.cc patched/chrome/browser/ui/gtk/chrome_browser_main_extra_parts_gtk.cc
--- orig/chrome/browser/ui/gtk/chrome_browser_main_extra_parts_gtk.cc	2013-09-28 19:10:31.000000000 +0200
+++ patched/chrome/browser/ui/gtk/chrome_browser_main_extra_parts_gtk.cc	2013-10-04 16:16:39.411149672 +0200
@@ -19,7 +19,7 @@
 }
 
 void ChromeBrowserMainExtraPartsGtk::PreEarlyInitialization() {
-  DetectRunningAsRoot();
+//  DetectRunningAsRoot();
 }
 
 
