--- a/data/gtali.desktop.in.in.orig	2011-01-24 23:46:40.000000000 +0100
+++ a/data/gtali.desktop.in.in	2011-10-20 18:29:51.000000000 +0200
@@ -5,7 +5,7 @@
 Icon=tali
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Game;CardGame;
+Categories=GNOME;GTK;Game;CardGame;X-UHU-Menu/Games/Card;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=BugBuddyBugs
