diff -Naur vixie-cron-4.1.orig/pathnames.h vixie-cron-4.1/pathnames.h
--- vixie-cron-4.1.orig/pathnames.h	2006-03-23 17:08:13.000000000 +0100
+++ vixie-cron-4.1/pathnames.h	2006-03-23 17:10:57.000000000 +0100
@@ -35,7 +35,7 @@
 			 * to; SPOOL_DIR, CRON_ALLOW, CRON_DENY, and LOG_FILE
 			 * are all relative to this directory.
 			 */
-#define CRONDIR		"/var/spool"
+#define CRONDIR		"/var/spool/cron"
 #endif
 
 			/* SPOOLDIR is where the crontabs live.
@@ -46,13 +46,13 @@
 			 * newer than they were last time around (or which
 			 * didn't exist last time around...)
 			 */
-#define SPOOL_DIR	"cron"
+#define SPOOL_DIR	"tabs"
 
 			/* cron allow/deny file.  At least cron.deny must
 			 * exist for ordinary users to run crontab.
 			 */
-#define	CRON_ALLOW	"/etc/cron.allow"
-#define	CRON_DENY	"/etc/cron.deny"
+#define	CRON_ALLOW	"allow"
+#define	CRON_DENY	"deny"
 
 			/* undefining this turns off logging to a file.  If
 			 * neither LOG_FILE or SYSLOG is defined, we don't log.
@@ -85,11 +85,11 @@
 #if defined(_PATH_VI)
 # define EDITOR _PATH_VI
 #else
-# define EDITOR "/usr/ucb/vi"
+# define EDITOR "/bin/vi"
 #endif
 
 #ifndef _PATH_SENDMAIL
-# define _PATH_SENDMAIL "/usr/lib/sendmail"
+# define _PATH_SENDMAIL "/usr/sbin/sendmail"
 #endif
 
 #ifndef _PATH_BSHELL
@@ -97,7 +97,7 @@
 #endif
 
 #ifndef _PATH_DEFPATH
-# define _PATH_DEFPATH "/usr/bin:/bin"
+# define _PATH_DEFPATH "/usr/local/bin:/bin:/usr/bin"
 #endif
 
 #ifndef _PATH_TMP
