diff -Naurd ncftp-3.1.5.orig/ncftp/pref.c ncftp-3.1.5/ncftp/pref.c
--- ncftp-3.1.5.orig/ncftp/pref.c	2002-03-27 11:34:58.000000000 +0100
+++ ncftp-3.1.5/ncftp/pref.c	2003-03-12 17:06:58.000000000 +0100
@@ -612,7 +612,7 @@
 	gAutoResume = 0;
 	gSOBufsize = 0;
 	gMaxLogSize = 10240;
-	gConfirmClose = 1;
+	gConfirmClose = 0;
 	gAutoSaveChangesToExistingBookmarks = 0;
 	gRedialDelay = kDefaultRedialDelay;
 	STRNCPY(gAutoAscii, "|.txt|.asc|.html|.htm|.css|.xml|.ini|.pl|.hqx|.cfg|.c|.h|.cpp|.hpp|.bat|.m3u|.pls|");
