diff -Naur orig/pppd/pathnames.h mod/pppd/pathnames.h
--- orig/pppd/pathnames.h	2009-11-16 23:26:07.000000000 +0100
+++ mod/pppd/pathnames.h	2014-02-06 14:51:07.000000000 +0100
@@ -57,7 +57,7 @@
 
 #ifdef PLUGIN
 #ifdef __STDC__
-#define _PATH_PLUGIN	DESTDIR "/lib/pppd/" VERSION
+#define _PATH_PLUGIN	DESTDIR "/usr/lib/pppd/" VERSION
 #else /* __STDC__ */
 #define _PATH_PLUGIN	"/usr/lib/pppd"
 #endif /* __STDC__ */
