--- a/easytag.desktop.in.orig	2013-02-12 20:22:39.000000000 +0100
+++ a/easytag.desktop.in	2013-02-12 20:22:58.000000000 +0100
@@ -5,7 +5,7 @@
 _Keywords=mp3;tag;audio;
 Type=Application
 Icon=easytag
-Categories=GTK;AudioVideo;AudioVideoEditing;
+Categories=GTK;AudioVideo;AudioVideoEditing;X-UHU-Menu/Multimedia/More;
 MimeType=x-directory/normal;inode/directory;audio/x-mp3;audio/x-mpeg;audio/mpeg;application/ogg;audio/x-vorbis+ogg;audio/x-flac;audio/x-musepack;audio/x-ape;
 Exec=easytag %F
 Terminal=false
