diff -Naur orig/data/easytag.desktop.in patched/data/easytag.desktop.in
--- orig/data/easytag.desktop.in	2014-01-16 17:06:25.000000000 +0100
+++ patched/data/easytag.desktop.in	2014-03-20 22:56:21.370107467 +0100
@@ -6,7 +6,7 @@
 _Keywords=mp3;tag;audio;
 Type=Application
 Icon=easytag
-Categories=GTK;AudioVideo;AudioVideoEditing;Audio;
+Categories=GTK;AudioVideo;AudioVideoEditing;Audio;X-UHU-Menu/Multimedia/More;
 MimeType=inode/directory;audio/x-mp3;audio/x-mpeg;audio/mpeg;application/ogg;audio/x-vorbis+ogg;audio/x-flac;audio/x-musepack;audio/x-ape;
 Exec=easytag %F
 Terminal=false
