diff -Naur orig/configure.ac patched/configure.ac
--- orig/configure.ac	2013-12-23 15:14:36.000000000 +0100
+++ patched/configure.ac	2014-01-05 20:35:00.509421564 +0100
@@ -7,8 +7,8 @@
 AC_LIBTOOL_DLOPEN
 AC_PROG_LIBTOOL
 
-CPPFLAGS="$CPPFLAGS -Wall -g -fexceptions"
-CXXFLAGS="$CXXFLAGS -Wall -g -fexceptions"
+CPPFLAGS="$CPPFLAGS"
+CXXFLAGS="$CXXFLAGS"
 
 WXCONFIG=wx-config
 LIBXMLCONFIG=xml2-config
