diff -Naur orig/libmate-desktop/mate-desktop-thumbnail.c patched/libmate-desktop/mate-desktop-thumbnail.c
--- orig/libmate-desktop/mate-desktop-thumbnail.c	2013-02-21 22:11:45.000000000 +0100
+++ patched/libmate-desktop/mate-desktop-thumbnail.c	2013-03-23 13:37:12.203580980 +0100
@@ -929,13 +929,8 @@
 
   file = g_strconcat (g_checksum_get_string (checksum), ".png", NULL);
   
-#if GLIB_CHECK_VERSION (2, 34, 0)
-  path = g_build_filename (g_get_user_cache_dir (),
-                           "thumbnails",
-#else
   path = g_build_filename (g_get_home_dir (),
                            ".thumbnails",
-#endif
                            (priv->size == MATE_DESKTOP_THUMBNAIL_SIZE_NORMAL)?"normal":"large",
                            file,
                            NULL);
@@ -995,13 +990,8 @@
 
   file = g_strconcat (g_checksum_get_string (checksum), ".png", NULL);
 
-#if GLIB_CHECK_VERSION (2, 34, 0)
-  path = g_build_filename (g_get_user_cache_dir (),
-                           "thumbnails/fail",
-#else
   path = g_build_filename (g_get_home_dir (),
                            ".thumbnails/fail",
-#endif
                            appname,
                            file,
                            NULL);
@@ -1359,13 +1349,8 @@
 
   res = FALSE;
 
-#if GLIB_CHECK_VERSION (2, 34, 0)
-  thumbnail_dir = g_build_filename (g_get_user_cache_dir (),
-                                    "thumbnails",
-#else
   thumbnail_dir = g_build_filename (g_get_home_dir (),
                                     ".thumbnails",
-#endif
                                     NULL);
   if (!g_file_test (thumbnail_dir, G_FILE_TEST_IS_DIR))
     {
@@ -1398,13 +1383,8 @@
 
   res = FALSE;
 
-#if GLIB_CHECK_VERSION (2, 34, 0)
-  thumbnail_dir = g_build_filename (g_get_user_cache_dir (),
-                                    "thumbnails",
-#else
   thumbnail_dir = g_build_filename (g_get_home_dir (),
                                     ".thumbnails",
-#endif
                                     NULL);
   if (!g_file_test (thumbnail_dir, G_FILE_TEST_IS_DIR))
     {
@@ -1478,13 +1458,8 @@
 
   file = g_strconcat (g_checksum_get_string (checksum), ".png", NULL);
 
-#if GLIB_CHECK_VERSION (2, 34, 0)
-  path = g_build_filename (g_get_user_cache_dir (),
-                           "thumbnails",
-#else
   path = g_build_filename (g_get_home_dir (),
                            ".thumbnails",
-#endif
                            (priv->size == MATE_DESKTOP_THUMBNAIL_SIZE_NORMAL)?"normal":"large",
                            file,
                            NULL);
@@ -1591,13 +1566,8 @@
 
   file = g_strconcat (g_checksum_get_string (checksum), ".png", NULL);
 
-#if GLIB_CHECK_VERSION (2, 34, 0)
-  path = g_build_filename (g_get_user_cache_dir (),
-                           "thumbnails/fail",
-#else
   path = g_build_filename (g_get_home_dir (),
                            ".thumbnails/fail",
-#endif
                            appname,
                            file,
                            NULL);
@@ -1687,13 +1657,8 @@
   file = g_strconcat (md5, ".png", NULL);
   g_free (md5);
   
-#if GLIB_CHECK_VERSION (2, 34, 0)
-  path = g_build_filename (g_get_user_cache_dir (),
-                           "thumbnails",
-#else
   path = g_build_filename (g_get_home_dir (),
                            ".thumbnails",
-#endif
                            (size == MATE_DESKTOP_THUMBNAIL_SIZE_NORMAL)?"normal":"large",
                            file,
                            NULL);
