diff -Naur orig/xfce4-terminal.desktop.in patched/xfce4-terminal.desktop.in
--- orig/xfce4-terminal.desktop.in	2013-01-01 13:09:30.000000000 +0100
+++ patched/xfce4-terminal.desktop.in	2013-03-02 13:50:25.200264822 +0100
@@ -7,5 +7,5 @@
 Icon=utilities-terminal
 Terminal=false
 Type=Application
-Categories=GTK;System;TerminalEmulator;
+Categories=GTK;System;TerminalEmulator;X-UHU-Menu/Tools/Terminals;
 StartupNotify=true
