diff -Naur ORIG/data/desktop/parole.desktop.in.in UHU/data/desktop/parole.desktop.in.in
--- ORIG/data/desktop/parole.desktop.in.in	2009-10-21 12:16:25.000000000 +0200
+++ UHU/data/desktop/parole.desktop.in.in	2009-11-07 18:32:21.000000000 +0100
@@ -4,5 +4,5 @@
 Exec=parole %F
 Terminal=false
 Type=Application
-Categories=GTK;AudioVideo;Player;Video;
+Categories=GTK;AudioVideo;Player;Video;X-UHU-Menu/Multimedia/VideoPlayers;
 StartupNotify=true
