--- a/data/gnobots2.desktop.in.in.orig	2011-01-24 23:46:40.000000000 +0100
+++ a/data/gnobots2.desktop.in.in	2011-10-20 18:26:56.000000000 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-robots
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Game;ArcadeGame;
+Categories=GNOME;GTK;Game;ArcadeGame;X-UHU-Menu/Games/Arcade;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=BugBuddyBugs
