diff -Naurd perl-5.10.1.orig/MANIFEST perl-5.10.1/MANIFEST
--- perl-5.10.1.orig/MANIFEST	2009-08-04 00:16:32.000000000 +0200
+++ perl-5.10.1/MANIFEST	2009-10-01 18:23:39.000000000 +0200
@@ -1026,7 +1026,6 @@
 ext/Module-Pluggable/t/lib/Acme/Foo-Bar.pm			Module::Pluggable tests
 ext/Module-Pluggable/t/lib/Acme/MyTest/Plugin/Foo.pm	Module::Pluggable tests
 ext/Module-Pluggable/t/lib/EditorJunk/Plugin/Bar.pm		Module::Pluggable tests
-ext/Module-Pluggable/t/lib/EditorJunk/Plugin/Bar.pm~	Module::Pluggable tests
 ext/Module-Pluggable/t/lib/EditorJunk/Plugin/Bar.pm.swo	Module::Pluggable tests
 ext/Module-Pluggable/t/lib/EditorJunk/Plugin/Bar.pm.swp	Module::Pluggable tests
 ext/Module-Pluggable/t/lib/EditorJunk/Plugin/Foo.pm		Module::Pluggable tests
