diff -Naur orig/config/lxqt-config-notificationd.desktop.in mod/config/lxqt-config-notificationd.desktop.in
--- orig/config/lxqt-config-notificationd.desktop.in	2014-05-07 09:32:57.000000000 +0200
+++ mod/config/lxqt-config-notificationd.desktop.in	2014-07-12 17:07:28.220363533 +0200
@@ -5,7 +5,6 @@
 Comment=Configure desktop notifications
 Exec=lxqt-config-notificationd
 Icon=preferences-desktop-theme
-Categories=Settings;DesktopSettings;Qt;LXQt;
-OnlyShowIn=LXQt;LXDE;
+Categories=Settings;DesktopSettings;Qt;
 
 #TRANSLATIONS_DIR=translations
