diff -Naurd orig/isomaster.desktop.src mod/isomaster.desktop.src
--- orig/isomaster.desktop.src	2008-12-28 05:03:53.000000000 +0100
+++ mod/isomaster.desktop.src	2009-06-09 06:43:13.000000000 +0200
@@ -6,9 +6,10 @@
 Comment=Read, write and modify ISO images
 Comment[ca]=Llegiu, escriviu i modifiqueu imatges ISO
 Comment[es]=Leer, escribir i modificar imagenes ISO
+Comment[hu]=ISO szerkesztő
 Exec=isomaster
 Terminal=false
 StartupNotify=true
 Type=Application
-Categories=AudioVideo;DiscBurning;
+Categories=AudioVideo;DiscBurning;X-UHU-Menu/Tools/Utilities;
 MimeType=application/x-iso;
diff -Naurd orig/Makefile mod/Makefile
--- orig/Makefile	2010-01-04 23:43:26.000000000 +0100
+++ mod/Makefile	2010-01-06 23:25:16.000000000 +0100
@@ -1,5 +1,5 @@
 # Other paths are relative to this.
-PREFIX ?= /usr/local
+PREFIX ?= /usr
 
 # Where to install the executable.
 BINPATH ?= $(PREFIX)/bin
