diff -Naur nagios-3.2.0-orig/daemon-init.in nagios-3.2.0/daemon-init.in
--- nagios-3.2.0-orig/daemon-init.in	2008-11-30 18:22:58.000000000 +0100
+++ nagios-3.2.0/daemon-init.in	2009-11-29 19:25:59.000000000 +0100
@@ -98,7 +98,7 @@
 NagiosCommandFile=@localstatedir@/rw/nagios.cmd
 NagiosVarDir=@localstatedir@
 NagiosRunFile=@lockfile@
-NagiosLockDir=/var/lock/subsys
+NagiosLockDir=/var/lock
 NagiosLockFile=nagios
 NagiosCGIDir=@sbindir@
 NagiosUser=@nagios_user@
