diff -Naur nagios-3.2.0-orig/daemon-init.in nagios-3.2.0/daemon-init.in
--- nagios-3.2.0-orig/daemon-init.in	2009-11-29 19:30:03.000000000 +0100
+++ nagios-3.2.0/daemon-init.in	2009-11-29 19:31:09.000000000 +0100
@@ -101,8 +101,8 @@
 NagiosLockDir=/var/lock
 NagiosLockFile=nagios
 NagiosCGIDir=@sbindir@
-NagiosUser=@nagios_user@
-NagiosGroup=@nagios_grp@
+NagiosUser=nagios
+NagiosGroup=nagios
           
 
 # Check that nagios exists.
