diff -Naurd audacious.orig/audacious2.desktop audacious/audacious2.desktop
--- audacious.orig/audacious.desktop	2010-04-06 22:01:24.000000000 +0200
+++ audacious/audacious.desktop	2010-05-01 10:10:00.000000000 +0200
@@ -31,7 +31,7 @@
 Comment[es]=Escucha música
 Icon=audacious
 
-Categories=AudioVideo;Audio;Player;GTK;
+Categories=GTK;AudioVideo;Audio;Player;X-UHU-Menu/Multimedia/MusicPlayers;
 
 Exec=audacious %U
 TryExec=audacious
