diff -Naur orig/src/audacity.desktop.in mod/src/audacity.desktop.in
--- orig/src/audacity.desktop.in	2015-03-16 21:34:05.000000000 +0100
+++ mod/src/audacity.desktop.in	2015-08-28 19:16:35.000000000 +0200
@@ -4,9 +4,11 @@
 Name[ru]=Audacity
 GenericName=Sound Editor
 GenericName[de]=Audio-Editor
+GenericName[hu]=Hangszerkesztő
 GenericName[ru]=Редактор звуковых файлов
 Comment=Record and edit audio files
 Comment[de]=Audio-Dateien aufnehmen und bearbeiten
+Comment[hu]=Hangfájlok felvétele és szerkesztése
 Comment[ru]=Запись и редактирование звуковых файлов
 
 Icon=@AUDACITY_NAME@
