diff -Naur orig/data/mozo.desktop.in.in patched/data/mozo.desktop.in.in
--- orig/data/mozo.desktop.in.in	2014-03-01 15:09:33.000000000 +0100
+++ patched/data/mozo.desktop.in.in	2014-03-05 19:33:32.048437338 +0100
@@ -5,7 +5,7 @@
 Terminal=false
 Type=Application
 StartupNotify=false
-Categories=Settings;DesktopSettings;
+Categories=Settings;DesktopSettings;X-UHU-Menu/Tools/Utilities;
 Keywords=application;main;menu;editor;customize;MATE;
 OnlyShowIn=MATE;
 Icon=mozo
