diff -Naur orig/supertux2.desktop mod/supertux2.desktop
--- orig/supertux2.desktop	2010-03-01 13:40:55.000000000 +0100
+++ mod/supertux2.desktop	2014-05-07 20:19:27.000000000 +0200
@@ -1,7 +1,5 @@
 [Desktop Entry]
 Type=Application
-Version=1.0
-Encoding=UTF-8
 Name=SuperTux 2
 Name[en]=SuperTux 2
 GenericName=Platform Game
@@ -35,7 +33,7 @@
 Comment[sv]=Ett klassiskt tvådimensionellt plattformsspel
 Comment[pt_BR]=Encarne o pinguin Tux neste jogo inspirado em clássicos de Pular&Correr
 Comment[hu]=Egy klasszikus 2D-s oldalnézeti játék
-Icon=supertux.png
+Icon=supertux
 Exec=supertux2
 Terminal=false
 StartupNotify=false
