diff -Naur orig/configure patched/configure
--- orig/configure	2013-12-05 17:08:46.000000000 +0100
+++ patched/configure	2014-01-26 20:54:15.000000000 +0100
@@ -16461,6 +16461,7 @@
 MOZ_SAMPLE_TYPE_S16=
 MOZ_OPUS=1
 MOZ_WEBM=1
+MOZ_WEBP=1
 MOZ_DASH=
 MOZ_DIRECTSHOW=
 MOZ_WMF=
@@ -20487,6 +20488,29 @@
         MOZ_DIRECTSHOW=1
 fi
 
+# Check whether --enable-webp or --disable-webp was given.
+if test "${enable_webm+set}" = set; then
+  enableval="$enable_webm"
+  if test "$enableval" = "no"; then
+    MOZ_WEBP=
+  elif test "$enableval" = "yes"; then
+    MOZ_WEBP=1
+  else
+    { echo "configure: error: Option, webp, does not take an argument ($enableval)." 1>&2; echo "configure: error: Option, webp, does not take an argument ($enableval)." 1>&5; exit 1; }
+  fi
+fi
+
+
+if test -n "$MOZ_WEBP"; then
+    cat >> confdefs.pytmp <<\EOF
+    (''' MOZ_WEBP ''', ' 1 ')
+EOF
+cat >> confdefs.h <<\EOF
+#define MOZ_WEBP 1
+EOF
+
+fi;
+
 # Check whether --enable-directshow or --disable-directshow was given.
 if test "${enable_directshow+set}" = set; then
   enableval="$enable_directshow"
@@ -28748,6 +28772,7 @@
     (''' MOZ_TREMOR ''', r''' $MOZ_TREMOR ''')
     (''' MOZ_OPUS ''', r''' $MOZ_OPUS ''')
     (''' MOZ_WEBM ''', r''' $MOZ_WEBM ''')
+    (''' MOZ_WEBP ''', r''' $MOZ_WEBP ''')
     (''' MOZ_DASH ''', r''' $MOZ_DASH ''')
     (''' MOZ_WMF ''', r''' $MOZ_WMF ''')
     (''' MOZ_DIRECTSHOW ''', r''' $MOZ_DIRECTSHOW ''')
diff -Naur orig/Makefile.in patched/Makefile.in
--- orig/Makefile.in	2013-12-05 17:07:20.000000000 +0100
+++ patched/Makefile.in	2014-01-27 05:32:15.962769835 +0100
@@ -72,14 +72,12 @@
 	@echo "Please rerun autoconf and re-configure your build directory."
 	@echo "To ignore this message, touch 'configure' in the source directory,"
 	@echo "but your build might not succeed."
-	@exit 1
 
 config.status: $(topsrcdir)/configure
 	@echo "STOP!  configure has changed and needs to be run in this build directory."
 	@echo "Please rerun configure."
 	@echo "To ignore this message, touch 'config.status' in the build directory,"
 	@echo "but your build might not succeed."
-	@exit 1
 
 export::
 	$(RM) -r $(DIST)/sdk
