diff -Naur orig/browser/components/preferences/applications.js patched/browser/components/preferences/applications.js
--- orig/browser/components/preferences/applications.js	2012-10-24 16:32:25.000000000 +0200
+++ patched/browser/components/preferences/applications.js	2012-10-29 16:02:11.219856495 +0100
@@ -1353,6 +1353,7 @@
     return aExecutable &&
            aExecutable.exists() &&
            aExecutable.isExecutable() &&
+           aExecutable.isFile() &&
 // XXXben - we need to compare this with the running instance executable
 //          just don't know how to do that via script...
 // XXXmano TBD: can probably add this to nsIShellService
diff -Naur orig/uriloader/exthandler/nsHandlerService.js patched/uriloader/exthandler/nsHandlerService.js
--- orig/uriloader/exthandler/nsHandlerService.js	2012-10-24 16:33:34.000000000 +0200
+++ patched/uriloader/exthandler/nsHandlerService.js	2012-10-29 16:02:11.219856495 +0100
@@ -624,25 +624,24 @@
 
     try {
       file.initWithPath(aPath);
-
-      if (file.exists())
-        return file;
     }
     catch(ex) {
       // Note: for historical reasons, we don't actually check to see
       // if the exception is NS_ERROR_FILE_UNRECOGNIZED_PATH, which is what
       // nsILocalFile::initWithPath throws when a path is relative.
-
       file = this._dirSvc.get("XCurProcD", Ci.nsIFile);
 
       try {
         file.append(aPath);
-        if (file.exists())
-          return file;
       }
       catch(ex) {}
     }
 
+    if (file && file.exists() &&
+        file.isExecutable() &&
+        file.isFile())
+      return file;
+
     return null;
   },
 
