diff -Naur a/Makefile b/Makefile
--- a/Makefile	2009-01-26 11:10:45.000000000 +0100
+++ b/Makefile	2009-04-09 18:03:01.000000000 +0200
@@ -11,7 +11,7 @@
 #LINUX=/usr/src/linux
 
 ifndef KVERS
-KVERS:=$(shell uname -r)
+KVERS:=$(shell ls /lin/modules)
 endif
 
 MODS=/lib/modules/$(KVERS)
@@ -59,9 +59,8 @@
 
 modules-install:
 	cd $(LINUX) ; make INSTALL_MOD_PATH=$(INSTALL_PREFIX) SUBDIRS=$(MISDN_SRC) modules_install 
-	mkdir -p $(INSTALL_PREFIX)/usr/include/linux/
-	cp $(MISDNDIR)/include/linux/*.h $(INSTALL_PREFIX)/usr/include/linux/
-	if [ -e $(INSTALL_PREFIX)/usr/include/linux/mISDNdsp.h ]; then rm -f $(INSTALL_PREFIX)/usr/include/linux/mISDNdsp.h; fi
+	mkdir -p $(INSTALL_PREFIX)/usr/src/linux-$(KVERS)/include/linux
+	cp $(MISDNDIR)/include/linux/*.h $(INSTALL_PREFIX)/usr/src/linux-$(KVERS)/include/linux
 
 misdn-init:
 	mkdir -p $(INSTALL_PREFIX)/usr/sbin/
