--- gnome-nettool-2.26.1.orig/src/gnome-nettool.desktop.in.in	2009-03-31 16:10:08.000000000 +0200
+++ gnome-nettool-2.26.1/src/gnome-nettool.desktop.in.in	2009-04-30 19:29:00.000000000 +0200
@@ -1,12 +1,11 @@
 [Desktop Entry]
-Encoding=UTF-8
-_Name=Network Tools
+_Name=GNOME Network Tools
 _Comment=View information about your network
 Exec=gnome-nettool
 Icon=gnome-nettool
 Terminal=false
 Type=Application
-Categories=System;Utility;GNOME;GTK;
+Categories=GTK;GNOME;System;Utility;X-UHU-Menu/Internet/Tools;
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-nettool
