diff -Naur orig/data/gdu-notification-daemon.desktop.in.in.in mod/data/gdu-notification-daemon.desktop.in.in.in
--- orig/data/gdu-notification-daemon.desktop.in.in.in	2011-07-08 16:56:26.000000000 +0200
+++ mod/data/gdu-notification-daemon.desktop.in.in.in	2013-02-12 22:20:39.000000000 +0100
@@ -6,7 +6,7 @@
 Terminal=false
 Type=Application
 Categories=
-OnlyShowIn=GNOME;Unity;
+NotShowIn=KDE;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-disk-utility
 X-GNOME-Bugzilla-Component=notifications
