diff -Naur orig/src/gdu-gtk/gdu-ata-smart-dialog.c mod/src/gdu-gtk/gdu-ata-smart-dialog.c
--- orig/src/gdu-gtk/gdu-ata-smart-dialog.c	2011-07-08 18:07:14.000000000 +0200
+++ mod/src/gdu-gtk/gdu-ata-smart-dialog.c	2014-01-24 17:50:10.000000000 +0100
@@ -1110,7 +1110,7 @@
         test_dialog = gtk_dialog_new_with_buttons (NULL,
                                                    GTK_WINDOW (dialog),
                                                    GTK_DIALOG_MODAL|GTK_DIALOG_DESTROY_WITH_PARENT,
-                                                   NULL);
+                                                   NULL,NULL);
         gtk_window_set_title (GTK_WINDOW (test_dialog), "");
 
 
diff -Naur orig/src/gdu-gtk/gdu-gtk.c mod/src/gdu-gtk/gdu-gtk.c
--- orig/src/gdu-gtk/gdu-gtk.c	2011-07-08 18:07:14.000000000 +0200
+++ mod/src/gdu-gtk/gdu-gtk.c	2014-01-24 17:46:31.000000000 +0100
@@ -346,7 +346,7 @@
         dialog = gtk_dialog_new_with_buttons (window_title,
                                               GTK_WINDOW (parent_window),
                                               GTK_DIALOG_MODAL|GTK_DIALOG_DESTROY_WITH_PARENT,
-                                              NULL);
+                                              NULL,NULL);
         content_area = gtk_dialog_get_content_area (GTK_DIALOG (dialog));
         action_area = gtk_dialog_get_action_area (GTK_DIALOG (dialog));
 
@@ -1162,7 +1162,7 @@
         dialog = gtk_dialog_new_with_buttons (title,
                                               GTK_WINDOW (parent_window),
                                               GTK_DIALOG_MODAL|GTK_DIALOG_DESTROY_WITH_PARENT,
-                                              NULL);
+                                              NULL,NULL);
         content_area = gtk_dialog_get_content_area (GTK_DIALOG (dialog));
         action_area = gtk_dialog_get_action_area (GTK_DIALOG (dialog));
 
