diff -Naur orig/doc/sample.wgetrc patched/doc/sample.wgetrc
--- orig/doc/sample.wgetrc	2011-01-01 13:12:33.000000000 +0100
+++ patched/doc/sample.wgetrc	2011-08-27 17:21:46.814784724 +0200
@@ -7,7 +7,7 @@
 ## not contain a comprehensive list of commands -- look at the manual
 ## to find out what you can put into this file.
 ##
-## Wget initialization file can reside in /usr/local/etc/wgetrc
+## Wget initialization file can reside in /etc/wgetrc
 ## (global, for all users) or $HOME/.wgetrc (for a single user).
 ##
 ## To use the settings in this file, you will have to uncomment them,
@@ -16,7 +16,7 @@
 
 
 ##
-## Global settings (useful for setting up in /usr/local/etc/wgetrc).
+## Global settings (useful for setting up in /etc/wgetrc).
 ## Think well before you change them, since they may reduce wget's
 ## functionality, and make it behave contrary to the documentation:
 ##
diff -Naur orig/doc/sample.wgetrc.munged_for_texi_inclusion patched/doc/sample.wgetrc.munged_for_texi_inclusion
--- orig/doc/sample.wgetrc.munged_for_texi_inclusion	2011-08-09 15:52:17.000000000 +0200
+++ patched/doc/sample.wgetrc.munged_for_texi_inclusion	2011-08-27 17:22:23.022783224 +0200
@@ -7,7 +7,7 @@
 ## not contain a comprehensive list of commands -- look at the manual
 ## to find out what you can put into this file.
 ##
-## Wget initialization file can reside in /usr/local/etc/wgetrc
+## Wget initialization file can reside in /etc/wgetrc
 ## (global, for all users) or $HOME/.wgetrc (for a single user).
 ##
 ## To use the settings in this file, you will have to uncomment them,
@@ -16,7 +16,7 @@
 
 
 ##
-## Global settings (useful for setting up in /usr/local/etc/wgetrc).
+## Global settings (useful for setting up in /etc/wgetrc).
 ## Think well before you change them, since they may reduce wget's
 ## functionality, and make it behave contrary to the documentation:
 ##
diff -Naur orig/doc/wget.info patched/doc/wget.info
--- orig/doc/wget.info	2011-08-09 15:52:17.000000000 +0200
+++ patched/doc/wget.info	2011-08-27 17:23:33.518783876 +0200
@@ -113,7 +113,7 @@
    * Most of the features are fully configurable, either through
      command line options, or via the initialization file `.wgetrc'
      (*note Startup File::).  Wget allows you to define "global"
-     startup files (`/usr/local/etc/wgetrc' by default) for site
+     startup files (`/etc/wgetrc' by default) for site
      settings. You can also specify the location of a startup file with
      the -config option.
 
@@ -2373,8 +2373,8 @@
 ===================
 
 When initializing, Wget will look for a "global" startup file,
-`/usr/local/etc/wgetrc' by default (or some prefix other than
-`/usr/local', if Wget was not installed there) and read commands from
+`/etc/wgetrc' by default (or some prefix other than
+`/usr', if Wget was not installed there) and read commands from
 there, if it exists.
 
    Then it will look for the user's file.  If the environmental variable
@@ -2385,7 +2385,7 @@
 
    The fact that user's settings are loaded after the system-wide ones
 means that in case of collision user's wgetrc _overrides_ the
-system-wide wgetrc (in `/usr/local/etc/wgetrc' by default).  Fascist
+system-wide wgetrc (in `/etc/wgetrc' by default).  Fascist
 admins, away!
 
 
@@ -2910,7 +2910,7 @@
      ## not contain a comprehensive list of commands -- look at the manual
      ## to find out what you can put into this file.
      ##
-     ## Wget initialization file can reside in /usr/local/etc/wgetrc
+     ## Wget initialization file can reside in /etc/wgetrc
      ## (global, for all users) or $HOME/.wgetrc (for a single user).
      ##
      ## To use the settings in this file, you will have to uncomment them,
@@ -2919,7 +2919,7 @@
 
 
      ##
-     ## Global settings (useful for setting up in /usr/local/etc/wgetrc).
+     ## Global settings (useful for setting up in /etc/wgetrc).
      ## Think well before you change them, since they may reduce wget's
      ## functionality, and make it behave contrary to the documentation:
      ##
diff -Naur orig/doc/wget.texi patched/doc/wget.texi
--- orig/doc/wget.texi	2011-08-06 12:22:58.000000000 +0200
+++ patched/doc/wget.texi	2011-08-27 17:24:11.650780528 +0200
@@ -190,14 +190,14 @@
 Most of the features are fully configurable, either through command line
 options, or via the initialization file @file{.wgetrc} (@pxref{Startup
 File}).  Wget allows you to define @dfn{global} startup files
-(@file{/usr/local/etc/wgetrc} by default) for site settings. You can also
+(@file{/etc/wgetrc} by default) for site settings. You can also
 specify the location of a startup file with the --config option.
  
 
 @ignore
 @c man begin FILES
 @table @samp
-@item /usr/local/etc/wgetrc
+@item /etc/wgetrc
 Default location of the @dfn{global} startup file.
 
 @item .wgetrc
@@ -2696,8 +2696,8 @@
 @cindex location of wgetrc
 
 When initializing, Wget will look for a @dfn{global} startup file,
-@file{/usr/local/etc/wgetrc} by default (or some prefix other than
-@file{/usr/local}, if Wget was not installed there) and read commands
+@file{/etc/wgetrc} by default (or some prefix other than
+@file{/usr}, if Wget was not installed there) and read commands
 from there, if it exists.
 
 Then it will look for the user's file.  If the environmental variable
@@ -2708,7 +2708,7 @@
 
 The fact that user's settings are loaded after the system-wide ones
 means that in case of collision user's wgetrc @emph{overrides} the
-system-wide wgetrc (in @file{/usr/local/etc/wgetrc} by default).
+system-wide wgetrc (in @file{/etc/wgetrc} by default).
 Fascist admins, away!
 
 @node Wgetrc Syntax, Wgetrc Commands, Wgetrc Location, Startup File
diff -Naur orig/NEWS patched/NEWS
--- orig/NEWS	2011-08-06 13:04:16.000000000 +0200
+++ patched/NEWS	2011-08-27 17:21:07.922783754 +0200
@@ -625,7 +625,7 @@
 
 ** Compiles on pre-ANSI compilers.
 
-** Global wgetrc now goes to /usr/local/etc (i.e. $sysconfdir).
+** Global wgetrc now goes to /etc (i.e. $sysconfdir).
 
 ** Lots of bugfixes.
 
diff -Naur orig/README patched/README
--- orig/README	2011-01-01 13:12:33.000000000 +0100
+++ patched/README	2011-08-27 17:20:55.186785805 +0200
@@ -33,7 +33,7 @@
 
 Most of the features are configurable, either through command-line
 options, or via initialization file .wgetrc.  Wget allows you to
-install a global startup file (/usr/local/etc/wgetrc by default) for
+install a global startup file (/etc/wgetrc by default) for
 site settings.
 
 Wget works under almost all Unix variants in use today and, unlike
