diff -Naur orig/avidemux2.desktop mod/avidemux2.desktop
--- orig/avidemux2.desktop	2014-03-12 07:15:23.000000000 +0100
+++ mod/avidemux2.desktop	2014-05-03 20:44:26.000000000 +0200
@@ -1,10 +1,9 @@
 [Desktop Entry]
-Encoding=UTF-8
-Name=avidemux2
+Name=avidemux
 Comment=Video Editor
 Exec=avidemux2_gtk
 MimeType=video/mpeg;video/quicktime;video/x-msvideo;video/x-anim;audio/x-mp3;audio/x-mp2;
-Icon=avidemux
+Icon=/usr/share/avidemux6/avidemux_icon.png
 Terminal=false
 Type=Application
-Categories=Application;AudioVideo
+Categories=AudioVideo;AudioVideoEditing;
