diff -Naurdp a/hald/linux/addons/addon-storage.c b/hald/linux/addons/addon-storage.c
--- a/hald/linux/addons/addon-storage.c	2007-04-03 18:38:50.000000000 +0200
+++ b/hald/linux/addons/addon-storage.c	2007-08-12 19:28:38.000000000 +0200
@@ -278,7 +278,7 @@ enum {
 
 static gboolean poll_for_media (gpointer user_data);
 
-static int interval_in_seconds = 2;
+static int interval_in_seconds = 1;
 
 static gboolean is_locked_by_hal = FALSE;
 static gboolean is_locked_via_o_excl = FALSE;
diff -Naurdp a/hald/linux/probing/probe-hiddev.c b/hald/linux/probing/probe-hiddev.c
--- a/hald/linux/probing/probe-hiddev.c	2007-04-03 18:38:50.000000000 +0200
+++ b/hald/linux/probing/probe-hiddev.c	2007-08-12 19:28:06.000000000 +0200
@@ -151,7 +151,7 @@ main (int argc, char *argv[])
 		main2 (ctx, "/org/freedesktop/Hal/devices/usb_device_51d_2_QB0435136106  _if0_hiddev", fd);
 	}
 	else
-		sleep (2);
+		sleep (1);
 #endif
 
 	/* success */
