diff -Naur orig/data/mate-power-preferences.desktop.in.in patched/data/mate-power-preferences.desktop.in.in
--- orig/data/mate-power-preferences.desktop.in.in	2013-03-16 18:33:30.000000000 +0100
+++ patched/data/mate-power-preferences.desktop.in.in	2013-03-30 18:05:56.100458108 +0100
@@ -5,7 +5,7 @@
 Exec=mate-power-preferences
 Terminal=false
 Type=Application
-Categories=Settings;HardwareSettings;
+Categories=Settings;HardwareSettings;X-UHU-Menu/Settings;
 OnlyShowIn=MATE;
 StartupNotify=true
 X-MATE-Bugzilla-Bugzilla=MATE
diff -Naur orig/data/mate-power-statistics.desktop.in.in patched/data/mate-power-statistics.desktop.in.in
--- orig/data/mate-power-statistics.desktop.in.in	2013-03-30 18:06:48.368940532 +0100
+++ patched/data/mate-power-statistics.desktop.in.in	2013-03-16 18:33:30.000000000 +0100
@@ -5,7 +5,7 @@
 Exec=mate-power-statistics
 Terminal=false
 Type=Application
-Categories=GTK;System;Monitor;
+Categories=GTK;System;Monitor;X-UHU-Menu/Tools/Utilities;
 OnlyShowIn=MATE;
 NoDisplay=false
 StartupNotify=true
