diff -Naur orig/lxqt-config-randr.desktop mod/lxqt-config-randr.desktop
--- orig/lxqt-config-randr.desktop	2014-05-07 05:36:35.000000000 +0200
+++ mod/lxqt-config-randr.desktop	2014-07-11 23:45:54.000000000 +0200
@@ -5,7 +5,6 @@
 Comment=Configure your monitors
 Exec=lxqt-config-randr
 Icon=preferences-desktop-display
-Categories=Settings;DesktopSettings;Qt;LXQt;
-OnlyShowIn=LXQt;
+Categories=Settings;DesktopSettings;Qt;
 
 #TRANSLATIONS_DIR=translations
