diff -Naur orig/test/desktop/sudoku.desktop mod/test/desktop/sudoku.desktop
--- orig/test/desktop/sudoku.desktop	2008-03-16 13:20:30.000000000 +0100
+++ mod/test/desktop/sudoku.desktop	2014-05-08 21:33:53.000000000 +0200
@@ -1,6 +1,8 @@
 [Desktop Entry]
-Name=Sudoku
+Name=Sudoku Game
+Name[hu]=Sudoku játék
 Comment=Suduku Number Game
+Comment[hu]=Sudoku számkitöltő
 TryExec=sudoku
 Exec=sudoku
 Icon=sudoku
