diff -Naur orig/test/desktop/blocks.desktop mod/test/desktop/blocks.desktop
--- orig/test/desktop/blocks.desktop	2008-03-16 13:20:30.000000000 +0100
+++ mod/test/desktop/blocks.desktop	2009-04-02 08:21:05.000000000 +0200
@@ -1,9 +1,11 @@
 [Desktop Entry]
+Encoding=UTF-8
 Name=Block Attack!
 Comment=Block Attack! Game
+Comment[hu]=Blokk támadás!
 TryExec=blocks
 Exec=blocks
 Icon=blocks
 Terminal=false
 Type=Application
-Categories=Game;BlocksGame;
+Categories=Game;BlocksGame;X-UHU-Menu/Games/Action;
