diff -Naur orig/test/desktop/checkers.desktop mod/test/desktop/checkers.desktop
--- orig/test/desktop/checkers.desktop	2008-03-16 13:20:30.000000000 +0100
+++ mod/test/desktop/checkers.desktop	2009-04-01 12:17:20.000000000 +0200
@@ -1,4 +1,6 @@
 [Desktop Entry]
+Encoding=UTF-8
+Name[hu]=Dáma játék
 Name=Checkers
 Comment=Checkers Game
 TryExec=checkers
@@ -6,4 +8,4 @@
 Icon=checkers
 Terminal=false
 Type=Application
-Categories=Game;BoardGame;
+Categories=Game;BoardGame;X-UHU-Menu/Games/Board;
