diff -Naurd a/avahi-daemon/avahi-dbus.conf b/avahi-daemon/avahi-dbus.conf
--- a/avahi-daemon/avahi-dbus.conf	2007-02-04 20:52:36.000000000 +0100
+++ b/avahi-daemon/avahi-dbus.conf	2007-03-23 16:07:39.000000000 +0100
@@ -19,9 +19,4 @@
     <deny send_interface="org.freedesktop.Avahi.Server" send_member="SetHostName"/>
   </policy>
 
-  <!-- Allow everything, including access to SetHostName to users of the group "netdev" -->
-  <policy group="netdev">
-    <allow send_destination="org.freedesktop.Avahi"/>
-    <allow receive_sender="org.freedesktop.Avahi"/>
-  </policy>
 </busconfig>
