diff -Naur scim-1.4.9-orig//extras/setup/scim-setup.desktop.in scim-1.4.9//extras/setup/scim-setup.desktop.in
--- scim-1.4.9-orig//extras/setup/scim-setup.desktop.in	2008-11-02 07:43:11.000000000 +0100
+++ scim-1.4.9//extras/setup/scim-setup.desktop.in	2010-06-25 21:52:04.000000000 +0200
@@ -7,4 +7,4 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=Applications;Settings;
+Categories=Applications;Settings;X-UHU-Menu/Settings;
