diff -Naur orig/lxqt-config-appearance/lxqt-config-appearance.desktop.in mod/lxqt-config-appearance/lxqt-config-appearance.desktop.in
--- orig/lxqt-config-appearance/lxqt-config-appearance.desktop.in	2014-05-07 10:13:55.000000000 +0200
+++ mod/lxqt-config-appearance/lxqt-config-appearance.desktop.in	2014-07-11 23:10:16.000000000 +0200
@@ -5,7 +5,6 @@
 Comment=Appearance settings for LXQt
 Exec=lxqt-config-appearance
 Icon=preferences-desktop-theme
-Categories=Settings;DesktopSettings;Qt;LXQt;
-OnlyShowIn=LXDE;LXQt;
+Categories=Settings;DesktopSettings;Qt;
 
 #TRANSLATIONS_DIR=translations
diff -Naur orig/lxqt-config-file-associations/lxqt-config-file-associations.desktop.in mod/lxqt-config-file-associations/lxqt-config-file-associations.desktop.in
--- orig/lxqt-config-file-associations/lxqt-config-file-associations.desktop.in	2014-05-07 10:13:55.000000000 +0200
+++ mod/lxqt-config-file-associations/lxqt-config-file-associations.desktop.in	2014-07-11 23:10:34.000000000 +0200
@@ -5,7 +5,6 @@
 Comment=Configure applications associated with known file types
 Exec=lxqt-config-file-associations
 Icon=preferences-desktop-filetype-association
-Categories=Settings;DesktopSettings;Qt;LXQt;
-OnlyShowIn=LXDE;LXQt;
+Categories=Settings;DesktopSettings;Qt;
 
 #TRANSLATIONS_DIR=translations
diff -Naur orig/lxqt-config-input/lxqt-config-input.desktop.in mod/lxqt-config-input/lxqt-config-input.desktop.in
--- orig/lxqt-config-input/lxqt-config-input.desktop.in	2014-05-07 10:13:55.000000000 +0200
+++ mod/lxqt-config-input/lxqt-config-input.desktop.in	2014-07-11 23:10:51.000000000 +0200
@@ -5,7 +5,6 @@
 Comment=Configure keyboard, mouse, and other input devices
 Exec=lxqt-config-input
 Icon=input-keyboard
-Categories=Qt;Settings;HardwareSettings;DesktopSettings;LXQt;
-OnlyShowIn=LXDE;LXQt;
+Categories=Qt;Settings;HardwareSettings;DesktopSettings;
 
 #TRANSLATIONS_DIR=translations
diff -Naur orig/lxqt-config-qtconfig/lxqt-config-qtconfig.desktop.in mod/lxqt-config-qtconfig/lxqt-config-qtconfig.desktop.in
--- orig/lxqt-config-qtconfig/lxqt-config-qtconfig.desktop.in	2014-05-07 10:13:55.000000000 +0200
+++ mod/lxqt-config-qtconfig/lxqt-config-qtconfig.desktop.in	2014-07-11 23:11:07.000000000 +0200
@@ -6,7 +6,6 @@
 Exec=qtconfig
 TryExec=qtconfig
 Icon=preferences-system
-Categories=Settings;DesktopSettings;Qt;LXQt;
-OnlyShowIn=LXQt;
+Categories=Settings;DesktopSettings;Qt;
 
 #TRANSLATIONS_DIR=translations
diff -Naur orig/src/lxqt-config.desktop.in mod/src/lxqt-config.desktop.in
--- orig/src/lxqt-config.desktop.in	2014-05-07 10:13:55.000000000 +0200
+++ mod/src/lxqt-config.desktop.in	2014-07-11 23:13:20.000000000 +0200
@@ -5,7 +5,6 @@
 Comment=Configure your system
 Exec=lxqt-config
 Icon=preferences-system
-Categories=Settings;DesktopSettings;Qt;LXQt;
-OnlyShowIn=LXDE;LXQt;
+Categories=Settings;DesktopSettings;Qt;
 
 #TRANSLATIONS_DIR=translations
