diff -Naurd a/src/init.c b/src/init.c
--- a/src/init.c	2004-09-02 13:29:53.000000000 +0200
+++ b/src/init.c	2004-09-02 14:07:09.959869534 +0200
@@ -453,6 +453,18 @@
 	else {
 		console_dev = CONSOLE;
 		tried_devcons++;
+		if ((fd = open(console_dev, O_RDONLY|O_NONBLOCK)) < 0) {
+			mode_t old_umask;
+			old_umask = umask(0);
+			mount("none", "/dev", "ramfs", 0, NULL);
+			mknod("/dev/null", 0666|S_IFCHR, 259);
+			mknod("/dev/zero", 0666|S_IFCHR, 261);
+			mknod("/dev/tty", 0666|S_IFCHR, 1280);
+			mknod("/dev/console", 0600|S_IFCHR, 1281);
+			umask(old_umask);
+		} else {
+			close (fd);
+		}
 	}
 
 	while ((fd = open(console_dev, O_RDONLY|O_NONBLOCK)) < 0) {
@@ -2245,6 +2257,7 @@
 #endif
 	close(pipe_fd);
 	pipe_fd = -1;
+	console_init();
 	DELSET(got_signals, SIGUSR1);
   }
 }
