diff -Naur fwbackups-1.43.4-orig/src/fwbackups-autostart.desktop fwbackups-1.43.4/src/fwbackups-autostart.desktop
--- fwbackups-1.43.4-orig/src/fwbackups-autostart.desktop	2010-11-28 04:21:03.000000000 +0100
+++ fwbackups-1.43.4/src/fwbackups-autostart.desktop	2011-02-02 19:18:13.386543542 +0100
@@ -2,7 +2,9 @@
 Name=fwbackups
 Comment=Backup or restore your files
 Name[en_US]=fwbackups
+Name[hu]=fwbackups
 Comment[en_US]=Backup or restore your files
+Comment[hu]=File-ok mentése/visszatöltése
 Exec=/usr/bin/fwbackups --start-minimized
 Type=Application
 Version=1.0
diff -Naur fwbackups-1.43.4-orig/src/fwbackups.desktop fwbackups-1.43.4/src/fwbackups.desktop
--- fwbackups-1.43.4-orig/src/fwbackups.desktop	2010-11-28 04:21:03.000000000 +0100
+++ fwbackups-1.43.4/src/fwbackups.desktop	2011-02-02 19:16:40.286542717 +0100
@@ -1,11 +1,13 @@
 [Desktop Entry]
 Name=fwbackups
+Name[hu]=fwbackups
 GenericName=Backup/Restore
 Comment=Backup or restore your files
+Comment[hu]=File-ok mentése/visszatöltése
 Exec=fwbackups
 Icon=fwbackups
 Terminal=false
 Type=Application
-Categories=GTK;System;
+Categories=GTK;System;X-UHU-Menu/Tools/Utilities;
 #MimeType=text/plain
 Version=1.0
