diff -Naur orig/data/org.mate.session.gschema.xml.in.in patched/data/org.mate.session.gschema.xml.in.in
--- orig/data/org.mate.session.gschema.xml.in.in	2013-06-30 21:46:09.000000000 +0200
+++ patched/data/org.mate.session.gschema.xml.in.in	2014-02-28 22:32:43.092729846 +0100
@@ -44,7 +44,7 @@
       <_description>The panel provides the bar at the top or bottom of the screen containing menus, the window list, status icons, the clock, etc.</_description>
     </key>
     <key name="filemanager" type="s">
-      <default>'caja'</default>
+      <default>'caja-desktop'</default>
       <_summary>File Manager</_summary>
       <_description>The file manager provides the desktop icons and allows you to interact with your saved files.</_description>
     </key>
