diff -Naur orig/mate-session/gsm-logout-dialog.c patched/mate-session/gsm-logout-dialog.c
--- orig/mate-session/gsm-logout-dialog.c	2013-02-08 00:02:44.000000000 +0100
+++ patched/mate-session/gsm-logout-dialog.c	2013-03-16 15:00:24.088754756 +0100
@@ -252,14 +252,7 @@
                 return FALSE;
         }
 
-        if (logout_dialog->priv->timeout <= 30) {
-                seconds_to_show = logout_dialog->priv->timeout;
-        } else {
-                seconds_to_show = (logout_dialog->priv->timeout/10) * 10;
-
-                if (logout_dialog->priv->timeout % 10)
-                        seconds_to_show += 10;
-        }
+        seconds_to_show = logout_dialog->priv->timeout;
 
         switch (logout_dialog->priv->type) {
         case GSM_DIALOG_LOGOUT_TYPE_LOGOUT:
