diff -Naur kdepimlibs-4.9.5-ori/cmake/modules/FindGpgme.cmake kdepimlibs-4.9.5/cmake/modules/FindGpgme.cmake
--- kdepimlibs-4.9.5-ori/cmake/modules/FindGpgme.cmake	2012-11-30 08:19:06.000000000 +0100
+++ kdepimlibs-4.9.5/cmake/modules/FindGpgme.cmake	2013-01-25 19:28:38.651278404 +0100
@@ -226,10 +226,9 @@
 	  set( _gpgme_config_pthread_libs )
 	endif()
 
-        exec_program( ${_GPGMECONFIG_EXECUTABLE} ARGS --thread=pth     --libs OUTPUT_VARIABLE _gpgme_config_pth_libs     RETURN_VALUE _ret )
-	if ( _ret )
+
+     # disable pth	
 	  set( _gpgme_config_pth_libs )
-	endif()
 
         # append -lgpg-error to the list of libraries, if necessary
         foreach ( _flavour vanilla pthread pth )
