diff -Naurdp a/lib/Makefile b/lib/Makefile
--- a/lib/Makefile	2006-08-17 15:57:31.000000000 +0200
+++ b/lib/Makefile	2007-02-27 17:39:49.000000000 +0100
@@ -58,7 +58,7 @@ distclean:
 
 install:
 	mkdir -p $(DESTDIR)$(libdir)/
-	install -m 0644 $(OUT_SLIB) $(DESTDIR)$(libdir)
+	install -m 0755 $(OUT_SLIB) $(DESTDIR)$(libdir)
 	rm -f $(DESTDIR)$(libdir)/$(LN1_SLIB)
 	$(LN) -s $(OUT_SLIB) $(DESTDIR)$(libdir)/$(LN1_SLIB)
 	rm -f $(DESTDIR)$(libdir)/$(LN_SLIB)
