diff -Naur orig/claws-mail.desktop patched/claws-mail.desktop
--- orig/claws-mail.desktop	2012-10-19 08:30:40.000000000 +0200
+++ patched/claws-mail.desktop	2012-11-15 11:30:48.299377318 +0100
@@ -17,7 +17,7 @@
 GenericName[sk]=Poštový klient
 Exec=claws-mail %u
 Icon=claws-mail
-Categories=Network;Office;Email;
+Categories=Network;Office;Email;X-UHU-Menu/Internet/Email;
 Comment=Lightweight and Fast GTK+ based Mail Client
 Comment[ca]=Client de correu electrònic ràpid i lleuger basat en GTK+
 Comment[cs]=Lehký a rychlý mailový klient založený na GTK+
