diff -Naur orig/src/plugins/rssyl/libfeed/date.c patched/src/plugins/rssyl/libfeed/date.c
--- orig/src/plugins/rssyl/libfeed/date.c	2014-10-27 19:58:16.000000000 +0100
+++ patched/src/plugins/rssyl/libfeed/date.c	2015-01-09 15:05:01.138478852 +0100
@@ -132,8 +132,11 @@
 	if (c != NULL)
 		setlocale(LC_TIME, "C");
 
-	if (!strptime(date, "%a, %d %b %Y %H:%M:%S %Z", &t) &&
-		!strptime(date, "%a, %d %b %Y %H:%M %Z", &t)) {
+	if (date != NULL &&
+			!strptime(date, "%a, %d %b %Y %H:%M:%S %Z", &t) &&
+			!strptime(date, "%a, %d %b %Y %H:%M %Z", &t) &&
+			!strptime(date, "%d %b %Y %H:%M:%S %Z", &t) &&
+			!strptime(date, "%d %b %Y %H:%M %Z", &t)) {
 		g_warning("Invalid RFC822 date!\n");
 		if (c != NULL)
 			setlocale(LC_TIME, c);
