diff -Naur orig/wireshark.desktop mod/wireshark.desktop
--- orig/wireshark.desktop	2012-06-05 18:33:42.000000000 +0200
+++ mod/wireshark.desktop	2017-01-31 09:47:43.526149712 +0100
@@ -68,7 +68,7 @@
 Comment[fi]=Verkkoliikenne analysaattori
 Comment[fr]=Analyseur de trafic réseau
 Comment[sv]=Nätverkstrafikanalysator
-Icon=wireshark.png
+Icon=wireshark
 TryExec=wireshark
 Exec=wireshark %f
 Terminal=false
