diff -Naur a/src/gui/Unix/QMPlay2.desktop b/src/gui/Unix/QMPlay2.desktop
--- a/src/gui/Unix/QMPlay2.desktop	2013-05-19 16:12:43.000000000 +0200
+++ b/src/gui/Unix/QMPlay2.desktop	2014-01-03 02:45:56.218556882 +0100
@@ -4,5 +4,5 @@
 Name=QMPlay2
 StartupNotify=false
 Type=Application
-Categories=Qt;AudioVideo;Player;Audio;Video;
+Categories=Qt;AudioVideo;Player;Audio;Video;X-UHU-Menu/Multimedia/VideoPlayers;
 MimeType=video/mp4;video/mpeg;video/ogg;video/quicktime;video/x-msvideo;video/x-flv;application/x-shockwave-flash;application/vnd.rn-realmedia;video/vnd.rn-realvideo;video/3gpp;video/3gpp2;video/dv;video;mp2t;video/webm;video/x-matroska;application/x-matroska;video/x-ms-asf;video/x-ms-wmv;video/x-ogm+ogg;video/x-theora+ogg;video/mp2t;audio/x-ape;audio/ac3;audio/mp4;audio/AMR;audio/aac;audio/flac;audio/webm;audio/x-matroska;audio/ac3;audio/flac;audio/mp2;audio/ogg;audio/x-flac+ogg;audio/x-aiff;audio/x-wav;audio/x-vorbis+ogg;audio/x-tta;audio/x-wavpack;audio/x-musepack;audio/x-ms-wma;audio/x-ms-asx;audio/x-mod;audio/x-s3m;audio/x-stm;audio/x-xi;audio/x-xm;audio/mpeg;audio/j2b;audio/umx;
diff -Naur a/src/gui/Unix/QMPlay2_enqueue.desktop b/src/gui/Unix/QMPlay2_enqueue.desktop
--- a/src/gui/Unix/QMPlay2_enqueue.desktop	2013-05-19 16:12:51.000000000 +0200
+++ b/src/gui/Unix/QMPlay2_enqueue.desktop	2014-01-03 02:45:57.330556916 +0100
@@ -5,6 +5,6 @@
 Name[pl]=Kolejkuj w QMPlay2
 StartupNotify=false
 Type=Application
-Categories=Qt;AudioVideo;Player;Audio;Video;
+Categories=Qt;AudioVideo;Player;Audio;Video;X-UHU-Menu/Multimedia/VideoPlayers;
 MimeType=video/mp4;video/mpeg;video/ogg;video/quicktime;video/x-msvideo;video/x-flv;application/x-shockwave-flash;application/vnd.rn-realmedia;video/vnd.rn-realvideo;video/3gpp;video/3gpp2;video/dv;video;mp2t;video/webm;video/x-matroska;application/x-matroska;video/x-ms-asf;video/x-ms-wmv;video/x-ogm+ogg;video/x-theora+ogg;video/mp2t;audio/x-ape;audio/ac3;audio/mp4;audio/AMR;audio/aac;audio/flac;audio/webm;audio/x-matroska;audio/ac3;audio/flac;audio/mp2;audio/ogg;audio/x-flac+ogg;audio/x-aiff;audio/x-wav;audio/x-vorbis+ogg;audio/x-tta;audio/x-wavpack;audio/x-musepack;audio/x-ms-wma;audio/x-ms-asx;audio/x-mod;audio/x-s3m;audio/x-stm;audio/x-xi;audio/x-xm;audio/mpeg;audio/j2b;audio/umx;
 NoDisplay=true
