diff -Naur orig/fluxbox-xdg-menu.py mod/fluxbox-xdg-menu.py
--- orig/fluxbox-xdg-menu.py	2014-12-22 21:14:13.000000000 +0100
+++ mod/fluxbox-xdg-menu.py	2014-12-23 20:17:14.030439463 +0100
@@ -29,12 +29,14 @@
 """
 
 __author__ = "Rudolf Kastl , Antonio Gomes, Michael Rice"
-__version__ = "$Revision: 1.2 $"
-__date__ = "$Date: 2006/10/09 23:20:10 $"
+__version__ = "$Revision: 13 $"
+__date__ = "$Date: 2010/01/20 21:50:10 $"
 __license__ = "GPL"
 
 
 import os,re,sys,glob,getopt
+reload(sys)
+sys.setdefaultencoding('utf-8')
 import xdg.Menu,xdg.DesktopEntry,xdg.IconTheme
 from os.path import isfile
 
@@ -44,10 +46,7 @@
 def header(wm="fluxbox"):
 	return """
 [begin] (Fluxbox)
-	[exec] (Web Browser) {htmlview}
-	[exec] (Email) {evolution}
-	[exec] (Terminal) {$TERM}
-	[exec] (Irc) {xchat}
+	[exec] (Terminal) {xterm} </usr/share/pixmaps/xterm-color_48x48.xpm>
 	[separator]\n"""
 
 def footer(wm="fluxbox"):
@@ -67,7 +66,7 @@
 			[exec] (Screenshot - JPG) {import screenshot.jpg && display -resize 50% screenshot.jpg}
 			[exec] (Screenshot - PNG) {import screenshot.png && display -resize 50% screenshot.png}
 			[exec] (Run) {fbrun }
-			[exec] (Regen Menu) {fluxbox-generate_menu --with-icons}
+			[exec] (Regen Menu) {fluxbox-xdg-menu --with-icons}
 		[end]
 		[submenu] (Window)
 			[restart] (kde) {startkde}
@@ -98,7 +97,7 @@
 		e.g. gaim --> /usr/share/pixmaps/gaim.png
 		e.g. fart.png --> /usr/share/pixmaps/fart.png
 	"""
-	retval=str(xdg.IconTheme.getIconPath(icon, 48, theme))
+	retval=str(xdg.IconTheme.getIconPath(icon, 48, theme,["png","xpm"]))
 	if retval == "None":
 		retval=""
 
@@ -236,8 +235,8 @@
 		fsock = open(file,'w')
 		saveout = sys.stdout
 		sys.stdout = fsock
-
-	menu=xdg.Menu.parse()
+	menufile="lxde-applications.menu"
+	menu=xdg.Menu.parse(menufile)
 # is done automatically now
 #	menu.setLocale(lang)
 
