diff -Naur gdm-2.19.6.orig/gui/gdmflexiserver.desktop.in gdm-2.19.6/gui/gdmflexiserver.desktop.in
--- gdm-2.19.6.orig/gui/gdmflexiserver.desktop.in	2007-08-15 20:14:17.000000000 +0200
+++ gdm-2.19.6/gui/gdmflexiserver.desktop.in	2007-08-23 17:03:00.000000000 +0200
@@ -7,7 +7,7 @@
 Icon=gdm
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;System;
+Categories=GNOME;GTK;System;X-UHU-Menu/Tools/Utilities;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gdm
 X-GNOME-Bugzilla-Component=general
diff -Naur gdm-2.19.6.orig/gui/gdmflexiserver-xnest.desktop.in.in gdm-2.19.6/gui/gdmflexiserver-xnest.desktop.in.in
--- gdm-2.19.6.orig/gui/gdmflexiserver-xnest.desktop.in.in	2007-08-15 20:14:17.000000000 +0200
+++ gdm-2.19.6/gui/gdmflexiserver-xnest.desktop.in.in	2007-08-23 17:03:04.000000000 +0200
@@ -8,7 +8,7 @@
 Icon=gdm-xnest
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;System;
+Categories=GNOME;GTK;System;X-UHU-Menu/Tools/Utilities;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gdm
 X-GNOME-Bugzilla-Component=general
diff -Naur gdm-2.19.6.orig/gui/gdmphotosetup.desktop.in gdm-2.19.6/gui/gdmphotosetup.desktop.in
--- gdm-2.19.6.orig/gui/gdmphotosetup.desktop.in	2007-08-15 20:14:17.000000000 +0200
+++ gdm-2.19.6/gui/gdmphotosetup.desktop.in	2007-08-23 17:03:41.000000000 +0200
@@ -9,8 +9,7 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-NotShowIn=GNOME;
-Categories=GNOME;GTK;Settings;
+Categories=GNOME;GTK;Settings;X-UHU-Menu/Settings;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gdm
 X-GNOME-Bugzilla-Component=general
diff -Naur gdm-2.19.6.orig/gui/gdmsetup.desktop.in.in gdm-2.19.6/gui/gdmsetup.desktop.in.in
--- gdm-2.19.6.orig/gui/gdmsetup.desktop.in.in	2007-08-15 20:14:17.000000000 +0200
+++ gdm-2.19.6/gui/gdmsetup.desktop.in.in	2007-08-23 17:04:14.000000000 +0200
@@ -2,13 +2,13 @@
 Encoding=UTF-8
 _Name=Login Window
 _Comment=Configure GDM login window appearance and behavior
-TryExec=@sbindir@/gdmsetup
-Exec=@sbindir@/gdmsetup
+TryExec=gdmsetup
+Exec=gdmsetup
 Icon=gdm-setup
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Settings;System;
+Categories=GNOME;GTK;Settings;System;X-UHU-Menu/Settings;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gdm
 X-GNOME-Bugzilla-Component=general
