diff -Naur SOAP-Lite-0.713.orig/Makefile.PL SOAP-Lite-0.713/Makefile.PL
--- SOAP-Lite-0.713.orig/Makefile.PL	2010-06-03 17:33:30.000000000 +0200
+++ SOAP-Lite-0.713/Makefile.PL	2015-12-01 14:53:05.383426614 +0100
@@ -31,20 +31,20 @@
         },
         2
     ],
-    ["Client HTTPS support","SOAP::Transport::HTTPS::Client",{"Crypt::SSLeay" => 0},0],
+    ["Client HTTPS support","SOAP::Transport::HTTPS::Client",{"Crypt::SSLeay" => 0},1],
     ["Client SMTP/sendmail support","SOAP::Transport::MAILTO::Client",{"MIME::Lite" => 0},1],
     ["Client FTP support","SOAP::Transport::FTP::Client",{"SOAP::Transport::FTP" => 0.712},0],
     ["Standalone HTTP server","SOAP::Transport::HTTP::Daemon",{"HTTP::Daemon" => 0},1],
     ["Apache/mod_perl server","SOAP::Transport::HTTP::Apache",{"Apache" => 0},0],
     ["FastCGI server","SOAP::Transport::HTTP::FCGI",{"FCGI" => 0},0],
     ["POP3 server","SOAP::Transport::POP3::Server",{"Net::POP3" => 0,"MIME::Parser" => 0},1],
-    ["IO server","SOAP::Transport::IO::Server",{"IO::File" => 0},0],
+    ["IO server","SOAP::Transport::IO::Server",{"IO::File" => 0},1],
     ["MQ transport support","SOAP::Transport::MQ",{"SOAP::Transport::MQ" => 0.712},1],
     ["JABBER transport support","SOAP::Transport::JABBER",{"SOAP::Transport::JABBER" => 0.712},0],
     ["MIME messages","SOAP::Packager::MIME",{"MIME::Parser" => 0},1],
     ["DIME messages","SOAP::Packager::DIME",{"IO::Scalar" => "2.105", "DIME::Tools" => 0.03, "Data::UUID" => "0.11"},0],
-    ["SSL Support for TCP Transport","SOAP::Transport::TCP",{"IO::Socket::SSL" => 0},0],
-    ["Compression support for HTTP","SOAP::Transport::HTTP",{"Compress::Zlib" => 0},0],
+    ["SSL Support for TCP Transport","SOAP::Transport::TCP",{"IO::Socket::SSL" => 0},1],
+    ["Compression support for HTTP","SOAP::Transport::HTTP",{"Compress::Zlib" => 0},1],
     ["MIME interoperability w/ Axis","SOAP::Lite",{"MIME::Parser" => "6.106"},0],
 );
 
