diff -Naur a/dist/vatsinator.desktop.in b/dist/vatsinator.desktop.in
--- a/dist/vatsinator.desktop.in	2013-11-04 12:39:06.000000000 +0100
+++ b/dist/vatsinator.desktop.in	2013-11-04 18:18:51.446564940 +0100
@@ -6,7 +6,7 @@
 GenericName[pl]=Monitor sieci Vatsim
 Comment=A simple Vatsim monitor
 Comment[pl]=Monitor sieci Vatsim
-TryExec=${CMAKE_INSTALL_PREFIX}/bin/vatsinator
-Exec=${CMAKE_INSTALL_PREFIX}/bin/vatsinator
-Categories=Network;Qt;
+TryExec=vatsinator
+Exec=vatsinator
+Categories=Network;Qt;X-UHU-Menu/Internet;
 Icon=vatsinator
diff -Naur a/dist/vatsinator-desktop.sh b/dist/vatsinator-desktop.sh
--- a/dist/vatsinator-desktop.sh	2013-11-04 12:39:06.000000000 +0100
+++ b/dist/vatsinator-desktop.sh	2013-11-04 18:18:53.070564895 +0100
@@ -8,8 +8,8 @@
 GenericName[pl]=Monitor sieci Vatsim
 Comment=A simple Vatsim monitor
 Comment[pl]=Monitor sieci Vatsim
-TryExec=$1/bin/vatsinator
-Exec=$1/bin/vatsinator
-Categories=Network;Qt;
+TryExec=vatsinator
+Exec=vatsinator
+Categories=Network;Qt;X-UHU-Menu/Internet;
 Icon=vatsinator
 EOF
