diff -urN parted-2.2.orig/configure parted-2.2/configure
--- parted-2.2.orig/configure	2010-02-26 15:25:45.000000000 +0100
+++ parted-2.2/configure	2010-09-06 22:26:27.000000000 +0200
@@ -13735,7 +13735,6 @@
      # native cc issues annoying warnings and then ignores it,
      # which would cause us to incorrectly conclude that it worked.
      for gl_flags in \
-        '-Wl,--as-needed' \
         '-Wl,-z,ignore' \
         '-z ignore'
      do
