diff -Naur orig/etc/mplayer.desktop mod/etc/mplayer.desktop
--- orig/etc/mplayer.desktop	2012-05-22 00:56:37.000000000 +0200
+++ mod/etc/mplayer.desktop	2014-05-29 13:08:51.204310241 +0200
@@ -7,6 +7,7 @@
 GenericName[es]=Reproductor multimedia
 GenericName[fr]=Lecteur multimédia
 GenericName[it]=Lettore multimediale
+GenericName[hu]=Média lejátszó
 GenericName[ja]=メディアプレーヤー
 X-GNOME-FullName=MPlayer Media Player
 X-GNOME-FullName[ca]=MPlayer Reproductor multimèdia
@@ -21,10 +22,11 @@
 Comment[es]=Reproduce vídeos y música
 Comment[fr]=Lit les films et musiques
 Comment[it]=Riproduce filmati e musica
+Comment[hu]=Filmek és zenék lejátszása
 Comment[ja]=動画や音声のファイルを再生します
 Icon=mplayer
 TryExec=gmplayer
 Exec=gmplayer %F
 Terminal=false
-Categories=GTK;AudioVideo;Audio;Video;Player;TV;
+Categories=GTK;AudioVideo;Video;Player;
 MimeType=application/mxf;application/x-netshow-channel;application/ogg;application/ram;application/vnd.rn-realmedia;application/x-shockwave-flash;application/smil;audio/ac3;audio/x-adpcm;audio/x-aiff;audio/AMR;audio/AMR-WB;audio/x-ape;audio/basic;audio/flac;audio/x-flac;audio/x-flac+ogg;audio/x-m4b;audio/x-matroska;audio/mp2;audio/mp4;audio/mpeg;audio/x-mpegurl;audio/x-ms-asx;audio/x-ms-wma;audio/x-musepack;audio/ogg;audio/vnd.rn-realaudio;audio/x-scpls;audio/x-voc;audio/x-vorbis+ogg;audio/x-wav;audio/x-wavpack;video/3gpp;video/3gpp2;video/dv;video/x-flic;video/x-flv;video/x-matroska;video/mp2t;video/mp4;video/mpeg;video/x-ms-asf;video/x-ms-wmv;video/x-msvideo;video/x-nsv;video/ogg;video/x-ogm+ogg;video/quicktime;video/vnd.rn-realvideo;video/x-theora+ogg;video/webm;
