diff -Naur orig/ffmpeg/libavcodec/libgsm.c patched/ffmpeg/libavcodec/libgsm.c
--- orig/ffmpeg/libavcodec/libgsm.c	2012-05-28 18:42:56.000000000 +0200
+++ patched/ffmpeg/libavcodec/libgsm.c	2012-05-28 18:54:32.785282885 +0200
@@ -27,7 +27,7 @@
 
 // The idiosyncrasies of GSM-in-WAV are explained at http://kbs.cs.tu-berlin.de/~jutta/toast.html
 
-#include <gsm/gsm.h>
+#include <gsm.h>
 
 #include "avcodec.h"
 #include "internal.h"
