diff -urN emelfm2-0.1.6.orig/docs/desktop_environment/emelfm2.desktop emelfm2-0.1.6/docs/desktop_environment/emelfm2.desktop
--- emelfm2-0.1.6.orig/docs/desktop_environment/emelfm2.desktop	2005-11-28 22:29:43.000000000 +0100
+++ emelfm2-0.1.6/docs/desktop_environment/emelfm2.desktop	2006-03-30 18:29:14.000000000 +0200
@@ -6,8 +6,9 @@
 GenericName=file manager
 Comment=2-pane Gtk+2 file manager
+Comment[hu]=Kétpaneles fájlkezelő
 TryExec=emelfm2
 Exec=emelfm2
-Icon=emelfm2
+Icon=/usr/share/pixmaps/emelfm2/emelfm2_48.png
 Terminal=false
 StartupNotify=false
-Categories=System;FileTools;GTK;FileManager;
+Categories=System;FileTools;GTK;FileManager;X-UHU-Menu/Tools/File_Managers;
