diff -Naur orig/Makefile patched/Makefile
--- orig/Makefile	2014-01-24 22:07:36.000000000 +0100
+++ patched/Makefile	2014-04-13 11:45:46.146572935 +0200
@@ -421,7 +421,7 @@
 
 $(TARGET): $(OBJECTS)
 	@echo "linking binary '$(TARGET)'"
-	@$(CC) $(LDFLAGS) $(OBJECTS) -o $(TARGET) -Wl,--as-needed $(lLIBS)
+	@$(CC) $(LDFLAGS) $(OBJECTS) -o $(TARGET) -lm -ldl -Wl,--as-needed $(lLIBS)
 ifeq ($(STRIP), 1)
 ifneq ($(DEBUG), 1)
 	@echo "stripping binary '$(TARGET)'"
