diff -Naur orig/po/emelfm2.desktop.in patched/po/emelfm2.desktop.in
--- orig/po/emelfm2.desktop.in	2013-12-01 22:05:38.000000000 +0100
+++ patched/po/emelfm2.desktop.in	2014-04-13 11:49:25.808600350 +0200
@@ -5,6 +5,7 @@
 GenericName=file manager
 Comment=2-pane file manager
 Comment[fr]=2-gestionnaire de fichiers à panneaux
+Comment[hu]=Kétpaneles fájlkezelő
 Comment[de]=2-Fenster Dateimanager
 Comment[ja]=2-ペイン ファイル manager
 Comment[ru]=2-панельный файловый менеджер
@@ -16,5 +17,5 @@
 Icon=emelfm2
 Terminal=false
 StartupNotify=false
-Categories=System;Utility;Core;FileTools;FileManager;GTK;
+Categories=System;FileTools;FileManager;GTK;
 MimeType=inode/directory;inode/mount-point;
