diff -Naur orig/configure patched/configure
--- orig/configure	2015-05-22 17:45:36.000000000 +0200
+++ patched/configure	2015-05-24 14:10:38.870840389 +0200
@@ -4805,12 +4805,6 @@
 ac_compiler_gnu=$ac_cv_cxx_compiler_gnu
 
 
-ac_fn_cxx_check_header_mongrel "$LINENO" "gtest/gtest.h" "ac_cv_header_gtest_gtest_h" "$ac_includes_default"
-if test "x$ac_cv_header_gtest_gtest_h" = xyes; then :
-
-else
-  as_fn_error $? "failed: I need gtest to build tests" "$LINENO" 5
-fi
 
 
 ac_ext=c
diff -Naur orig/configure.ac patched/configure.ac
--- orig/configure.ac	2015-05-22 17:45:35.000000000 +0200
+++ patched/configure.ac	2015-05-24 14:10:17.102732447 +0200
@@ -89,12 +89,6 @@
      AC_MSG_ERROR([failed: I need CURL due https support]),
 )
 
-AC_LANG_PUSH([C++])
-AC_CHECK_HEADER(gtest/gtest.h,,
-     AC_MSG_ERROR([failed: I need gtest to build tests]),
-)
-AC_LANG_POP([C++])
-
 
 AC_SUBST(BDBLIB)
 
diff -Naur orig/Makefile patched/Makefile
--- orig/Makefile	2015-05-22 17:44:31.000000000 +0200
+++ patched/Makefile	2015-05-24 14:09:57.522635355 +0200
@@ -21,7 +21,6 @@
 	$(MAKE) -C dselect $@
 	$(MAKE) -C doc $@
 	$(MAKE) -C po $@
-	$(MAKE) -C test $@
 
 all headers library clean veryclean binary program doc manpages docbook test update-po: startup dirs
 
