diff -Naur orig/unix/chessx.desktop mod/unix/chessx.desktop
--- orig/unix/chessx.desktop	2014-01-02 18:50:14.000000000 +0100
+++ mod/unix/chessx.desktop	2014-05-07 13:49:55.447550473 +0200
@@ -2,14 +2,15 @@
 Type=Application
 Name=ChessX
 Comment=Free chess database
-Comment=Chess database
 Comment[de]=Schachdatenbank
+Comment[hu]=Sakkadatbázis
 GenericName=Chess Game
 GenericName[de]=Schachspiel
+GenericName[hu]=Sakkjáték
 Icon=chessx
 Exec=chessx
 Terminal=false
 Categories=Game;BoardGame;
 MimeType=application/x-chess-pgn;
-Keywords=PGN;FEN;Database
-Keywords[de]=PGN;FEN;Datenbank
+Keywords=PGN;FEN;Database;
+Keywords[de]=PGN;FEN;Datenbank;
