diff -Naur orig/data/eom.desktop.in.in patched/data/eom.desktop.in.in
--- orig/data/eom.desktop.in.in	2014-02-19 19:44:47.000000000 +0100
+++ patched/data/eom.desktop.in.in	2014-02-24 19:52:35.021462156 +0100
@@ -7,7 +7,7 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GTK;Graphics;RasterGraphics;Viewer;
+Categories=GTK;Graphics;RasterGraphics;Viewer;X-UHU-Menu/Graphics;
 Keywords=image;viewer;JPEG;PNG;TIFF;SVG;MATE;photos;browse;thumbnails;rotate;
 X-MATE-Bugzilla-Bugzilla=MATE
 X-MATE-Bugzilla-Product=EOM
