diff -Naur orig/data/eom.desktop.in.in patched/data/eom.desktop.in.in
--- orig/data/eom.desktop.in.in	2013-03-24 16:19:41.000000000 +0100
+++ patched/data/eom.desktop.in.in	2013-04-01 21:34:19.471210070 +0200
@@ -7,7 +7,7 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GTK;Graphics;RasterGraphics;Viewer;
+Categories=GTK;Graphics;RasterGraphics;Viewer;X-UHU-Menu/Graphics;
 X-MATE-Bugzilla-Bugzilla=MATE
 X-MATE-Bugzilla-Product=EOM
 X-MATE-Bugzilla-Component=general
