diff -Naur a/src/kamoso.desktop b/src/kamoso.desktop
--- a/src/kamoso.desktop	2011-05-25 00:34:42.000000000 +0200
+++ b/src/kamoso.desktop	2013-06-01 12:34:48.273921452 +0200
@@ -8,6 +8,7 @@
 Name[et]=Kamoso
 Name[fr]=Kamoso
 Name[ga]=Kamoso
+Name[hu]=Kamoso
 Name[it]=Kamoso
 Name[mai]=Kamoso
 Name[nds]=Kamoso
@@ -27,6 +28,7 @@
 GenericName[es]=Capturador de imágenes
 GenericName[et]=Pildihankija
 GenericName[fr]=Outils de recherche d'images
+GenericName[hu]=Fényképező
 GenericName[it]=Acquisitore di immagini
 GenericName[nds]=Bildgrieper
 GenericName[nl]=Neemt een afbeelding
@@ -45,6 +47,7 @@
 Comment[es]=Hacer una foto con su cámara web
 Comment[et]=Pildi tegemine oma veebikaameraga
 Comment[fr]=Prenez un cliché quelconque avec votre webcam
+Comment[hu]=Kép mentése webkamerával
 Comment[it]=Fai qualsiasi foto con la tua webcam
 Comment[nds]=Jichtenseen Bild mit Dien Nettkamera opnehmen
 Comment[nl]=Neem elke afbeelding met uw webcam
@@ -60,4 +63,4 @@
 Icon=kamoso
 Type=Application
 Terminal=false
-Categories=Qt;KDE;Graphics;
+Categories=Qt;KDE;Graphics;X-UHU-Menu/Graphics;
