diff -Naur OpenGTL-0.9.17-orig/CMakeLists.txt OpenGTL-0.9.17/CMakeLists.txt
--- OpenGTL-0.9.17-orig/CMakeLists.txt	2012-06-04 16:51:43.000000000 +0200
+++ OpenGTL-0.9.17/CMakeLists.txt	2012-06-05 20:37:44.000000000 +0200
@@ -47,7 +47,7 @@
 
 find_package(LLVM REQUIRED)
 
-if( NOT MSVC AND NOT (LLVM_VERSION STREQUAL "3.1" OR LLVM_VERSION STREQUAL "3.0" ) ) # There is no way with MSVC to know the llvm version
+if( NOT MSVC AND NOT (LLVM_VERSION STREQUAL "3.1" OR LLVM_VERSION STREQUAL "3.1svn" OR LLVM_VERSION STREQUAL "3.0" ) ) # There is no way with MSVC to know the llvm version
   message(FATAL_ERROR "LLVM 3.0 or 3.1 is required.")
 endif()
 
