diff -Naur a/debian/usr/share/applications/Socializer.desktop b/debian/usr/share/applications/Socializer.desktop
--- a/debian/usr/share/applications/Socializer.desktop	2013-10-16 13:24:49.000000000 +0200
+++ b/debian/usr/share/applications/Socializer.desktop	2013-12-25 22:44:05.477441347 +0100
@@ -1,11 +1,11 @@
-#!/usr/bin/env xdg-open
 [Desktop Entry]
 Version=1.0
 Type=Application
 Terminal=false
-Exec=sh /opt/Socializer/Socializer.sh
-Comment[en_GB]=Facebook browser for your Ubuntu desktop.
+Exec=Socializer.sh
+Comment[en_GB]=Facebook browser for your desktop.
+Comment[hu]=Facebook böngésző.
 Name=Socializer
-Categories=
+Categories=Network;Application;X-UHU-Menu/Internet/Messengers;
 Comment=Facebook browser for your Ubuntu desktop.
-Icon=/opt/Socializer/Socializer.png
+Icon=Socializer
