diff -Naur orig/quitcount.desktop patched/quitcount.desktop
--- orig/quitcount.desktop	2014-05-29 21:01:32.000000000 +0200
+++ patched/quitcount.desktop	2014-05-29 22:11:36.887946624 +0200
@@ -4,6 +4,7 @@
 Icon=quitcount
 Categories=Utility;
 Comment=Simple counter showing your gains since you quit smoking
+Comment[hu]=Egyszerű számláló, amely megmutatja, mit nyertél a dohányzás abbahagyása óta
 Terminal=false
 Type=Application
 StartupNotify=false
