--- trunk/cdr/cdr_sqlite3_custom.c	None	252313
+++ trunk/cdr/cdr_sqlite3_custom.c	2010/03/14 17:43:46	252314
@@ -219,7 +219,7 @@
 	}
 }
 
-static int sqlite3_log(struct ast_cdr *cdr)
+static int write_cdr(struct ast_cdr *cdr)
 {
 	int res = 0;
 	char *error = NULL;
@@ -329,7 +329,7 @@
 		}
 	}
 
-	res = ast_cdr_register(name, desc, sqlite3_log);
+	res = ast_cdr_register(name, desc, write_cdr);
 	if (res) {
 		ast_log(LOG_ERROR, "Unable to register custom SQLite3 CDR handling\n");
 		free_config(0);
