From 3de5f1498b79814a6d973b98afc9a9b3e095c5cf Mon Sep 17 00:00:00 2001
From: Martin Wimpress <code@flexion.org>
Date: Tue, 28 Apr 2015 22:26:08 +0100
Subject: [PATCH] Fallback to .local for application edits.

---
 mate_menu/plugins/applications.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mate_menu/plugins/applications.py b/mate_menu/plugins/applications.py
index 5229f4a..52fdf00 100644
--- a/mate_menu/plugins/applications.py
+++ b/mate_menu/plugins/applications.py
@@ -121,7 +121,7 @@ def get_system_item_paths():
     if os.environ.has_key('XDG_DATA_DIRS'):
         item_dirs = os.environ['XDG_DATA_DIRS'].split(":")
     else:
-        item_dirs = [os.path.join('usr', 'share')]
+        item_dirs = [os.path.join(os.environ['HOME'], '.local', 'share')]
 
     return item_dirs
 
-- 
2.1.1
